<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>MDD TWAS</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/united.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">MDD-TWAS</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/opain/MDD-TWAS">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">MDD TWAS</h1>

</div>


<style>
p.caption {
  font-size: 1.5em;
}
</style>
<style type="text/css">
pre code, pre, code {
  white-space: pre !important;
  overflow-x: scroll !important;
  word-break: keep-all !important;
  word-wrap: initial !important;
}
</style>
<hr />
<p>This documents an MDD TWAS study. The analysis was carried out by Lorenza Dall’Aglio and Oliver Pain, with supervision from Cathryn Lewis.</p>
<p>This study used GWAS summary statistics from the Wray et al Major Depression GWAS, including 23andMe participants. We used SNP-weights derived by FUSION for brain tissues, HPA tissues, HPT tissues, and blood.</p>
<p>After reviewer comments, we also included SNP-weights derived by the PsychENCODE team for the DLPFC. The PsychENCODE SNP-weights were derived using all HRC imputed variants within the PsychENCODE dataset. The standard FUSION LD reference is restricted to HapMap3 variants, so when using the PsychENCODE SNP-weight we used an unrestricted version of the 1KG Phase 3 reference to improve SNP overlap.</p>
<hr />
<div id="estimating-transcriptome-wide-significance-threshold" class="section level1">
<h1><span class="header-section-number">1</span> Estimating transcriptome-wide significance threshold</h1>
<p>First we need to estimate our transcriptome-wide significance threshold to account for the number of features tested. We will use a permutation-based approach to determine the appropriate transcriptome-wide significance threshold. It works by performing TWAS using the relevent SNP-weight panels.</p>
<details>
<p><summary>Show a list of SNP-weight panels in the TWAS</summary></p>
<table>
<caption>SNP-weight panels used in MDD TWAS</caption>
<thead>
<tr class="header">
<th align="left">Panel</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Adrenal_Gland</td>
</tr>
<tr class="even">
<td align="left">Brain_Amygdala</td>
</tr>
<tr class="odd">
<td align="left">Brain_Anterior_cingulate_cortex_BA24</td>
</tr>
<tr class="even">
<td align="left">Brain_Caudate_basal_ganglia</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellar_Hemisphere</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellum</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cortex</td>
</tr>
<tr class="even">
<td align="left">Brain_Frontal_Cortex_BA9</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hippocampus</td>
</tr>
<tr class="even">
<td align="left">Brain_Hypothalamus</td>
</tr>
<tr class="odd">
<td align="left">Brain_Nucleus_accumbens_basal_ganglia</td>
</tr>
<tr class="even">
<td align="left">Brain_Putamen_basal_ganglia</td>
</tr>
<tr class="odd">
<td align="left">Brain_Substantia_nigra</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING</td>
</tr>
<tr class="even">
<td align="left">NTR.BLOOD.RNAARR</td>
</tr>
<tr class="odd">
<td align="left">Pituitary</td>
</tr>
<tr class="even">
<td align="left">Thyroid</td>
</tr>
<tr class="odd">
<td align="left">Whole_Blood</td>
</tr>
<tr class="even">
<td align="left">YFS.BLOOD.RNAARR</td>
</tr>
<tr class="odd">
<td align="left">PsychENCODE</td>
</tr>
</tbody>
</table>
</details>
<details>
<p><summary>Generate null distribution</summary></p>
<pre class="bash"><code>mkdir -p /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/EstTWSig

for batch in $(seq 1 20); do
sbatch -p brc,shared -n 1 --mem=10G /users/k1806347/brc_scratch/Software/Rscript.sh /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/EstTWSig/TWASPermuThr.R \
--nperm 50 \
--ncore 1 \
--weights /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWASweights_list_withPsychENCODE.txt \
--output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/EstTWSig/Batch${batch}
done</code></pre>
</details>
<details>
<p><summary>Estimate transcriptome-wide significance threshold</summary></p>
<pre class="r"><code>library(data.table)
library(MKmisc)

# Create list of files containing minimum p values
batches&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/EstTWSig/&#39;,pattern=&#39;Batch*&#39;)

# Combine all the minimum p-values
min_P_all&lt;-NULL
for(batch in batches){      
    min_P_all&lt;-c(min_P_all,fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/EstTWSig/&#39;,batch), header=F)$V1)
}

# Calculate the 5th percentile of the minimum p-values
TWalpha&lt;-MKmisc::quantileCI(x=min_P_all, prob=0.05, method=&quot;exact&quot;,conf.level=0.95)
TWalpha # 1.368572e-06

# Calculate the 0.1th percentile which will be needed for the high-confidence associations section
TWalpha_001&lt;-MKmisc::quantileCI(x=min_P_all, prob=0.001, method=&quot;exact&quot;,conf.level=0.99)
TWalpha_001 # 3.685926e-08

# Save the R object for future reference
saveRDS(TWalpha,file=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/EstTWSig/TWASsign_05.RDS&#39;)
saveRDS(TWalpha_001,file=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/EstTWSig/TWASsign_001.RDS&#39;)</code></pre>
</details>
<hr />
</div>
<div id="twas" class="section level1">
<h1><span class="header-section-number">2</span> TWAS</h1>
<hr />
<div id="fusion-snp-weights" class="section level2">
<h2><span class="header-section-number">2.1</span> FUSION SNP-weights</h2>
<hr />
<div id="gwas-summary-statistics-preparation" class="section level3">
<h3><span class="header-section-number">2.1.1</span> GWAS summary statistics preparation</h3>
<p>GWAS summary statistics were munged using the LDSC munge_sumstats.py.</p>
<details>
<p><summary>Show munge log file</summary></p>
<pre><code>## *********************************************************************
## * LD Score Regression (LDSC)
## * Version 1.0.0
## * (C) 2014-2015 Brendan Bulik-Sullivan and Hilary Finucane
## * Broad Institute of MIT and Harvard / MIT Department of Mathematics
## * GNU General Public License v3
## *********************************************************************
## Call: 
## ./munge_sumstats.py \
## --N-con-col Ncon \
## --out /mnt/lustre/groups/ukbiobank/sumstats/munged/DEPR01 \
## --merge-alleles /mnt/lustre/groups/ukbiobank/Edinburgh_Data/usr/helenaG/ldsc-master/w_hm3.snplist \
## --N-cas-col Ncas \
## --N-col N \
## --info-min 0.6 \
## --sumstats /mnt/lustre/groups/ukbiobank/sumstats/cleaned/DEPR01.gz 
## 
## Interpreting column names as follows:
## INFO:    INFO score (imputation quality; higher --&gt; better imputation)
## SNP: Variant ID (e.g., rs number)
## N:   Sample size
## A1:  Allele 1, interpreted as ref allele for signed sumstat.
## P:   p-Value
## A2:  Allele 2, interpreted as non-ref allele for signed sumstat.
## Ncon:    Number of controls
## Ncas:    Number of cases
## FREQ:    Allele frequency
## OR:  Odds ratio (1 --&gt; no effect; above 1 --&gt; A1 is risk increasing)
## 
## Reading list of SNPs for allele merge from /mnt/lustre/groups/ukbiobank/Edinburgh_Data/usr/helenaG/ldsc-master/w_hm3.snplist
## Read 1217311 SNPs for allele merge.
## Reading sumstats from /mnt/lustre/groups/ukbiobank/sumstats/cleaned/DEPR01.gz into memory 5000000.0 SNPs at a time.
## Read 10155339 SNPs from --sumstats file.
## Removed 8953119 SNPs not in --merge-alleles.
## Removed 0 SNPs with missing values.
## Removed 0 SNPs with INFO &lt;= 0.6.
## Removed 17182 SNPs with MAF &lt;= 0.01.
## Removed 0 SNPs with out-of-bounds p-values.
## Removed 0 variants that were not SNPs or were strand-ambiguous.
## 1185038 SNPs remain.
## Removed 0 SNPs with duplicated rs numbers (1185038 SNPs remain).
## Removed 0 SNPs with N &lt; 307422.666667 (1185038 SNPs remain).
## Median value of OR was 1.0, which seems sensible.
## Removed 0 SNPs whose alleles did not match --merge-alleles (1185038 SNPs remain).
## Writing summary statistics for 1217311 SNPs (1185038 with nonmissing beta) to /mnt/lustre/groups/ukbiobank/sumstats/munged/DEPR01.sumstats.gz.
## 
## Metadata:
## Mean chi^2 = 1.534
## Lambda GC = 1.419
## Max chi^2 = 79.045
## 597 Genome-wide significant SNPs (some may have been removed by filtering).
## 
## Conversion finished at Wed Feb 14 12:45:03 2018
## Total time elapsed: 2.0m:19.39s</code></pre>
</details>
<p>Then, I modified the sumstats to remove the rows with missing values.</p>
<details>
<p><summary>Show code</summary></p>
<pre class="bash"><code>module add general/R/3.5.0
R

library(data.table)

# Read in the LDSC munged sumstats using zcat to unzip the file
sumstats&lt;-data.frame(fread(&#39;zcat /mnt/lustre/groups/ukbiobank/sumstats/munged/DEPR01.sumstats.gz&#39;))

# Remove rows containing NA values
sumstats&lt;-sumstats[complete.cases(sumstats),]

# Save the reformatted sumstats and compress
write.table(sumstats, &#39;/users/k1806347/brc_scratch/Data/GWAS_sumstats/DEPR01.sumstats.noNA&#39;, col.names=T, row.names=F,quote=F)

q()
n

gzip /users/k1806347/brc_scratch/Data/GWAS_sumstats/DEPR01.sumstats.noNA
</code></pre>
</details>
<hr />
</div>
<div id="twas-analysis" class="section level3">
<h3><span class="header-section-number">2.1.2</span> TWAS analysis</h3>
<details>
<p><summary>Run TWAS</summary></p>
<pre class="bash"><code># Run analysis for each chromomsome and each panel
for chr in $(seq 1 22); do
  for weights in Adrenal_Gland Brain_Amygdala Brain_Anterior_cingulate_cortex_BA24 Brain_Caudate_basal_ganglia Brain_Cerebellar_Hemisphere Brain_Cerebellum Brain_Cortex Brain_Frontal_Cortex_BA9 Brain_Hippocampus Brain_Hypothalamus Brain_Nucleus_accumbens_basal_ganglia Brain_Putamen_basal_ganglia Brain_Substantia_nigra CMC.BRAIN.RNASEQ CMC.BRAIN.RNASEQ_SPLICING NTR.BLOOD.RNAARR Pituitary Thyroid Whole_Blood YFS.BLOOD.RNAARR; do
    qsub -cwd /mnt/lustre/users/k1894478/scripts/Rscript_correct.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/fusion_twas-master/FUSION.assoc_test.R \
      --sumstats /users/k1806347/brc_scratch/Data/GWAS_sumstats/DEPR01.sumstats.noNA.gz \
      --weights /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/SNP-weights/${weights}/${weights}.pos \
      --weights_dir /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/SNP-weights/${weights} \
      --ref_ld_chr /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/LDREF/1000G.EUR. \
      --out /mnt/lustre/users/k1894478/output_correct/wray_output/test.${weights}.chr${chr} \
      --chr ${chr} \
      --coloc_P 4.432625e-06 \
      --GWASN 480359
  done
done

# Check each chromosome finished
for weights in Adrenal_Gland Brain_Amygdala Brain_Anterior_cingulate_cortex_BA24 Brain_Caudate_basal_ganglia Brain_Cerebellar_Hemisphere Brain_Cerebellum Brain_Cortex Brain_Frontal_Cortex_BA9 Brain_Hippocampus Brain_Hypothalamus Brain_Nucleus_accumbens_basal_ganglia Brain_Putamen_basal_ganglia Brain_Substantia_nigra CMC.BRAIN.RNASEQ CMC.BRAIN.RNASEQ_SPLICING NTR.BLOOD.RNAARR Pituitary Thyroid Whole_Blood YFS.BLOOD.RNAARR; do
echo $weights
ls /mnt/lustre/users/k1894478/output_correct/wray_output/test.${weights}.chr* | wc -l  #this tells you how many files there are with the name test.X.chrX
done

# To combine per chromosome results file, without duplicating the header
for weights in Adrenal_Gland Brain_Amygdala Brain_Anterior_cingulate_cortex_BA24 Brain_Caudate_basal_ganglia Brain_Cerebellar_Hemisphere Brain_Cerebellum Brain_Cortex Brain_Frontal_Cortex_BA9 Brain_Hippocampus Brain_Hypothalamus Brain_Nucleus_accumbens_basal_ganglia Brain_Putamen_basal_ganglia Brain_Substantia_nigra CMC.BRAIN.RNASEQ CMC.BRAIN.RNASEQ_SPLICING NTR.BLOOD.RNAARR Pituitary Thyroid Whole_Blood YFS.BLOOD.RNAARR; do
head -n 1 /mnt/lustre/users/k1894478/output_correct/wray_output/test.${weights}.chr1 &gt; /mnt/lustre/users/k1894478/output_correct/wray_output/test.${weights}.GW      
tail -n +2 -q /mnt/lustre/users/k1894478/output_correct/wray_output/test.${weights}.chr* &gt;&gt; /mnt/lustre/users/k1894478/output_correct/wray_output/test.${weights}.GW    
done

# Create file containing results for all tissues
awk &#39;
    FNR==1 &amp;&amp; NR!=1 { while (/TWAS.P/) getline; }
    1 {print}
&#39; /mnt/lustre/users/k1894478/output_correct/wray_output/test.*.GW &gt;/mnt/lustre/users/k1894478/output_correct/wray_output/AllTissues.GW


# Check the .GW files which were created
ls *.GW

# Delete the per chromosome files
for chr in $(seq 1 22); do
  rm /mnt/lustre/users/k1894478/output_correct/wray_output/test.*.chr${chr}
done

rm /mnt/lustre/users/k1894478/output_correct/wray_output/test.*.chr6.MHC</code></pre>
</details>
<hr />
</div>
</div>
<div id="psychencode-snp-weights" class="section level2">
<h2><span class="header-section-number">2.2</span> PsychENCODE SNP-weights</h2>
<hr />
<div id="gwas-summary-statistics-preparation-1" class="section level3">
<h3><span class="header-section-number">2.2.1</span> GWAS summary statistics preparation</h3>
<p>The PsychENCODE SNP-weights are not restricted to HapMap3 variants, so we need to use unrestricted GWAS summary statistics also. To achieve this I use the FOCUS software munge script.</p>
<details>
<p><summary>Show code</summary></p>
<pre class="bash"><code># Edit BP column name
zcat /mnt/lustre/groups/ukbiobank/sumstats/cleaned/DEPR01.gz | sed -e &#39;s/ORIGBP/BP/g&#39; &gt; /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01_BP

# Munge
/users/k1806347/brc_scratch/Software/focus.sh munge /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01_BP --output /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01.focus

# Delete temporary file
rm /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01_BP</code></pre>
</details>
<details>
<p><summary>Show munge log file</summary></p>
<pre><code>## ===================================
##               FOCUS v0.6.10            
## ===================================
## focus munge
##  /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01_BP
##  --output /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01.focus
## 
## Starting log...
## [2019-11-14 10:54:16 - INFO] Interpreting column names as follows:
## [2019-11-14 10:54:16 - INFO] SNP: Variant ID (e.g., rs number)
## [2019-11-14 10:54:16 - INFO] CHR: Chromsome
## [2019-11-14 10:54:16 - INFO] BP: Base position
## [2019-11-14 10:54:16 - INFO] A1: Allele 1, interpreted as ref allele for signed sumstat
## [2019-11-14 10:54:16 - INFO] A2: Allele 2, interpreted as non-ref allele for signed sumstat
## [2019-11-14 10:54:16 - INFO] P: p-Value
## [2019-11-14 10:54:16 - INFO] INFO: INFO score (imputation quality; higher --&gt; better imputation)
## [2019-11-14 10:54:16 - INFO] OR: Odds ratio (1 --&gt; no effect; above 1 --&gt; A1 is risk increasing)
## [2019-11-14 10:54:16 - INFO] N: Sample size
## [2019-11-14 10:54:16 - INFO] Reading sumstats from /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01_BP into memory 5000000 SNPs at a time
## [2019-11-14 10:54:24 - INFO] Reading SNP chunk 1
## [2019-11-14 10:54:44 - INFO] Reading SNP chunk 2
## [2019-11-14 10:54:52 - INFO] Reading SNP chunk 3
## [2019-11-14 10:54:52 - INFO] Done reading SNP chunks
## [2019-11-14 10:54:58 - INFO] Read 10155339 SNPs from --sumstats file
## [2019-11-14 10:54:58 - INFO] Removed 0 SNPs with missing values
## [2019-11-14 10:54:58 - INFO] Removed 2561651 SNPs with INFO &lt;= 0.9
## [2019-11-14 10:54:58 - INFO] Removed 0 SNPs with MAF &lt;= 0.01
## [2019-11-14 10:54:58 - INFO] Removed 0 SNPs with out-of-bounds p-values
## [2019-11-14 10:54:58 - INFO] Removed 1656695 variants that were not SNPs or were strand-ambiguous
## [2019-11-14 10:54:58 - INFO] 5936993 SNPs remain
## [2019-11-14 10:55:04 - INFO] Removed 0 SNPs with duplicated rs numbers (5936993 SNPs remain).
## [2019-11-14 10:55:06 - INFO] Removed 188 SNPs with N &lt; 307422.6666666667 (5936805 SNPs remain)
## [2019-11-14 10:56:30 - INFO] Median value of OR was 1.0, which seems sensible.
## [2019-11-14 10:56:31 - INFO] Writing summary statistics for 5936805 SNPs (5936805 with nonmissing beta) to /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01.focus.sumstats.gz.
## [2019-11-14 10:57:53 - INFO] METADATA - Mean chi^2 = 1.492
## [2019-11-14 10:57:54 - INFO] METADATA - Lambda GC = 1.383
## [2019-11-14 10:57:54 - INFO] METADATA - Max chi^2 = 79.045
## [2019-11-14 10:57:54 - INFO] METADATA - 2960 Genome-wide significant SNPs (some may have been removed by filtering)
## [2019-11-14 10:57:54 - INFO] Conversion finished</code></pre>
</details>
<hr />
</div>
<div id="twas-analysis-1" class="section level3">
<h3><span class="header-section-number">2.2.2</span> TWAS analysis</h3>
<details>
<p><summary>Run TWAS</summary></p>
<pre class="bash"><code>for chr in $(seq 1 22); do
  sbatch -p brc,shared --mem=20G /users/k1806347/brc_scratch/Software/Rscript.sh /scratch/groups/biomarkers-brc-mh/TWAS_resource/FUSION/fusion_twas-master/FUSION.assoc_test.R \
    --sumstats /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01.focus.sumstats.gz \
    --weights /scratch/groups/biomarkers-brc-mh/TWAS_resource/PsychEncode/PEC_TWAS_weights/PEC_TWAS_weights.pos \
    --weights_dir /scratch/groups/biomarkers-brc-mh/TWAS_resource/PsychEncode/PEC_TWAS_weights \
    --ref_ld_chr /scratch/groups/biomarkers-brc-mh/Reference_data/1KG_Phase3/PLINK/EUR/EUR_phase3.MAF_001.chr \
    --out /users/k1806347/brc_scratch/Analyses/Lorenza/PsychENCODE/MDD_TWAS_PsychENCODE.chr${chr} \
    --chr ${chr} \
    --coloc_P 4.432625e-06 \
    --GWASN 480359
done</code></pre>
</details>
<details>
<p><summary>Format to match FUSION panel results</summary></p>
<pre class="r"><code>library(data.table)

psychENCODE&lt;-NULL
for(i in 1:22){
    if(i == 6){
        tmp1&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/PsychENCODE/MDD_TWAS_PsychENCODE.chr&#39;,i))
        tmp2&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/PsychENCODE/MDD_TWAS_PsychENCODE.chr&#39;,i,&#39;.MHC&#39;))
        tmp&lt;-rbind(tmp1,tmp2)
    } else {
        tmp&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/PsychENCODE/MDD_TWAS_PsychENCODE.chr&#39;,i))
    }
    
    psychENCODE&lt;-rbind(psychENCODE, tmp)
}

col_order&lt;-names(psychENCODE)

psychENCODE$PANEL&lt;-as.character(psychENCODE$PANEL)
psychENCODE$PANEL&lt;-&#39;PsychENCODE&#39;

library(biomaRt)
ensembl = useEnsembl(biomart=&quot;ensembl&quot;, dataset=&quot;hsapiens_gene_ensembl&quot;, GRCh=37)
listAttributes(ensembl)
Genes&lt;-getBM(attributes=c(&#39;ensembl_gene_id&#39;,&#39;external_gene_name&#39;), mart = ensembl)

psychENCODE&lt;-merge(psychENCODE, Genes, by.x=&#39;ID&#39;, by.y=&#39;ensembl_gene_id&#39;)
psychENCODE$ID&lt;-psychENCODE$external_gene_name
psychENCODE$external_gene_name&lt;-NULL

psychENCODE&lt;-psychENCODE[,col_order, with=F]

write.table(psychENCODE, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/PsychENCODE/MDD_TWAS_PsychENCODE.GW&#39;, col.names=T, row.names=F, quote=F)</code></pre>
</details>
<hr />
</div>
</div>
<div id="combine-results-across-fusion-and-psychencode-panels" class="section level2">
<h2><span class="header-section-number">2.3</span> Combine results across FUSION and PsychENCODE panels</h2>
<details>
<p><summary>Format to match FUSION panel results</summary></p>
<pre class="r"><code>library(data.table)

psych&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/PsychENCODE/MDD_TWAS_PsychENCODE.GW&#39;)
fusion&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/AllTissues.GW&#39;)

all&lt;-rbind(psych,fusion)

# Write out full results
write.table(all, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt&#39;, row.names=F, col.names=T, quote=F)

# Write out transcriptome-wide significant results
write.table(all[which(all$TWAS.P &lt; 1.368572e-06),], &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig.txt&#39;, row.names=F, col.names=T, quote=F)

dim(all[which(all$TWAS.P &lt; 1.368572e-06),]) # 176 hits
length(unique(all[which(all$TWAS.P &lt; 1.368572e-06),]$ID)) # 94 unique genes

# Check how much PsychENCODE adds
dim(fusion[which(fusion$TWAS.P &lt; 1.368572e-06),]) # 154 hits
length(unique(fusion[which(fusion$TWAS.P &lt; 1.368572e-06),]$ID)) # 84 unique genes</code></pre>
</details>
<hr />
</div>
</div>
<div id="post-twas" class="section level1">
<h1><span class="header-section-number">3</span> Post-TWAS</h1>
<hr />
<div id="create-manahattan-style-plots" class="section level2">
<h2><span class="header-section-number">3.1</span> Create Manahattan-style plots</h2>
<details>
<p><summary>Show code</summary></p>
<pre class="bash"><code># Manhattan plot based on permutation significance
/users/k1806347/brc_scratch/Software/Rscript_singularity.sh /scratch/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-plotter/TWAS-plotter.V1.0.r \
--twas /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt \
--sig_p 1.368572e-06 \
--output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_Manhattan \
--width 3500 \
--height 2500

#Manhattan plot for high confidence associations
/users/k1806347/brc_scratch/Software/Rscript_singularity.sh /scratch/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-plotter/TWAS-plotter.V1.0.r \
--twas /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt \
--sig_p 3.685926e-08 \
--output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_HighConf_Manhattan \
--width 3500 \
--height 2500
</code></pre>
</details>
<details>
<p><summary>Show Manhattan plots</summary></p>
<center>
<div class="figure">
<img src="Images/MDD_TWAS_AllTissues_Manhattan.png" alt="MDD TWAS Manhattan Plot with transcriptome-wide signficance" />
<p class="caption">MDD TWAS Manhattan Plot with transcriptome-wide signficance</p>
</div>
<hr />
<div class="figure">
<img src="Images/MDD_TWAS_AllTissues_HighConf_Manhattan.png" alt="MDD TWAS Manhattan Plot with high-confidence threshold" />
<p class="caption">MDD TWAS Manhattan Plot with high-confidence threshold</p>
</div>

</details>
<hr />
</div>
<div id="conditional-analysis" class="section level2">
<h2><span class="header-section-number">3.2</span> Conditional analysis</h2>
<details>
<p><summary>Run post_process.R script</summary></p>
<pre class="bash"><code># Change directory to location of glist-hg19 file
cd /users/k1806347/brc_scratch/Data/Gene_Locations

mkdir -p /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Conditional

for chr in $(seq 1 22); do

  status=$(awk -v var=&quot;${chr}&quot; &#39;$4 == var {print &quot;Present&quot;;exit;}&#39; /users/k1806347/brc_scratch/Analyses/Lorenza/PsychENCODE/post-TWAS/MDD_TWAS_AllTissues_TWSig.txt )
  
  if [ &quot;$status&quot; == &quot;Present&quot; ]; then
    sbatch -p brc,shared --mem 25G -n 1 /users/k1806347/brc_scratch/Software/Rscript_singularity.sh /scratch/groups/biomarkers-brc-mh/TWAS_resource/FUSION/fusion_twas-master/FUSION.post_process.R \
      --input /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig.txt \
      --sumstats /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01.focus.sumstats.gz \
      --report \
      --ref_ld_chr /scratch/groups/biomarkers-brc-mh/Reference_data/1KG_Phase3/PLINK/EUR/EUR_phase3.MAF_001.chr \
      --out /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Conditional/test.cond.chr${chr} \
      --chr ${chr} \
      --plot \
      --plot_legend all \
      --save_loci \
      --locus_win 500000
  fi

done</code></pre>
</details>
<hr />
</div>
<div id="process-twas-results" class="section level2">
<h2><span class="header-section-number">3.3</span> Process TWAS results</h2>
<details>
<p><summary>Clean the TWAS results</summary></p>
<pre class="r"><code>###
# Clean file PANEL names 
###

rm(list=ls())
library(data.table)
twas_sign &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig.txt&quot;)
twas &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt&quot;)

dim(twas)[1] # 97733 features in PANELS
sum(!is.na(twas$TWAS.P)) # 95720 features that could be imputed in GWAS
length(unique(twas$ID)) # 22006 unique genes in PANELS
length(unique(twas$ID[!is.na(twas$TWAS.P)])) # 21507 unique gene that could be imputed in GWAS

twas_sign$BEST.GWAS.P&lt;-2*pnorm(-abs(twas_sign$BEST.GWAS.Z))
sum(twas_sign$BEST.GWAS.P &gt; 5e-8) # 63
sum(twas_sign$BEST.GWAS.P &lt; 5e-8) # 113

str(twas_sign)
str(twas)

#clean the PANEL names of the output df containing results on all tested features
twas$PANEL_clean&lt;-gsub(&#39;_&#39;,&#39; &#39;,twas$PANEL)
twas$PANEL_clean&lt;-gsub(&#39;CMC.BRAIN.RNASEQ&#39;,&#39;CMC DLPFC&#39;,twas$PANEL_clean)
twas$PANEL_clean&lt;-gsub(&#39;SPLICING&#39;,&#39;Splicing&#39;,twas$PANEL_clean)
twas$PANEL_clean&lt;-gsub(&#39;NTR.BLOOD.RNAARR&#39;,&#39;NTR Blood&#39;,twas$PANEL_clean)
twas$PANEL_clean&lt;-gsub(&#39;YFS.BLOOD.RNAARR&#39;,&#39;YFS Blood&#39;,twas$PANEL_clean)
twas$PANEL_clean[!grepl(&#39;CMC|NTR|YFS|PsychENCODE&#39;, twas$PANEL)]&lt;-paste0(&#39;GTEx &#39;,twas$PANEL_clean[!grepl(&#39;CMC|NTR|YFS|PsychENCODE&#39;, twas$PANEL)])
#to add gtex to each of the snp weights which don&#39;t have CMC NTR or YFS in front
twas$PANEL_clean&lt;-gsub(&#39;Brain&#39;, &#39;&#39;, twas$PANEL_clean)
twas$PANEL_clean &lt;- gsub(&#39;Anterior cingulate cortex&#39;, &#39;ACC&#39;, twas$PANEL_clean)
twas$PANEL_clean &lt;- gsub(&#39;basal ganglia&#39;, &#39;&#39;, twas$PANEL_clean)
twas$PANEL_clean &lt;- gsub(&#39;BA9&#39;, &#39;&#39;, twas$PANEL_clean)
twas$PANEL_clean &lt;- gsub(&#39;BA24&#39;, &#39;&#39;, twas$PANEL_clean)
twas$PANEL_clean &lt;- gsub(&#39;  &#39;, &#39; &#39;, twas$PANEL_clean)

# Shorten panel name to plot easily
twas$PANEL_clean_short&lt;-substr(twas$PANEL_clean, start = 1, stop = 25)  #start the name at the first character and stop at the 25th
twas$PANEL_clean_short[nchar(twas$PANEL_clean) &gt; 25]&lt;-paste0(twas$PANEL_clean_short[nchar(twas$PANEL_clean) &gt; 25], &quot;...&quot;)

#do the same for the output file with sign features only
twas_sign$PANEL_clean&lt;-gsub(&#39;_&#39;,&#39; &#39;,twas_sign$PANEL)
twas_sign$PANEL_clean&lt;-gsub(&#39;CMC.BRAIN.RNASEQ&#39;,&#39;CMC DLPFC&#39;,twas_sign$PANEL_clean)
twas_sign$PANEL_clean&lt;-gsub(&#39;SPLICING&#39;,&#39;Splicing&#39;,twas_sign$PANEL_clean)
twas_sign$PANEL_clean&lt;-gsub(&#39;NTR.BLOOD.RNAARR&#39;,&#39;NTR Blood&#39;,twas_sign$PANEL_clean)
twas_sign$PANEL_clean&lt;-gsub(&#39;YFS.BLOOD.RNAARR&#39;,&#39;YFS Blood&#39;,twas_sign$PANEL_clean)
twas_sign$PANEL_clean[!grepl(&#39;CMC|NTR|YFS|PsychENCODE&#39;, twas_sign$PANEL)]&lt;-paste0(&#39;GTEx &#39;,twas_sign$PANEL_clean[!grepl(&#39;CMC|NTR|YFS|PsychENCODE&#39;, twas_sign$PANEL)])
#to add gtex to each of the snp weights which don&#39;t have CMC NTR or YFS in front
twas_sign$PANEL_clean&lt;-gsub(&#39;Brain&#39;, &#39;&#39;, twas_sign$PANEL_clean)
twas_sign$PANEL_clean &lt;- gsub(&#39;Anterior cingulate cortex&#39;, &#39;ACC&#39;, twas_sign$PANEL_clean)
twas_sign$PANEL_clean &lt;- gsub(&#39;basal ganglia&#39;, &#39;&#39;, twas_sign$PANEL_clean)
twas_sign$PANEL_clean &lt;- gsub(&#39;BA9&#39;, &#39;&#39;, twas_sign$PANEL_clean)
twas_sign$PANEL_clean &lt;- gsub(&#39;BA24&#39;, &#39;&#39;, twas_sign$PANEL_clean)
twas_sign$PANEL_clean &lt;- gsub(&#39;  &#39;, &#39; &#39;, twas_sign$PANEL_clean)

# Shorten panel name to plot easily
twas_sign$PANEL_clean_short&lt;-substr(twas_sign$PANEL_clean, start = 1, stop = 25)  #start the name at the first character and stop at the 25th
twas_sign$PANEL_clean_short[nchar(twas_sign$PANEL_clean) &gt; 25]&lt;-paste0(twas_sign$PANEL_clean_short[nchar(twas_sign$PANEL_clean) &gt; 25], &quot;...&quot;)

#check the variables
str(twas)
str(twas_sign)

###
# Deal with missingness and subset for the relevant cols only
###

##TWAS df
#exclude missings
twas&lt;-twas[!is.na(twas$TWAS.Z),]
twas&lt;-twas[!is.na(twas$TWAS.P),]

#subset columns needed 
twas_sub &lt;- twas[,c(&#39;FILE&#39;, &#39;ID&#39;,&#39;PANEL&#39;, &#39;PANEL_clean_short&#39;,&#39;PANEL_clean&#39;,&#39;CHR&#39;,&#39;P0&#39;, &#39;P1&#39;, &#39;TWAS.Z&#39;, &#39;TWAS.P&#39;, &#39;COLOC.PP0&#39;, &#39;COLOC.PP1&#39;, &#39;COLOC.PP2&#39;, &#39;COLOC.PP3&#39;, &#39;COLOC.PP4&#39;)]
str(twas_sub)

##TWAS sign df
#exclude missings
twas_sign&lt;-twas_sign[!is.na(twas_sign$TWAS.Z),]
twas_sign&lt;-twas_sign[!is.na(twas_sign$TWAS.P),]

#subset columns needed 
twas_sign_sub &lt;- twas_sign[,c(&#39;FILE&#39;, &#39;ID&#39;,&#39;PANEL&#39;, &#39;PANEL_clean_short&#39;,&#39;PANEL_clean&#39;,&#39;CHR&#39;,&#39;P0&#39;, &#39;P1&#39;, &#39;TWAS.Z&#39;, &#39;TWAS.P&#39;, &#39;COLOC.PP0&#39;, &#39;COLOC.PP1&#39;, &#39;COLOC.PP2&#39;, &#39;COLOC.PP3&#39;, &#39;COLOC.PP4&#39;)]
str(twas_sign_sub)

###
# Update positions
###

# Rationale: the positions in the output files created by FUSION are rounded, thus not completely accurate. 
# Therefore, we need to update the positions (P0 and P1) based on the pos files in Rosalind. This needs to be done on Putty though. A new file will be saved and reopened here. 

setwd(&#39;/mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/SNP-weights/&#39;)

weights&lt;-c(&#39;Adrenal_Gland&#39;, &#39;Brain_Amygdala&#39;, &#39;Brain_Anterior_cingulate_cortex_BA24&#39;, &#39;Brain_Caudate_basal_ganglia&#39;, &#39;Brain_Cerebellar_Hemisphere&#39;, &#39;Brain_Cerebellum&#39;, &#39;Brain_Cortex&#39;, &#39;Brain_Frontal_Cortex_BA9&#39;, &#39;Brain_Hippocampus&#39;, &#39;Brain_Hypothalamus&#39;, &#39;Brain_Nucleus_accumbens_basal_ganglia&#39;, &#39;Brain_Putamen_basal_ganglia&#39;, &#39;Brain_Substantia_nigra&#39;, &#39;CMC.BRAIN.RNASEQ&#39;, &#39;CMC.BRAIN.RNASEQ_SPLICING&#39;, &#39;NTR.BLOOD.RNAARR&#39;, &#39;Pituitary&#39;, &#39;Thyroid&#39;, &#39;Whole_Blood&#39;, &#39;YFS.BLOOD.RNAARR&#39;)

#Get all pos files within the SNP-weight sets and bind them 
FUSION_pos&lt;-NULL
for(i in weights){
FUSION_pos_temp&lt;-read.table(paste(i, &#39;/&#39;,i, &#39;.pos&#39;,sep=&#39;&#39;), header=T, stringsAsFactors=F)   #repeating i twice with / in the middle is to get one folder further 
FUSION_pos&lt;-rbind(FUSION_pos, FUSION_pos_temp)
}

PsychENCODE_pos&lt;-read.table(&#39;/scratch/groups/biomarkers-brc-mh/TWAS_resource/PsychEncode/PEC_TWAS_weights/PEC_TWAS_weights.pos&#39;, header=T, stringsAsFactors=F)
PsychENCODE_pos$PANEL&lt;-&#39;PsychENCODE&#39;

# Combine pos files
FUSION_pos&lt;-rbind(FUSION_pos, PsychENCODE_pos)

write.table(FUSION_pos,&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos&#39;, col.names=T, row.names=F, quote=F)

str(FUSION_pos)   #97733 observations of 7 variables (PANEL, WGT, ID, CHR, P0, P1, N)

###
# Merge the pos file with the twas_sub and twas_sign_sub df
###

#the pos file and the output file do not have the same columns with the same information. We therefore need to slightly modify the TWAS columns 
twas_sub$tmp&lt;-gsub(&#39;/mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/SNP-weights/&#39;,&#39;&#39;,twas_sub$FILE)
twas_sub$tmp&lt;-gsub(&#39;/scratch/groups/biomarkers-brc-mh/TWAS_resource/PsychEncode/PEC_TWAS_weights/&#39;,&#39;&#39;,twas_sub$tmp)
#to delete the full pathway of the file and just keep the important information 
twas_sub$PANEL&lt;-sub(&#39;/.*&#39;,&#39;&#39;, twas_sub$tmp)
twas_sub$Feature&lt;-gsub(&#39;.*/&#39;,&#39;&#39;,twas_sub$tmp)
twas_sub$WGT&lt;-paste0(twas_sub$PANEL, &#39;/&#39;, twas_sub$Feature)
twas_sub$PANEL&lt;-NULL
twas_sub$tmp&lt;-NULL
twas_sub$Feature&lt;-NULL

twas_sub[order(twas_sub$WGT), ]
FUSION_pos[order(FUSION_pos$WGT), ]

#merge
twas_sub_correct &lt;- merge(twas_sub, FUSION_pos, by=&quot;WGT&quot;)

#check
head(twas_sub_correct)

#clean
twas_sub_correct$ID.y&lt;-NULL
colnames(twas_sub_correct)
names(twas_sub_correct)[3]&lt;-&#39;ID&#39;   #to change the name  of IDx to ID
head(twas_sub_correct)
  
#repeat everything for the twas_sign_sub file
twas_sign_sub$tmp&lt;-gsub(&#39;/mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/SNP-weights/&#39;,&#39;&#39;,twas_sign_sub$FILE)
twas_sign_sub$tmp&lt;-gsub(&#39;/scratch/groups/biomarkers-brc-mh/TWAS_resource/PsychEncode/PEC_TWAS_weights/&#39;,&#39;&#39;,twas_sign_sub$tmp)

#to delete the full pathway of the file and just keep the important information 
twas_sign_sub$PANEL&lt;-sub(&#39;/.*&#39;,&#39;&#39;, twas_sign_sub$tmp)
twas_sign_sub$Feature&lt;-gsub(&#39;.*/&#39;,&#39;&#39;,twas_sign_sub$tmp)
twas_sign_sub$WGT&lt;-paste0(twas_sign_sub$PANEL, &#39;/&#39;, twas_sign_sub$Feature)
twas_sign_sub$PANEL&lt;-NULL
twas_sign_sub$tmp&lt;-NULL
twas_sign_sub$Feature&lt;-NULL

# twas_sign_sub[order(twas_sign_sub$WGT), ]

#merge
twas_sign_sub_correct &lt;- merge(twas_sign_sub, FUSION_pos, by=&quot;WGT&quot;)

#check
head(twas_sign_sub_correct)

#clean
twas_sign_sub_correct$ID.y&lt;-NULL
colnames(twas_sign_sub_correct)
names(twas_sign_sub_correct)[3]&lt;-&#39;ID&#39;   #to change the name  of IDx to ID
 
head(twas_sign_sub_correct)
dim(twas_sign_sub_correct)
dim(twas_sub_correct)

###
# Clean output files for future scripts
###

#clean both output files to have clean outputs to use in future scripts

#twas sign sub correct df
colnames(twas_sign_sub_correct)

names(twas_sign_sub_correct)[6] &lt;- &quot;CHR&quot;
twas_sign_sub_correct$CHR.y &lt;- NULL
colnames(twas_sign_sub_correct)
names(twas_sign_sub_correct)[17] &lt;- &quot;P0&quot; #turn POy into P0 - nb P0y is the one withh the more accurate positions
names(twas_sign_sub_correct)[18] &lt;- &quot;P1&quot; 

twas_sign_sub_correct$P0.x &lt;- NULL
twas_sign_sub_correct$P1.x &lt;- NULL


colnames(twas_sign_sub_correct)
twas_sign_sub_correct$N &lt;- NULL
str(twas_sign_sub_correct)

#change the variable types for those which are wrong 

#turn PO and P1 into numerical variables
twas_sign_sub_correct$P0 &lt;- as.numeric(as.character(twas_sign_sub_correct$P0))
twas_sign_sub_correct$P1 &lt;- as.numeric(as.character(twas_sign_sub_correct$P1))

str(twas_sign_sub_correct)

#twas_sub correct df
colnames(twas_sub_correct)

names(twas_sub_correct)[6]&lt;- &quot;CHR&quot;
names(twas_sub_correct)[18]&lt;- &quot;P0&quot;
names(twas_sub_correct)[19]&lt;- &quot;P1&quot;
twas_sub_correct$CHR.y &lt;- NULL
twas_sub_correct$P0.x &lt;- NULL
twas_sub_correct$P1.x &lt;- NULL

colnames(twas_sub_correct)
twas_sub_correct$N &lt;- NULL

str(twas_sub_correct)

#change variable type for PO and P1
twas_sub_correct$P0 &lt;- as.numeric(as.character(twas_sub_correct$P0))
twas_sub_correct$P1 &lt;- as.numeric(as.character(twas_sub_correct$P1))

#save 
write.table(twas_sub_correct, file = &quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_CLEAN.txt&quot;, sep = &quot; &quot;, col.names = T, row.names = F)
write.table(twas_sign_sub_correct, file = &quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig_CLEAN.txt&quot;, sep = &quot; &quot;, col.names = T, row.names=F)

q()
n</code></pre>
</details>
<details>
<p><summary>Create a table with the transcriptome-wide significant findings</summary></p>
<pre class="r"><code>rm(list=ls())
library(data.table)

twas_sign &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig_CLEAN.txt&quot;)

str(twas_sign)
twas_sign$CHR &lt;- as.numeric(as.character(twas_sign$CHR))
twas_sign$P0 &lt;- as.numeric(as.character(twas_sign$P0))
twas_sign$P1 &lt;- as.numeric(as.character(twas_sign$P1))

twas_sign &lt;- twas_sign[order(twas_sign$CHR, twas_sign$P0), ]

twas_sign$Location &lt;- paste0(&#39;chr&#39;,twas_sign$CHR,&#39;:&#39;,twas_sign$P0,&#39;-&#39;,twas_sign$P1) 

colnames(twas_sign)

library(dplyr)
library(tibble)
twas_sign &lt;- as_data_frame(twas_sign)

col_order &lt;- c(&quot;Location&quot;, &quot;ID&quot;, &quot;PANEL_clean_short&quot;, &quot;TWAS.Z&quot;, &quot;TWAS.P&quot;)

twas_sign_final &lt;- twas_sign[, col_order]

write.csv(twas_sign_final, &quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig_CLEAN.brief.csv&quot;, row.names=F)

twas_sign_final</code></pre>
</details>
<details>
<p><summary>Show transcriptome-wide significant table</summary></p>
<table>
<caption>Transcriptome-wide significant associations with Major Depression</caption>
<thead>
<tr class="header">
<th align="left">Location</th>
<th align="left">ID</th>
<th align="left">PANEL_clean_short</th>
<th align="right">TWAS.Z</th>
<th align="right">TWAS.P</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">chr1:8412457-8877702</td>
<td align="left">RERE</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-5.095707</td>
<td align="right">3.47e-07</td>
</tr>
<tr class="even">
<td align="left">chr1:8412457-8877702</td>
<td align="left">RERE</td>
<td align="left">YFS Blood</td>
<td align="right">-5.310078</td>
<td align="right">1.10e-07</td>
</tr>
<tr class="odd">
<td align="left">chr1:8484705-8494898</td>
<td align="left">RP5-1115A15.1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.175240</td>
<td align="right">2.28e-07</td>
</tr>
<tr class="even">
<td align="left">chr1:8484705-8494898</td>
<td align="left">RP5-1115A15.1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-4.866386</td>
<td align="right">1.14e-06</td>
</tr>
<tr class="odd">
<td align="left">chr1:36884051-36884179</td>
<td align="left">SNORA63</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">4.848870</td>
<td align="right">1.24e-06</td>
</tr>
<tr class="even">
<td align="left">chr1:71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Caudate</td>
<td align="right">5.780100</td>
<td align="right">7.47e-09</td>
</tr>
<tr class="odd">
<td align="left">chr1:71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Putamen</td>
<td align="right">5.548510</td>
<td align="right">2.88e-08</td>
</tr>
<tr class="even">
<td align="left">chr1:71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">8.760622</td>
<td align="right">1.94e-18</td>
</tr>
<tr class="odd">
<td align="left">chr1:72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-7.785520</td>
<td align="right">6.94e-15</td>
</tr>
<tr class="even">
<td align="left">chr1:72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-7.708820</td>
<td align="right">1.27e-14</td>
</tr>
<tr class="odd">
<td align="left">chr1:72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">PsychENCODE</td>
<td align="right">-7.742756</td>
<td align="right">9.73e-15</td>
</tr>
<tr class="even">
<td align="left">chr1:175873898-175889649</td>
<td align="left">RP11-318C24.2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.027510</td>
<td align="right">4.97e-07</td>
</tr>
<tr class="odd">
<td align="left">chr1:175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-4.958690</td>
<td align="right">7.10e-07</td>
</tr>
<tr class="even">
<td align="left">chr1:175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.039850</td>
<td align="right">4.66e-07</td>
</tr>
<tr class="odd">
<td align="left">chr1:175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.005960</td>
<td align="right">5.56e-07</td>
</tr>
<tr class="even">
<td align="left">chr1:181452685-181775921</td>
<td align="left">CACNA1E</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-4.989390</td>
<td align="right">6.06e-07</td>
</tr>
<tr class="odd">
<td align="left">chr1:197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC</td>
<td align="right">4.848374</td>
<td align="right">1.24e-06</td>
</tr>
<tr class="even">
<td align="left">chr1:197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.421950</td>
<td align="right">5.90e-08</td>
</tr>
<tr class="odd">
<td align="left">chr1:197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.018050</td>
<td align="right">5.22e-07</td>
</tr>
<tr class="even">
<td align="left">chr2:58386377-58468515</td>
<td align="left">FANCL</td>
<td align="left">CMC DLPFC</td>
<td align="right">-5.183180</td>
<td align="right">2.18e-07</td>
</tr>
<tr class="odd">
<td align="left">chr2:58386377-58468515</td>
<td align="left">FANCL</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.897476</td>
<td align="right">9.71e-07</td>
</tr>
<tr class="even">
<td align="left">chr2:197831741-198175897</td>
<td align="left">ANKRD44</td>
<td align="left">YFS Blood</td>
<td align="right">-5.690140</td>
<td align="right">1.27e-08</td>
</tr>
<tr class="odd">
<td align="left">chr2:198254508-198299815</td>
<td align="left">SF3B1</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">5.214900</td>
<td align="right">1.84e-07</td>
</tr>
<tr class="even">
<td align="left">chr3:44481261-44561226</td>
<td align="left">ZNF445</td>
<td align="left">CMC DLPFC</td>
<td align="right">-5.103280</td>
<td align="right">3.34e-07</td>
</tr>
<tr class="odd">
<td align="left">chr4:41937137-41962589</td>
<td align="left">TMEM33</td>
<td align="left">PsychENCODE</td>
<td align="right">4.837418</td>
<td align="right">1.32e-06</td>
</tr>
<tr class="even">
<td align="left">chr4:41983713-41988476</td>
<td align="left">DCAF4L1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.128000</td>
<td align="right">2.93e-07</td>
</tr>
<tr class="odd">
<td align="left">chr4:41990758-41991254</td>
<td align="left">RP11-814H16.2</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">5.009600</td>
<td align="right">5.45e-07</td>
</tr>
<tr class="even">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Amygdala</td>
<td align="right">-5.253400</td>
<td align="right">1.49e-07</td>
</tr>
<tr class="odd">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx ACC</td>
<td align="right">-5.001690</td>
<td align="right">5.68e-07</td>
</tr>
<tr class="even">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Caudate</td>
<td align="right">-4.854800</td>
<td align="right">1.21e-06</td>
</tr>
<tr class="odd">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Cortex</td>
<td align="right">-5.774530</td>
<td align="right">7.72e-09</td>
</tr>
<tr class="even">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">-5.085140</td>
<td align="right">3.67e-07</td>
</tr>
<tr class="odd">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">-5.602700</td>
<td align="right">2.11e-08</td>
</tr>
<tr class="even">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.259200</td>
<td align="right">1.45e-07</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564712-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">PsychENCODE</td>
<td align="right">6.091010</td>
<td align="right">1.12e-09</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.360090</td>
<td align="right">8.32e-08</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Amygdala</td>
<td align="right">6.118500</td>
<td align="right">9.45e-10</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx ACC</td>
<td align="right">6.445500</td>
<td align="right">1.15e-10</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Caudate</td>
<td align="right">6.282167</td>
<td align="right">3.34e-10</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">6.011700</td>
<td align="right">1.84e-09</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">6.053050</td>
<td align="right">1.42e-09</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cortex</td>
<td align="right">6.021420</td>
<td align="right">1.73e-09</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">6.720000</td>
<td align="right">1.82e-11</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">5.875800</td>
<td align="right">4.21e-09</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">6.010490</td>
<td align="right">1.85e-09</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Putamen</td>
<td align="right">6.372050</td>
<td align="right">1.87e-10</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Substantia nigra</td>
<td align="right">6.057270</td>
<td align="right">1.38e-09</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Pituitary</td>
<td align="right">6.048500</td>
<td align="right">1.46e-09</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">5.889760</td>
<td align="right">3.87e-09</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.526440</td>
<td align="right">3.27e-08</td>
</tr>
<tr class="odd">
<td align="left">chr5:87729709-87794514</td>
<td align="left">CTC-498M16.4</td>
<td align="left">GTEx Substantia nigra</td>
<td align="right">5.403610</td>
<td align="right">6.53e-08</td>
</tr>
<tr class="even">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx ACC</td>
<td align="right">-5.813700</td>
<td align="right">6.11e-09</td>
</tr>
<tr class="odd">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-5.861000</td>
<td align="right">4.60e-09</td>
</tr>
<tr class="even">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Cortex</td>
<td align="right">-6.510990</td>
<td align="right">7.47e-11</td>
</tr>
<tr class="odd">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">-7.091600</td>
<td align="right">1.33e-12</td>
</tr>
<tr class="even">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">PsychENCODE</td>
<td align="right">-6.097890</td>
<td align="right">1.07e-09</td>
</tr>
<tr class="odd">
<td align="left">chr5:140024947-140027370</td>
<td align="left">NDUFA2</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.190020</td>
<td align="right">2.10e-07</td>
</tr>
<tr class="even">
<td align="left">chr5:140201222-140203811</td>
<td align="left">PCDHA5</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.402970</td>
<td align="right">6.55e-08</td>
</tr>
<tr class="odd">
<td align="left">chr5:140220907-140223351</td>
<td align="left">PCDHA8</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-4.980100</td>
<td align="right">6.36e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:26188921-26189323</td>
<td align="left">HIST1H4D</td>
<td align="left">NTR Blood</td>
<td align="right">-4.987600</td>
<td align="right">6.11e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:26365386-26378540</td>
<td align="left">BTN3A2</td>
<td align="left">NTR Blood</td>
<td align="right">5.326600</td>
<td align="right">1.00e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">5.188200</td>
<td align="right">2.12e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">4.963000</td>
<td align="right">6.96e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Pituitary</td>
<td align="right">5.898930</td>
<td align="right">3.66e-09</td>
</tr>
<tr class="odd">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">5.481600</td>
<td align="right">4.22e-08</td>
</tr>
<tr class="even">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.086960</td>
<td align="right">3.64e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:26538633-26546482</td>
<td align="left">HMGN4</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.395400</td>
<td align="right">6.84e-08</td>
</tr>
<tr class="even">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-4.891200</td>
<td align="right">1.00e-06</td>
</tr>
<tr class="odd">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-4.947900</td>
<td align="right">7.50e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">-5.045000</td>
<td align="right">4.54e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.916080</td>
<td align="right">3.30e-09</td>
</tr>
<tr class="even">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-5.335920</td>
<td align="right">9.51e-08</td>
</tr>
<tr class="odd">
<td align="left">chr6:27325604-27339304</td>
<td align="left">ZNF204P</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">-5.032700</td>
<td align="right">4.84e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:27371789-27374743</td>
<td align="left">RP1-153G14.4</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">5.354000</td>
<td align="right">8.60e-08</td>
</tr>
<tr class="odd">
<td align="left">chr6:27418522-27440897</td>
<td align="left">ZNF184</td>
<td align="left">GTEx Caudate</td>
<td align="right">-6.325200</td>
<td align="right">2.53e-10</td>
</tr>
<tr class="even">
<td align="left">chr6:27418522-27440897</td>
<td align="left">ZNF184</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">-4.952200</td>
<td align="right">7.34e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:27840926-27841289</td>
<td align="left">HIST1H4L</td>
<td align="left">NTR Blood</td>
<td align="right">4.870800</td>
<td align="right">1.11e-06</td>
</tr>
<tr class="even">
<td align="left">chr6:28058932-28061442</td>
<td align="left">ZSCAN12P1</td>
<td align="left">PsychENCODE</td>
<td align="right">6.268010</td>
<td align="right">3.66e-10</td>
</tr>
<tr class="odd">
<td align="left">chr6:28058932-28061442</td>
<td align="left">ZSCAN12P1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-4.936930</td>
<td align="right">7.94e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:28083406-28084329</td>
<td align="left">RP1-265C24.5</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">5.532000</td>
<td align="right">3.16e-08</td>
</tr>
<tr class="odd">
<td align="left">chr6:28092338-28097860</td>
<td align="left">ZSCAN16</td>
<td align="left">YFS Blood</td>
<td align="right">-6.109000</td>
<td align="right">1.00e-09</td>
</tr>
<tr class="even">
<td align="left">chr6:28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-5.307800</td>
<td align="right">1.11e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">-6.017000</td>
<td align="right">1.77e-09</td>
</tr>
<tr class="even">
<td align="left">chr6:28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-6.159020</td>
<td align="right">7.32e-10</td>
</tr>
<tr class="odd">
<td align="left">chr6:28227098-28228736</td>
<td align="left">NKAPL</td>
<td align="left">PsychENCODE</td>
<td align="right">5.002860</td>
<td align="right">5.65e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.094600</td>
<td align="right">3.49e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.062800</td>
<td align="right">4.13e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">5.198000</td>
<td align="right">2.01e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Putamen</td>
<td align="right">5.739000</td>
<td align="right">9.52e-09</td>
</tr>
<tr class="even">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Thyroid</td>
<td align="right">5.338400</td>
<td align="right">9.38e-08</td>
</tr>
<tr class="odd">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.662330</td>
<td align="right">1.49e-08</td>
</tr>
<tr class="even">
<td align="left">chr6:28249314-28270326</td>
<td align="left">PGBD1</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-6.313100</td>
<td align="right">2.74e-10</td>
</tr>
<tr class="odd">
<td align="left">chr6:28292470-28324048</td>
<td align="left">ZSCAN31</td>
<td align="left">GTEx Amygdala</td>
<td align="right">-5.084150</td>
<td align="right">3.69e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Amygdala</td>
<td align="right">4.949900</td>
<td align="right">7.43e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">4.951000</td>
<td align="right">7.37e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Thyroid</td>
<td align="right">6.093300</td>
<td align="right">1.11e-09</td>
</tr>
<tr class="odd">
<td align="left">chr6:28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">-5.777500</td>
<td align="right">7.58e-09</td>
</tr>
<tr class="even">
<td align="left">chr6:28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Putamen</td>
<td align="right">-4.894000</td>
<td align="right">9.90e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-4.953290</td>
<td align="right">7.30e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:30644166-30655672</td>
<td align="left">PPP1R18</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">4.910200</td>
<td align="right">9.10e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.299700</td>
<td align="right">1.16e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.067100</td>
<td align="right">4.04e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.936600</td>
<td align="right">7.95e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-5.299000</td>
<td align="right">1.16e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.253270</td>
<td align="right">1.49e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.557400</td>
<td align="right">2.74e-08</td>
</tr>
<tr class="odd">
<td align="left">chr6:30881982-30894236</td>
<td align="left">VARS2</td>
<td align="left">GTEx Cortex</td>
<td align="right">5.922000</td>
<td align="right">3.18e-09</td>
</tr>
<tr class="even">
<td align="left">chr6:30881982-30894236</td>
<td align="left">VARS2</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">6.323130</td>
<td align="right">2.56e-10</td>
</tr>
<tr class="odd">
<td align="left">chr6:31255287-31256741</td>
<td align="left">WASF5P</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.156240</td>
<td align="right">2.52e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:31368479-31445283</td>
<td align="left">HCP5</td>
<td align="left">GTEx Thyroid</td>
<td align="right">6.400800</td>
<td align="right">1.55e-10</td>
</tr>
<tr class="odd">
<td align="left">chr6:31462658-31478901</td>
<td align="left">MICB</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.557000</td>
<td align="right">2.74e-08</td>
</tr>
<tr class="even">
<td align="left">chr6:31606805-31620482</td>
<td align="left">BAG6</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.580000</td>
<td align="right">2.40e-08</td>
</tr>
<tr class="odd">
<td align="left">chr6:31694815-31698357</td>
<td align="left">DDAH2</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">5.409500</td>
<td align="right">6.32e-08</td>
</tr>
<tr class="even">
<td align="left">chr6:31694816-31698039</td>
<td align="left">DDAH2</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.344500</td>
<td align="right">9.07e-08</td>
</tr>
<tr class="odd">
<td align="left">chr6:99817347-99842082</td>
<td align="left">COQ3</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.146560</td>
<td align="right">2.65e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:105404922-105531207</td>
<td align="left">LIN28B</td>
<td align="left">CMC DLPFC</td>
<td align="right">-5.232050</td>
<td align="right">1.68e-07</td>
</tr>
<tr class="odd">
<td align="left">chr6:105404923-105531207</td>
<td align="left">LIN28B</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.105689</td>
<td align="right">3.30e-07</td>
</tr>
<tr class="even">
<td align="left">chr6:105584224-105617820</td>
<td align="left">BVES-AS1</td>
<td align="left">GTEx Amygdala</td>
<td align="right">-5.578300</td>
<td align="right">2.43e-08</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.505026</td>
<td align="right">3.69e-08</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.790690</td>
<td align="right">7.01e-09</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.531000</td>
<td align="right">3.18e-08</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12276886</td>
<td align="left">TMEM106B</td>
<td align="left">YFS Blood</td>
<td align="right">5.373600</td>
<td align="right">7.72e-08</td>
</tr>
<tr class="odd">
<td align="left">chr7:24836158-25021253</td>
<td align="left">OSBPL3</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.622890</td>
<td align="right">1.88e-08</td>
</tr>
<tr class="even">
<td align="left">chr8:52232136-52722005</td>
<td align="left">PXDNL</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.887460</td>
<td align="right">3.92e-09</td>
</tr>
<tr class="odd">
<td align="left">chr8:61297147-61429354</td>
<td align="left">RP11-163N6.2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.336530</td>
<td align="right">9.47e-08</td>
</tr>
<tr class="even">
<td align="left">chr9:126605315-126605965</td>
<td align="left">PIGFP2</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.305600</td>
<td align="right">1.12e-07</td>
</tr>
<tr class="odd">
<td align="left">chr11:57067112-57092426</td>
<td align="left">TNKS1BP1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">4.922610</td>
<td align="right">8.54e-07</td>
</tr>
<tr class="even">
<td align="left">chr11:57405497-57420263</td>
<td align="left">AP000662.4</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-4.980256</td>
<td align="right">6.35e-07</td>
</tr>
<tr class="odd">
<td align="left">chr11:57424488-57429340</td>
<td align="left">CLP1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.195860</td>
<td align="right">2.04e-07</td>
</tr>
<tr class="even">
<td align="left">chr11:61535973-61560274</td>
<td align="left">TMEM258</td>
<td align="left">PsychENCODE</td>
<td align="right">5.021730</td>
<td align="right">5.12e-07</td>
</tr>
<tr class="odd">
<td align="left">chr11:113280318-113346111</td>
<td align="left">DRD2</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">-5.073787</td>
<td align="right">3.90e-07</td>
</tr>
<tr class="even">
<td align="left">chr13:53602875-53626196</td>
<td align="left">OLFM4</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.091290</td>
<td align="right">3.56e-07</td>
</tr>
<tr class="odd">
<td align="left">chr14:42057064-42074059</td>
<td align="left">CTD-2298J14.2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.678860</td>
<td align="right">1.36e-08</td>
</tr>
<tr class="even">
<td align="left">chr14:42076773-42373752</td>
<td align="left">LRFN5</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">5.423400</td>
<td align="right">5.85e-08</td>
</tr>
<tr class="odd">
<td align="left">chr14:42076773-42373752</td>
<td align="left">LRFN5</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.597540</td>
<td align="right">2.17e-08</td>
</tr>
<tr class="even">
<td align="left">chr14:59951161-59971429</td>
<td align="left">JKAMP</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.125100</td>
<td align="right">2.97e-07</td>
</tr>
<tr class="odd">
<td align="left">chr14:59971779-60043549</td>
<td align="left">CCDC175</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.478850</td>
<td align="right">4.28e-08</td>
</tr>
<tr class="even">
<td align="left">chr14:60062693-60337557</td>
<td align="left">RTN1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-4.874920</td>
<td align="right">1.09e-06</td>
</tr>
<tr class="odd">
<td align="left">chr14:60062695-60337684</td>
<td align="left">RTN1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.348450</td>
<td align="right">8.87e-08</td>
</tr>
<tr class="even">
<td align="left">chr14:64319682-64693151</td>
<td align="left">SYNE2</td>
<td align="left">NTR Blood</td>
<td align="right">5.609528</td>
<td align="right">2.03e-08</td>
</tr>
<tr class="odd">
<td align="left">chr14:64550950-64770377</td>
<td align="left">ESR2</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.982300</td>
<td align="right">2.20e-09</td>
</tr>
<tr class="even">
<td align="left">chr14:64550950-64770377</td>
<td align="left">ESR2</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-5.655371</td>
<td align="right">1.56e-08</td>
</tr>
<tr class="odd">
<td align="left">chr14:75120140-75179818</td>
<td align="left">AREL1</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.015110</td>
<td align="right">5.30e-07</td>
</tr>
<tr class="even">
<td align="left">chr14:75319736-75330537</td>
<td align="left">PROX2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.758100</td>
<td align="right">8.51e-09</td>
</tr>
<tr class="odd">
<td align="left">chr14:75348593-75370450</td>
<td align="left">DLST</td>
<td align="left">CMC DLPFC</td>
<td align="right">4.981400</td>
<td align="right">6.31e-07</td>
</tr>
<tr class="even">
<td align="left">chr14:75348594-75370448</td>
<td align="left">DLST</td>
<td align="left">PsychENCODE</td>
<td align="right">5.089700</td>
<td align="right">3.59e-07</td>
</tr>
<tr class="odd">
<td align="left">chr14:75370656-75389188</td>
<td align="left">RPS6KL1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.023810</td>
<td align="right">5.07e-07</td>
</tr>
<tr class="even">
<td align="left">chr14:75370657-75390099</td>
<td align="left">RPS6KL1</td>
<td align="left">PsychENCODE</td>
<td align="right">-4.952550</td>
<td align="right">7.32e-07</td>
</tr>
<tr class="odd">
<td align="left">chr14:103878456-103879098</td>
<td align="left">RP11-600F24.2</td>
<td align="left">PsychENCODE</td>
<td align="right">5.185660</td>
<td align="right">2.15e-07</td>
</tr>
<tr class="even">
<td align="left">chr14:103985996-103989448</td>
<td align="left">CKB</td>
<td align="left">YFS Blood</td>
<td align="right">5.346000</td>
<td align="right">8.99e-08</td>
</tr>
<tr class="odd">
<td align="left">chr14:103995508-104003410</td>
<td align="left">TRMT61A</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.051300</td>
<td align="right">4.39e-07</td>
</tr>
<tr class="even">
<td align="left">chr14:103995521-104003410</td>
<td align="left">TRMT61A</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">4.977593</td>
<td align="right">6.44e-07</td>
</tr>
<tr class="odd">
<td align="left">chr14:104019758-104028214</td>
<td align="left">RP11-894P9.2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.462560</td>
<td align="right">4.69e-08</td>
</tr>
<tr class="even">
<td align="left">chr14:104153913-104154464</td>
<td align="left">RP11-73M18.6</td>
<td align="left">PsychENCODE</td>
<td align="right">5.031320</td>
<td align="right">4.87e-07</td>
</tr>
<tr class="odd">
<td align="left">chr14:104160897-104161507</td>
<td align="left">RP11-73M18.7</td>
<td align="left">PsychENCODE</td>
<td align="right">4.856130</td>
<td align="right">1.20e-06</td>
</tr>
<tr class="even">
<td align="left">chr14:104162690-104163500</td>
<td align="left">RP11-73M18.8</td>
<td align="left">GTEx Amygdala</td>
<td align="right">5.142000</td>
<td align="right">2.72e-07</td>
</tr>
<tr class="odd">
<td align="left">chr14:104177607-104179149</td>
<td align="left">AL049840.1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.029540</td>
<td align="right">4.92e-07</td>
</tr>
<tr class="even">
<td align="left">chr14:104177607-104179149</td>
<td align="left">AL049840.1</td>
<td align="left">GTEx Cortex</td>
<td align="right">5.143620</td>
<td align="right">2.69e-07</td>
</tr>
<tr class="odd">
<td align="left">chr14:104179904-104180441</td>
<td align="left">RP11-73M18.9</td>
<td align="left">GTEx Cortex</td>
<td align="right">4.977330</td>
<td align="right">6.45e-07</td>
</tr>
<tr class="even">
<td align="left">chr14:104179904-104180586</td>
<td align="left">RP11-73M18.9</td>
<td align="left">PsychENCODE</td>
<td align="right">4.830100</td>
<td align="right">1.36e-06</td>
</tr>
<tr class="odd">
<td align="left">chr16:72146056-72210777</td>
<td align="left">PMFBP1</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.160620</td>
<td align="right">2.46e-07</td>
</tr>
<tr class="even">
<td align="left">chr17:27400528-27507430</td>
<td align="left">MYO18A</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">-5.128570</td>
<td align="right">2.92e-07</td>
</tr>
<tr class="odd">
<td align="left">chr17:27401933-27405875</td>
<td align="left">TIAF1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">-5.361200</td>
<td align="right">8.27e-08</td>
</tr>
<tr class="even">
<td align="left">chr17:65520597-65521538</td>
<td align="left">CTD-2653B5.1</td>
<td align="left">PsychENCODE</td>
<td align="right">5.105730</td>
<td align="right">3.30e-07</td>
</tr>
<tr class="odd">
<td align="left">chr18:52385091-52562747</td>
<td align="left">RAB27B</td>
<td align="left">PsychENCODE</td>
<td align="right">5.012900</td>
<td align="right">5.36e-07</td>
</tr>
<tr class="even">
<td align="left">chr18:52495707-52562747</td>
<td align="left">RAB27B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.843190</td>
<td align="right">1.28e-06</td>
</tr>
<tr class="odd">
<td align="left">chr20:47835831-47860614</td>
<td align="left">DDX27</td>
<td align="left">CMC DLPFC</td>
<td align="right">4.836260</td>
<td align="right">1.32e-06</td>
</tr>
<tr class="even">
<td align="left">chr22:41165634-41215403</td>
<td align="left">SLC25A17</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">5.076990</td>
<td align="right">3.83e-07</td>
</tr>
<tr class="odd">
<td align="left">chr22:41165634-41215403</td>
<td align="left">SLC25A17</td>
<td align="left">GTEx Thyroid</td>
<td align="right">4.896100</td>
<td align="right">9.78e-07</td>
</tr>
<tr class="even">
<td align="left">chr22:41253088-41351450</td>
<td align="left">XPNPEP3</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">4.951000</td>
<td align="right">7.38e-07</td>
</tr>
<tr class="odd">
<td align="left">chr22:41258260-41363888</td>
<td align="left">XPNPEP3</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.110000</td>
<td align="right">3.21e-07</td>
</tr>
<tr class="even">
<td align="left">chr22:41487790-41576081</td>
<td align="left">EP300</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.493900</td>
<td align="right">3.93e-08</td>
</tr>
<tr class="odd">
<td align="left">chr22:41487790-41576081</td>
<td align="left">EP300</td>
<td align="left">YFS Blood</td>
<td align="right">5.059100</td>
<td align="right">4.21e-07</td>
</tr>
<tr class="even">
<td align="left">chr22:41641614-41682216</td>
<td align="left">RANGAP1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.240100</td>
<td align="right">1.61e-07</td>
</tr>
<tr class="odd">
<td align="left">chr22:41641615-41682255</td>
<td align="left">RANGAP1</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.575273</td>
<td align="right">2.47e-08</td>
</tr>
<tr class="even">
<td align="left">chr22:41697526-41756151</td>
<td align="left">ZC3H7B</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.729100</td>
<td align="right">1.01e-08</td>
</tr>
</tbody>
</table>
</details>
<hr />
</div>
<div id="plot-twas-results" class="section level2">
<h2><span class="header-section-number">3.4</span> Plot TWAS results</h2>
<details>
<p><summary>Create QQ-plot and histogram of p-values</summary></p>
<pre class="r"><code>####
#QQplot
###

#load file with all hits - not just sign, ones
twas_sub_correct &lt;- read.table(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_CLEAN.txt&quot;, header=T, stringsAsFactors = F)

ggd.qqplot = function(pvector, main=NULL, ...) {
  o = -log10(sort(pvector,decreasing=F))
  e = -log10( 1:length(o)/length(o) )
  plot(e,o,pch=19,cex=1, main=main, ...,
       xlab=expression(Expected~~-log[10](italic(p))),
       ylab=expression(Observed~~-log[10](italic(p))),
       xlim=c(0,max(e)), ylim=c(0,max(o)))
  lines(e,e,col=&quot;red&quot;)
}

pvalues &lt;- twas_sub_correct$TWAS.P

# Add a title
png(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_QQplot.png&quot;,width = 2000, height = 2000, units = &quot;px&quot;, res=300)
ggd.qqplot(pvalues, &quot;QQ-plot of TWAS p-values&quot;)
dev.off() 

###
# Histogram of p-values
###

library(ggplot2)

## HISTOGRAM OF P-VALUES
png(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_pValHist.png&quot;,width = 2000, height = 2000, units = &quot;px&quot;, res=300)
hist(twas_sub_correct$TWAS.P,
     main = &quot;Histogram of TWAS p-values&quot;,  
     xlab =&quot;P-values&quot;, 
     ylab = &quot;Frequency&quot;)
dev.off()</code></pre>
</details>
<details>
<p><summary>Show plots</summary></p>
<center>
<div class="figure">
<img src="Images/MDD_TWAS_AllTissues_QQplot.png" alt="MDD TWAS QQ-plot" />
<p class="caption">MDD TWAS QQ-plot</p>
</div>
<p>As shown below in the QQ-plot, our p-values were smaller than expected, indicating the presence of multiple significant associations. Inflation is present, but this expected due to the polygenicity of Major Depression and the the correlation between predicted expression of genes.</p>
<hr />
<div class="figure">
<img src="Images/MDD_TWAS_AllTissues_pValHist.png" alt="MDD TWAS P-value histogram" />
<p class="caption">MDD TWAS P-value histogram</p>
</div>
<p>As shown in the histogram of p-values, our p-values followed a normal distribution as evidenced in the bottom of the graph where a similar amount of p-values is present. Additionally, a peak in correspondence to very small p-values is present at the top of the graph, indicating the presence of signal for our alternative hypothesis.</p>

</details>
<details>
<p><summary>Create heatmaps for shared and unique associations</summary></p>
<pre class="r"><code># Given the high number of hits we identified (N=177), from 91 unique genes, a single heatmap representing all of such genes cannot be created. Therefore, we depicted our results in two heatmaps: 1) heatmap of genes differentially expressed across multiple SNP-weight sets &amp; 2) heatmap of genes differentially across a single SNP-weight

rm(list=ls())
library(data.table)
library(ggplot2)
library(cowplot)

#load data
twas &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_CLEAN.txt&quot;)
head(twas)

# Extract only certain columns
twas_sub &lt;- twas[,c(&#39;ID&#39;,&#39;PANEL_clean_short&#39;,&#39;CHR&#39;,&#39;P0&#39;,&#39;P1&#39;,&#39;TWAS.Z&#39;, &#39;TWAS.P&#39;)]
str(twas_sub)
str(twas)

#filter for sign. gene IDs only 
sign_feat&lt;-twas_sub[twas_sub$TWAS.P &lt; 1.368572e-06,&#39;ID&#39;]  #to get a vector with the gene IDs of the significant features
str(sign_feat)  #find 176 features as supposed to - genes are repeated though --&gt; you need single gene IDs

sign_genes&lt;-unique(sign_feat)  #to identify unique gene IDs
str(sign_genes)  #94 gene IDs as expected

twas_sub[order(twas_sub$ID), ]
sign_genes[order(sign_genes$ID), ]

twas_sub &lt;- twas_sub[(twas_sub$ID %in% sign_genes$ID), ]
str(twas_sub)  #611 observations where each observation corresponds to an ID in the sign-feat vector

#prepare a vector of gene IDs which are duplicated
duplicates_df&lt;-sign_feat[duplicated(sign_feat)] 
duplicates &lt;- duplicates_df$ID

# The following code is necessary to depict results from the CMC DLPFC splicing panel 
# Since in RNA-seq splicing multiple transcripts of the same gene are generally tested and we can depict just one in the heatmap, we need to pick the most significant one. This is done below. 

### 
# RETAIN THE MOST SIGN. FEATURE FROM THE CMC BRAIN SPLICING RESULTS
###

twas_sub &lt;- twas_sub[order(twas_sub$TWAS.P), ] #we order by p-value to make sure that the most significant 
#CMC DLPFC feature for a given gene is kept and that its duplicates (which are less sign.) are excluded
head(twas_sub)
tail(twas_sub)

library(dplyr)
twas_sub  &lt;- twas_sub %&gt;% distinct(ID, PANEL_clean_short, .keep_all = T)  #to get rid of rows which contain duplicates based on the ID and PANEL cols 
#in our case, this is just for duplicates in the cmc brain splicing weights with the same gene id

twas_sub_temp &lt;- twas_sub[order(twas_sub$ID), ] #to check that it worked 
#there should be just one of the same gene id from brain seq splicing weights
#this is the case, with the mmost sign. one being kept. 
#NB 43 gene IDs from the same snp weight were gotten rid of in this df for a total of 568 features instead of 611 

####
# Heatmap of genes differentially expressed across multiple SNP-weight sets
####
###
# make a list of features significant in multiple tissues
###

#order files  
twas_sub2&lt;-twas_sub[order(twas_sub$ID),] 
duplicates&lt;-sort(duplicates, decreasing = FALSE)   

#filter the twas_sub datatable by the duplicates vector to obtain a dt with features expressed across diff. tissues only (i.e. no unique features)
twas_sub2&lt;-twas_sub[(twas_sub$ID %in% duplicates), ] #283 obs.
str(twas_sub2)

# Sort the data.frame by CHR and P0 
twas_sub2&lt;-twas_sub2[order(twas_sub2$CHR,twas_sub2$P0),]

# Make ID a factor for plotting where unique gene IDs are the levels/categories of such factor 
twas_sub2$ID&lt;-factor(twas_sub2$ID, levels=unique(twas_sub2$ID))
str(twas_sub2) #there are 36 levels (i.e. 36 unique genes differentially expressed across multiple weights)

#create a vector with all TWAS.Z values
TWAS.Z &lt;- twas_sub2$TWAS.Z

twas_sub2_unique&lt;-twas_sub2[!duplicated(twas_sub2$ID),]
vline_1&lt;-min(which(twas_sub2_unique$CHR == 6 &amp; twas_sub2_unique$P0 &gt; 26e6 &amp; twas_sub2_unique$P1 &lt; 34e6))
vline_2&lt;-max(which(twas_sub2_unique$CHR == 6 &amp; twas_sub2_unique$P0 &gt; 26e6 &amp; twas_sub2_unique$P1 &lt; 34e6))

#create the heatmap
png(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS.TWAS_Z_heatmap.shared.png&quot;,width = 3500, height = 1750, units = &quot;px&quot;, res=300)

ggplot(data = twas_sub2, aes(x = ID, y = PANEL_clean_short)) +
  #genes as x axis, panel as y axis  
  theme_bw()    +    #saying that there will be grid lines
  geom_tile(aes(fill = TWAS.Z), colour = &#39;black&#39;) +
  scale_fill_gradientn(colours=c(&quot;dodgerblue2&quot;,&quot;white&quot;,&quot;red&quot;), na.value = &#39;white&#39;,name = &quot;Z-score&quot;) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1),plot.title = element_text(hjust = 0.5)) +
  geom_text(aes(label=round(TWAS.Z,1)), color=&quot;black&quot;, size=3) +
  labs(title=&quot;Genes differentially expressed across multiple SNP-weight sets&quot;,  x =&quot;Gene ID&quot;, y = &quot;SNP-weight sets&quot;) +
  geom_vline(xintercept = vline_1-0.5, size=1) +
  geom_vline(xintercept = vline_2+0.5, size=1)

#title and labels shown for the x and y axes
dev.off()

# Note significant features presented a z-score &gt; 4.83 or &lt; -4.83.  

#### 
# Heatmap of genes differentially expressed in one SNP-weight 
####

#create a dt with the duplicates removed (!), so that we obtain only unique features
twas_unique&lt;-twas_sub[(!twas_sub$ID %in% duplicates), ]  
str(twas_unique)   #we get 285 features from differentially expressed genes uniquely differentially expressed in one tissue

# Sort the data.frame by CHR and P0
twas_unique&lt;-twas_unique[order(twas_unique$CHR,twas_unique$P0),]  #it&#39;s important to order by both
#if you order by pos only, the chr11 stuff will come first!

# Make ID a factor for plotting where unique gene IDs are the levels/categories of such factor 
twas_unique$ID&lt;-factor(twas_unique$ID, levels=unique(twas_unique$ID))
str(twas_unique)

#create a vector with all TWAS.Z values
TWAS.Z &lt;- twas_unique$TWAS.Z

twas_unique_unique&lt;-twas_unique[!duplicated(twas_unique$ID),]
vline_1&lt;-min(which(twas_unique_unique$CHR == 6 &amp; twas_unique_unique$P0 &gt; 26e6 &amp; twas_unique_unique$P1 &lt; 34e6))
vline_2&lt;-max(which(twas_unique_unique$CHR == 6 &amp; twas_unique_unique$P0 &gt; 26e6 &amp; twas_unique_unique$P1 &lt; 34e6))

#create the heatmap
png(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS.TWAS_Z_heatmap.unique.png&quot;,width = 4650, height = 1750, units = &quot;px&quot;, res=300)

ggplot(data = twas_unique, aes(x = ID, y = PANEL_clean_short)) +
  theme_bw()    +    
  geom_tile(aes(fill = TWAS.Z), colour = &#39;black&#39;) +
  scale_fill_gradientn(colours=c(&quot;dodgerblue2&quot;,&quot;white&quot;,&quot;red&quot;), na.value = &#39;white&#39;,name = &quot;Z-score&quot;) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1),plot.title = element_text(hjust = 0.5)) +
  geom_text(aes(label=round(TWAS.Z,1)), color=&quot;black&quot;, size=3) +
  labs(title=&quot;Genes differentially expressed in single SNP-weight sets&quot;,  x =&quot;Gene ID&quot;, y = &quot;SNP-weight sets&quot;) +
  geom_vline(xintercept = vline_1-0.5, size=1) +
  geom_vline(xintercept = vline_2+0.5, size=1)

dev.off()

# Note significant features presented a z-score &gt; 4.83 or &lt; -4.83.  </code></pre>
</details>
<details>
<p><summary>Create heatmaps for tissue groups</summary></p>
<pre class="r"><code># Heatmaps for groups of tissues were made to show the overlap across SNP-weight panels. 

rm(list=ls())
library(data.table)
library(ggplot2)
library(cowplot)

###
# Load and prepare data
###

#load
twas &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_CLEAN.txt&quot;)

#subset columns needed 
twas_sub &lt;- twas[,c(&#39;ID&#39;,&#39;PANEL&#39;, &#39;PANEL_clean_short&#39;, &#39;CHR&#39;,&#39;P0&#39;, &#39;P1&#39;, &#39;TWAS.Z&#39;, &#39;TWAS.P&#39;)]
str(twas_sub)

#turn CHR and P0 into numerical variables 
twas_sub$CHR &lt;- as.numeric(as.character(twas_sub$CHR))
twas_sub$P0 &lt;- as.numeric(as.character(twas_sub$P0))
str(twas_sub)

###
# Keep only the most sign. feature from the CMC brain splicing results
###
twas_sub2 &lt;- twas_sub[order(twas_sub$TWAS.P), ]
head(twas_sub2)
tail(twas_sub2)

library(dplyr)
twas_sub  &lt;- twas_sub2 %&gt;% distinct(ID, PANEL, .keep_all = T)


###
# Create df for groups of tissues
###

#order by CHR and P0 first
twas2 &lt;- twas_sub[order(twas_sub$CHR,twas_sub$P0),]
twas_sub &lt;- twas2


#create df
twas_brain.df &lt;- twas_sub[twas_sub$PANEL %in% c(&quot;Brain_Amygdala&quot;, &quot;Brain_Anterior_cingulate_cortex_BA24&quot;, &quot;Brain_Caudate_basal_ganglia&quot;, &quot;Brain_Cerebellar_Hemisphere&quot;, &quot;Brain_Cerebellum&quot;, &quot;Brain_Cortex&quot;, &quot;Brain_Frontal_Cortex_BA9&quot;, &quot;Brain_Hippocampus&quot;, &quot;Brain_Hypothalamus&quot;, &quot;Brain_Nucleus_accumbens_basal_ganglia&quot;, &quot;Brain_Putamen_basal_ganglia&quot;, &quot;Brain_Substantia_nigra&quot;, &quot;CMC.BRAIN.RNASEQ&quot;, &quot;CMC.BRAIN.RNASEQ_SPLICING&quot;,&quot;PsychENCODE&quot;), ]
twas_blood.df &lt;- twas_sub[twas_sub$PANEL %in% c(&quot;Whole_Blood&quot;, &quot;NTR.BLOOD.RNAARR&quot;, &quot;YFS.BLOOD.RNAARR&quot;), ]
twas_HPA.df &lt;- twas_sub[twas_sub$PANEL %in% c(&quot;Brain_Hypothalamus&quot;, &quot;Pituitary&quot;, &quot;Adrenal_Gland&quot;), ]
twas_HPT.df &lt;- twas_sub[twas_sub$PANEL %in% c(&quot;Brain_Hypothalamus&quot;, &quot;Pituitary&quot;,&quot;Thyroid&quot;), ]

###
#filter for gene IDs significant in a given group of tissues only 
###

#get sign. gene ID per group of tissues
sign_feat_brain&lt;-twas_brain.df[twas_brain.df$TWAS.P &lt; 1.368572e-06,]$ID  #to get a vector with the gene IDs of the significant features
#101 features are sign. within brain snp weight sets
str(sign_feat_brain)
sign_feat_brain &lt;- unique(sign_feat_brain)  #111 unique genes differentially expressed in brain snp weights


sign_feat_blood&lt;-twas_blood.df[twas_blood.df$TWAS.P &lt; 1.368572e-06,]$ID 
str(sign_feat_blood) #26
sign_feat_blood &lt;- unique(sign_feat_blood)#23


sign_feat_HPA&lt;-twas_HPA.df[twas_HPA.df$TWAS.P &lt; 1.368572e-06,]$ID  #to get a vector with the gene IDs of the significant features
str(sign_feat_HPA) #28
sign_feat_HPA &lt;- unique(sign_feat_HPA)  #22


sign_feat_HPT&lt;-twas_HPT.df[twas_HPT.df$TWAS.P &lt; 1.368572e-06,]$ID  #to get a vector with the gene IDs of the significant features
str(sign_feat_HPT) #41
sign_feat_HPT &lt;- unique(sign_feat_HPT)  #34

#filter
twas_brain.df &lt;- twas_brain.df[(twas_brain.df$ID %in% sign_feat_brain), ]
#320 obs.
twas_blood.df &lt;- twas_blood.df[(twas_blood.df$ID %in% sign_feat_blood), ] #32
twas_HPA.df &lt;- twas_HPA.df[(twas_HPA.df$ID %in% sign_feat_HPA), ] #32
twas_HPT.df &lt;- twas_HPT.df[(twas_HPT.df$ID %in% sign_feat_HPT), ] #51 features with the gene ID within the vector sign feat...

#create vectors with the z scores of the features within specific tissues
TWAS.Z.brain &lt;- twas_brain.df$TWAS.Z
TWAS.Z.blood &lt;- twas_blood.df$TWAS.Z
TWAS.Z.HPA &lt;- twas_HPA.df$TWAS.Z
TWAS.Z.HPT &lt;- twas_HPT.df$TWAS.Z

# Make ID a factor for plotting
twas_brain.df$ID&lt;-factor(twas_brain.df$ID, levels = unique(twas_brain.df$ID))
twas_blood.df$ID&lt;-factor(twas_blood.df$ID, levels = unique(twas_blood.df$ID))
twas_HPA.df$ID&lt;-factor(twas_HPA.df$ID, levels = unique(twas_HPA.df$ID))
twas_HPT.df$ID&lt;-factor(twas_HPT.df$ID, levels = unique(twas_HPT.df$ID))

#####
# Heatmap for brain SNP-weight sets
#####

twas_brain.df_unique&lt;-twas_brain.df[!duplicated(twas_brain.df$ID),]
vline_1&lt;-min(which(twas_brain.df_unique$CHR == 6 &amp; twas_brain.df_unique$P0 &gt; 26e6 &amp; twas_brain.df_unique$P1 &lt; 34e6))
vline_2&lt;-max(which(twas_brain.df_unique$CHR == 6 &amp; twas_brain.df_unique$P0 &gt; 26e6 &amp; twas_brain.df_unique$P1 &lt; 34e6))

#Plot brain SNP weights
png(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS.TWAS_Z_heatmap_brain.png&quot;,width = 5100, height = 1400, units = &quot;px&quot;, res=300)

ggplot(data = twas_brain.df, aes(x = ID, y = PANEL_clean_short)) +
  theme_bw()    +   
  geom_tile(aes(fill = TWAS.Z.brain), colour = &#39;black&#39;) +
  scale_fill_gradientn(colours=c(&quot;dodgerblue2&quot;,&quot;white&quot;,&quot;red&quot;), na.value = &#39;white&#39;,name = &quot;Z-score&quot;) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1),plot.title = element_text(hjust = 0.5)) +
  geom_text(aes(label=round(TWAS.Z.brain,1)), color=&quot;black&quot;, size=3) +
  labs(title=&quot;Genes differentially expressed in brain SNP-weight sets&quot;,  x =&quot;Gene ID&quot;, y = &quot;SNP-weight sets&quot;) +
  geom_vline(xintercept = vline_1-0.5, size=1) +
  geom_vline(xintercept = vline_2+0.5, size=1)

dev.off()

# Note significant features presented a z-score &gt; 4.83 or &lt; -4.83.  

#####
# Plot for blood SNP-weight sets
#####

twas_blood.df_unique&lt;-twas_blood.df[!duplicated(twas_blood.df$ID),]
vline_1&lt;-min(which(twas_blood.df_unique$CHR == 6 &amp; twas_blood.df_unique$P0 &gt; 26e6 &amp; twas_blood.df_unique$P1 &lt; 34e6))
vline_2&lt;-max(which(twas_blood.df_unique$CHR == 6 &amp; twas_blood.df_unique$P0 &gt; 26e6 &amp; twas_blood.df_unique$P1 &lt; 34e6))

#Plot blood findings
png(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS.TWAS_Z_heatmap_blood.png&quot;,width = 2400, height = 800, units = &quot;px&quot;, res=300)

ggplot(data = twas_blood.df, aes(x = ID, y = PANEL_clean_short)) +
  theme_bw()    +   
  geom_tile(aes(fill = TWAS.Z.blood), colour = &#39;black&#39;) +
  scale_fill_gradientn(colours=c(&quot;dodgerblue2&quot;,&quot;white&quot;,&quot;red&quot;), na.value = &#39;white&#39;,name = &quot;Z-score&quot;) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1),plot.title = element_text(hjust = 0.5)) +
  geom_text(aes(label=round(TWAS.Z.blood,1)), color=&quot;black&quot;, size=3) +
  labs(title=&quot;Genes differentially expressed in blood SNP-weight sets&quot;,  x =&quot;Gene ID&quot;, y = &quot;SNP-weight sets&quot;) +
  geom_vline(xintercept = vline_1-0.5, size=1) +
  geom_vline(xintercept = vline_2+0.5, size=1)
dev.off()

# Note significant features presented a z-score &gt; 4.83 or &lt; -4.83.  

#####
# Plot for HPA axis SNP-weight sets
#####

twas_HPA.df_unique&lt;-twas_HPA.df[!duplicated(twas_HPA.df$ID),]
vline_1&lt;-min(which(twas_HPA.df_unique$CHR == 6 &amp; twas_HPA.df_unique$P0 &gt; 26e6 &amp; twas_HPA.df_unique$P1 &lt; 34e6))
vline_2&lt;-max(which(twas_HPA.df_unique$CHR == 6 &amp; twas_HPA.df_unique$P0 &gt; 26e6 &amp; twas_HPA.df_unique$P1 &lt; 34e6))

#Plot HPA axis findings
png(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS.TWAS_Z_heatmap_HPA.png&quot;,width = 2400, height = 800, units = &quot;px&quot;, res=300)

ggplot(data = twas_HPA.df, aes(x = ID, y = PANEL_clean_short)) +
  theme_bw()    +   
  geom_tile(aes(fill = TWAS.Z.HPA), colour = &#39;black&#39;) +
  scale_fill_gradientn(colours=c(&quot;dodgerblue2&quot;,&quot;white&quot;,&quot;red&quot;), na.value = &#39;white&#39;,name = &quot;Z-score&quot;) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1),plot.title = element_text(hjust = 0.5)) +
  geom_text(aes(label=round(TWAS.Z.HPA,1)), color=&quot;black&quot;, size=3) +
  labs(title=&quot;Genes differentially expressed in HPA axis SNP-weight sets&quot;,  x =&quot;Gene ID&quot;, y = &quot;SNP-weight sets&quot;) +
  geom_vline(xintercept = vline_1-0.5, size=1) +
  geom_vline(xintercept = vline_2+0.5, size=1)
dev.off()

# Note significant features presented a z-score &gt; 4.83 or &lt; -4.83.  

#####
# Plot for HPT axis SNP-weight sets
#####

twas_HPT.df_unique&lt;-twas_HPT.df[!duplicated(twas_HPT.df$ID),]
vline_1&lt;-min(which(twas_HPT.df_unique$CHR == 6 &amp; twas_HPT.df_unique$P0 &gt; 26e6 &amp; twas_HPT.df_unique$P1 &lt; 34e6))
vline_2&lt;-max(which(twas_HPT.df_unique$CHR == 6 &amp; twas_HPT.df_unique$P0 &gt; 26e6 &amp; twas_HPT.df_unique$P1 &lt; 34e6))

#Plot HPT axis findings
png(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS.TWAS_Z_heatmap_HPT.png&quot;,width = 3000, height = 800, units = &quot;px&quot;, res=300)
ggplot(data = twas_HPT.df, aes(x = ID, y = PANEL_clean_short)) +
  theme_bw()    +   
  geom_tile(aes(fill = TWAS.Z.HPT), colour = &#39;black&#39;) +
  scale_fill_gradientn(colours=c(&quot;dodgerblue2&quot;,&quot;white&quot;,&quot;red&quot;), na.value = &#39;white&#39;,name = &quot;Z-score&quot;) +
  theme(axis.text.x = element_text(angle = 45, hjust = 1),plot.title = element_text(hjust = 0.5)) +
  geom_text(aes(label=round(TWAS.Z.HPT,1)), color=&quot;black&quot;, size=3) +
  labs(title=&quot;Genes differentially expressed in HPT axis SNP-weight sets&quot;,  x =&quot;Gene ID&quot;, y = &quot;SNP-weight sets&quot;) +
  geom_vline(xintercept = vline_1-0.5, size=1) +
  geom_vline(xintercept = vline_2+0.5, size=1)
dev.off()

# Note significant features presented a z-score &gt; 4.83 or &lt; -4.83.  </code></pre>
</details>
<details>
<p><summary>Show plots</summary></p>
<center>
<div class="figure">
<img src="Images/MDD_TWAS.TWAS_Z_heatmap.shared.png" alt="MDD TWAS Heatmap Shared" />
<p class="caption">MDD TWAS Heatmap Shared</p>
</div>
<hr />
<div class="figure">
<img src="Images/MDD_TWAS.TWAS_Z_heatmap.unique.png" alt="MDD TWAS Heatmap Unique" />
<p class="caption">MDD TWAS Heatmap Unique</p>
</div>
<hr />
<p><img src="Images/MDD_TWAS.TWAS_Z_heatmap_brain.png" alt="MDD TWAS Heatmap Brain" /> ***</p>
<div class="figure">
<img src="Images/MDD_TWAS.TWAS_Z_heatmap_blood.png" alt="MDD TWAS Heatmap Blood" />
<p class="caption">MDD TWAS Heatmap Blood</p>
</div>
<hr />
<div class="figure">
<img src="Images/MDD_TWAS.TWAS_Z_heatmap_HPA.png" alt="MDD TWAS Heatmap HPA" />
<p class="caption">MDD TWAS Heatmap HPA</p>
</div>
<hr />
<div class="figure">
<img src="Images/MDD_TWAS.TWAS_Z_heatmap_HPT.png" alt="MDD TWAS Heatmap HPT" />
<p class="caption">MDD TWAS Heatmap HPT</p>
</div>
<hr />

</details>
<hr />
</div>
<div id="process-colocalisation-results" class="section level2">
<h2><span class="header-section-number">3.5</span> Process colocalisation results</h2>
<p>Colocalisation identified whether TWAS and GWAS associations result from the same or distinct causal SNP.</p>
<details>
<p><summary>Organise coloc results</summary></p>
<pre class="r"><code>###
# Create a table with colocalisation results for all significant features
###
rm(list=ls())
library(data.table)
twas_sign &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig_CLEAN.txt&quot;)

library(tibble)
twas_sign &lt;- as_data_frame(twas_sign)
colnames(twas_sign)

twas_sign$Location&lt;-paste0(&#39;chr&#39;,twas_sign$CHR,&#39;:&#39;,twas_sign$P0,&#39;-&#39;,twas_sign$P1)   

#transform variables into numeric and then order by them
str(twas_sign)
as.numeric(as.character(twas_sign$CHR))
as.numeric(as.character(twas_sign$P0))
twas_sign_ordered &lt;- twas_sign[order(twas_sign$CHR, twas_sign$P0), ]

col_order &lt;- c(&quot;Location&quot;, &quot;ID&quot;, &quot;PANEL_clean_short&quot;, &quot;TWAS.Z&quot;, &quot;TWAS.P&quot;, &quot;COLOC.PP0&quot;, &quot;COLOC.PP1&quot;, &quot;COLOC.PP2&quot;, &quot;COLOC.PP3&quot;, &quot;COLOC.PP4&quot;)
twas_sign_ordered &lt;- twas_sign_ordered[, col_order]
twas_sign_ordered

###
#Create a couple of additional columns specifying whether the feature is colocalised or not
###

#to specify coloc pp4 &gt; 0.8 (see gusev et al (2019) Nat Genet on epithelial ovarian cancer)
twas_sign_ordered$High_PP4_0.8 &lt;- NULL
twas_sign_ordered$High_PP4_0.8 &lt;- ifelse(twas_sign_ordered$COLOC.PP4 &gt; 0.8, &quot;Yes&quot;, &quot;No&quot;)
sum(twas_sign_ordered$High_PP4_0.8 == &quot;Yes&quot;)  #97 features present a PP4 greater than 0.8

#to specify coloc pp3 &lt; 0.2 
twas_sign_ordered$Low_PP3_0.2 &lt;- NULL
twas_sign_ordered$Low_PP3_0.2 &lt;- ifelse(twas_sign_ordered$COLOC.PP3 &lt; 0.2, &quot;Yes&quot;, &quot;No&quot;)
sum(twas_sign_ordered$Low_PP3_0.2 == &quot;Yes&quot;) #140 features present a PP3 smaller than 0.2

#specify whether both conditions are satisfied (NB PP4 &gt; 0.8 is much more of a stringent threshold)
twas_sign_ordered$Colocalised &lt;- NULL
twas_sign_ordered$Colocalised &lt;- ifelse(twas_sign_ordered$High_PP4_0.8 == &quot;Yes&quot; &amp; twas_sign_ordered$Low_PP3_0.2 == &quot;Yes&quot;, &quot;Yes&quot;, &quot;No&quot;)
sum(twas_sign_ordered$Colocalised == &quot;Yes&quot;) #97 features are colocalised

#get the number of unique genes which were colocalised
colocalised_df &lt;- twas_sign_ordered[twas_sign_ordered$Colocalised == &quot;Yes&quot;, ] #as expected, dim = 97, 13
colocalised_vector &lt;- colocalised_df$ID 
unique_genes_colocalised &lt;- unique(colocalised_vector) #57 unique genes which were colocalised


###
# Clean and Save 
###
col_order2 &lt;- c(&quot;Location&quot;, &quot;ID&quot;, &quot;PANEL_clean_short&quot;, &quot;TWAS.Z&quot;, &quot;TWAS.P&quot;, &quot;COLOC.PP0&quot;, &quot;COLOC.PP1&quot;, &quot;COLOC.PP2&quot;, &quot;COLOC.PP3&quot;, &quot;COLOC.PP4&quot;, &quot;Low_PP3_0.2&quot;, &quot;High_PP4_0.8&quot;, &quot;Colocalised&quot;)
twas_sign_ordered &lt;- twas_sign_ordered[, col_order2]
twas_sign_ordered

write.csv(twas_sign_ordered, &quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_colocalisation.csv&quot;, row.names = F)

q()
n</code></pre>
</details>
<details>
<p><summary>Show colocalisation table</summary></p>
<table>
<caption>MDD TWAS Colocalisation Results</caption>
<thead>
<tr class="header">
<th align="left">Location</th>
<th align="left">ID</th>
<th align="left">PANEL_clean_short</th>
<th align="right">TWAS.Z</th>
<th align="right">TWAS.P</th>
<th align="right">COLOC.PP0</th>
<th align="right">COLOC.PP1</th>
<th align="right">COLOC.PP2</th>
<th align="right">COLOC.PP3</th>
<th align="right">COLOC.PP4</th>
<th align="left">Low_PP3_0.2</th>
<th align="left">High_PP4_0.8</th>
<th align="left">Colocalised</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">chr1:8412457-8877702</td>
<td align="left">RERE</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-5.095707</td>
<td align="right">3.47e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.006</td>
<td align="right">0.993</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:8412457-8877702</td>
<td align="left">RERE</td>
<td align="left">YFS Blood</td>
<td align="right">-5.310078</td>
<td align="right">1.10e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.006</td>
<td align="right">0.993</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr1:8484705-8494898</td>
<td align="left">RP5-1115A15.1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.175240</td>
<td align="right">2.28e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.004</td>
<td align="right">0.995</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:8484705-8494898</td>
<td align="left">RP5-1115A15.1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-4.866386</td>
<td align="right">1.14e-06</td>
<td align="right">0.012</td>
<td align="right">0.001</td>
<td align="right">0.088</td>
<td align="right">0.008</td>
<td align="right">0.891</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr1:36884051-36884179</td>
<td align="left">SNORA63</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">4.848870</td>
<td align="right">1.24e-06</td>
<td align="right">0.067</td>
<td align="right">0.006</td>
<td align="right">0.257</td>
<td align="right">0.023</td>
<td align="right">0.647</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr1:71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Caudate</td>
<td align="right">5.780100</td>
<td align="right">7.47e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.284</td>
<td align="right">0.036</td>
<td align="right">0.681</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr1:71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Putamen</td>
<td align="right">5.548510</td>
<td align="right">2.88e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.018</td>
<td align="right">0.014</td>
<td align="right">0.968</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">8.760622</td>
<td align="right">1.94e-18</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.007</td>
<td align="right">0.993</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr1:72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-7.785520</td>
<td align="right">6.94e-15</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.010</td>
<td align="right">0.990</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-7.708820</td>
<td align="right">1.27e-14</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.006</td>
<td align="right">0.994</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr1:72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">PsychENCODE</td>
<td align="right">-7.742756</td>
<td align="right">9.73e-15</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.007</td>
<td align="right">0.993</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:175873898-175889649</td>
<td align="left">RP11-318C24.2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.027510</td>
<td align="right">4.97e-07</td>
<td align="right">0.004</td>
<td align="right">0.007</td>
<td align="right">0.012</td>
<td align="right">0.018</td>
<td align="right">0.959</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr1:175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-4.958690</td>
<td align="right">7.10e-07</td>
<td align="right">0.000</td>
<td align="right">0.008</td>
<td align="right">0.000</td>
<td align="right">0.021</td>
<td align="right">0.971</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.039850</td>
<td align="right">4.66e-07</td>
<td align="right">0.000</td>
<td align="right">0.007</td>
<td align="right">0.000</td>
<td align="right">0.020</td>
<td align="right">0.973</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr1:175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.005960</td>
<td align="right">5.56e-07</td>
<td align="right">0.000</td>
<td align="right">0.007</td>
<td align="right">0.000</td>
<td align="right">0.019</td>
<td align="right">0.973</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:181452685-181775921</td>
<td align="left">CACNA1E</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-4.989390</td>
<td align="right">6.06e-07</td>
<td align="right">0.000</td>
<td align="right">0.151</td>
<td align="right">0.000</td>
<td align="right">0.420</td>
<td align="right">0.429</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr1:197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC</td>
<td align="right">4.848374</td>
<td align="right">1.24e-06</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.011</td>
<td align="right">0.988</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.421950</td>
<td align="right">5.90e-08</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.010</td>
<td align="right">0.989</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr1:197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.018050</td>
<td align="right">5.22e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.010</td>
<td align="right">0.989</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr2:58386377-58468515</td>
<td align="left">FANCL</td>
<td align="left">CMC DLPFC</td>
<td align="right">-5.183180</td>
<td align="right">2.18e-07</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.056</td>
<td align="right">0.027</td>
<td align="right">0.916</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr2:58386377-58468515</td>
<td align="left">FANCL</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.897476</td>
<td align="right">9.71e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.007</td>
<td align="right">0.104</td>
<td align="right">0.888</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr2:197831741-198175897</td>
<td align="left">ANKRD44</td>
<td align="left">YFS Blood</td>
<td align="right">-5.690140</td>
<td align="right">1.27e-08</td>
<td align="right">0.062</td>
<td align="right">0.023</td>
<td align="right">0.168</td>
<td align="right">0.061</td>
<td align="right">0.686</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr2:198254508-198299815</td>
<td align="left">SF3B1</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">5.214900</td>
<td align="right">1.84e-07</td>
<td align="right">0.071</td>
<td align="right">0.015</td>
<td align="right">0.319</td>
<td align="right">0.068</td>
<td align="right">0.526</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr3:44481261-44561226</td>
<td align="left">ZNF445</td>
<td align="left">CMC DLPFC</td>
<td align="right">-5.103280</td>
<td align="right">3.34e-07</td>
<td align="right">0.000</td>
<td align="right">0.087</td>
<td align="right">0.002</td>
<td align="right">0.601</td>
<td align="right">0.310</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr4:41937137-41962589</td>
<td align="left">TMEM33</td>
<td align="left">PsychENCODE</td>
<td align="right">4.837418</td>
<td align="right">1.32e-06</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.074</td>
<td align="right">0.925</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr4:41983713-41988476</td>
<td align="left">DCAF4L1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.128000</td>
<td align="right">2.93e-07</td>
<td align="right">0.003</td>
<td align="right">0.001</td>
<td align="right">0.291</td>
<td align="right">0.091</td>
<td align="right">0.615</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr4:41990758-41991254</td>
<td align="left">RP11-814H16.2</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">5.009600</td>
<td align="right">5.45e-07</td>
<td align="right">0.005</td>
<td align="right">0.000</td>
<td align="right">0.561</td>
<td align="right">0.055</td>
<td align="right">0.378</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Amygdala</td>
<td align="right">-5.253400</td>
<td align="right">1.49e-07</td>
<td align="right">0.004</td>
<td align="right">0.001</td>
<td align="right">0.476</td>
<td align="right">0.070</td>
<td align="right">0.450</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx ACC</td>
<td align="right">-5.001690</td>
<td align="right">5.68e-07</td>
<td align="right">0.003</td>
<td align="right">0.001</td>
<td align="right">0.399</td>
<td align="right">0.101</td>
<td align="right">0.496</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Caudate</td>
<td align="right">-4.854800</td>
<td align="right">1.21e-06</td>
<td align="right">0.003</td>
<td align="right">0.001</td>
<td align="right">0.388</td>
<td align="right">0.086</td>
<td align="right">0.521</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Cortex</td>
<td align="right">-5.774530</td>
<td align="right">7.72e-09</td>
<td align="right">0.003</td>
<td align="right">0.001</td>
<td align="right">0.340</td>
<td align="right">0.113</td>
<td align="right">0.543</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">-5.085140</td>
<td align="right">3.67e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.016</td>
<td align="right">0.039</td>
<td align="right">0.944</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">-5.602700</td>
<td align="right">2.11e-08</td>
<td align="right">0.001</td>
<td align="right">0.001</td>
<td align="right">0.072</td>
<td align="right">0.119</td>
<td align="right">0.808</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr4:41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.259200</td>
<td align="right">1.45e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.026</td>
<td align="right">0.974</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564712-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">PsychENCODE</td>
<td align="right">6.091010</td>
<td align="right">1.12e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.117</td>
<td align="right">0.883</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.360090</td>
<td align="right">8.32e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.086</td>
<td align="right">0.914</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Amygdala</td>
<td align="right">6.118500</td>
<td align="right">9.45e-10</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.004</td>
<td align="right">0.058</td>
<td align="right">0.938</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx ACC</td>
<td align="right">6.445500</td>
<td align="right">1.15e-10</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.056</td>
<td align="right">0.944</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Caudate</td>
<td align="right">6.282167</td>
<td align="right">3.34e-10</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.062</td>
<td align="right">0.938</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">6.011700</td>
<td align="right">1.84e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.062</td>
<td align="right">0.938</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">6.053050</td>
<td align="right">1.42e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.052</td>
<td align="right">0.948</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cortex</td>
<td align="right">6.021420</td>
<td align="right">1.73e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.070</td>
<td align="right">0.930</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">6.720000</td>
<td align="right">1.82e-11</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.086</td>
<td align="right">0.914</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">5.875800</td>
<td align="right">4.21e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.060</td>
<td align="right">0.940</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">6.010490</td>
<td align="right">1.85e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.059</td>
<td align="right">0.941</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Putamen</td>
<td align="right">6.372050</td>
<td align="right">1.87e-10</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.054</td>
<td align="right">0.946</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Substantia nigra</td>
<td align="right">6.057270</td>
<td align="right">1.38e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.009</td>
<td align="right">0.054</td>
<td align="right">0.937</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Pituitary</td>
<td align="right">6.048500</td>
<td align="right">1.46e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.050</td>
<td align="right">0.950</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">5.889760</td>
<td align="right">3.87e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.079</td>
<td align="right">0.920</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.526440</td>
<td align="right">3.27e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.048</td>
<td align="right">0.952</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87729709-87794514</td>
<td align="left">CTC-498M16.4</td>
<td align="left">GTEx Substantia nigra</td>
<td align="right">5.403610</td>
<td align="right">6.53e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.391</td>
<td align="right">0.037</td>
<td align="right">0.572</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx ACC</td>
<td align="right">-5.813700</td>
<td align="right">6.11e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.631</td>
<td align="right">0.053</td>
<td align="right">0.317</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-5.861000</td>
<td align="right">4.60e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.056</td>
<td align="right">0.051</td>
<td align="right">0.893</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Cortex</td>
<td align="right">-6.510990</td>
<td align="right">7.47e-11</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.091</td>
<td align="right">0.040</td>
<td align="right">0.870</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">-7.091600</td>
<td align="right">1.33e-12</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.114</td>
<td align="right">0.035</td>
<td align="right">0.850</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">PsychENCODE</td>
<td align="right">-6.097890</td>
<td align="right">1.07e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.035</td>
<td align="right">0.251</td>
<td align="right">0.715</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr5:140024947-140027370</td>
<td align="left">NDUFA2</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.190020</td>
<td align="right">2.10e-07</td>
<td align="right">0.104</td>
<td align="right">0.037</td>
<td align="right">0.170</td>
<td align="right">0.060</td>
<td align="right">0.629</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr5:140201222-140203811</td>
<td align="left">PCDHA5</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.402970</td>
<td align="right">6.55e-08</td>
<td align="right">0.055</td>
<td align="right">0.028</td>
<td align="right">0.090</td>
<td align="right">0.046</td>
<td align="right">0.781</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr5:140220907-140223351</td>
<td align="left">PCDHA8</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-4.980100</td>
<td align="right">6.36e-07</td>
<td align="right">0.001</td>
<td align="right">0.032</td>
<td align="right">0.001</td>
<td align="right">0.051</td>
<td align="right">0.915</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:26188921-26189323</td>
<td align="left">HIST1H4D</td>
<td align="left">NTR Blood</td>
<td align="right">-4.987600</td>
<td align="right">6.11e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.309</td>
<td align="right">0.059</td>
<td align="right">0.632</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:26365386-26378540</td>
<td align="left">BTN3A2</td>
<td align="left">NTR Blood</td>
<td align="right">5.326600</td>
<td align="right">1.00e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.562</td>
<td align="right">0.438</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">5.188200</td>
<td align="right">2.12e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.505</td>
<td align="right">0.495</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">4.963000</td>
<td align="right">6.96e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.506</td>
<td align="right">0.494</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Pituitary</td>
<td align="right">5.898930</td>
<td align="right">3.66e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.501</td>
<td align="right">0.499</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">5.481600</td>
<td align="right">4.22e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.507</td>
<td align="right">0.492</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.086960</td>
<td align="right">3.64e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.543</td>
<td align="right">0.457</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:26538633-26546482</td>
<td align="left">HMGN4</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.395400</td>
<td align="right">6.84e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.404</td>
<td align="right">0.299</td>
<td align="right">0.297</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-4.891200</td>
<td align="right">1.00e-06</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.096</td>
<td align="right">0.322</td>
<td align="right">0.582</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-4.947900</td>
<td align="right">7.50e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.962</td>
<td align="right">0.037</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">-5.045000</td>
<td align="right">4.54e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.366</td>
<td align="right">0.056</td>
<td align="right">0.577</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.916080</td>
<td align="right">3.30e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.044</td>
<td align="right">0.035</td>
<td align="right">0.921</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-5.335920</td>
<td align="right">9.51e-08</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.047</td>
<td align="right">0.474</td>
<td align="right">0.479</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:27325604-27339304</td>
<td align="left">ZNF204P</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">-5.032700</td>
<td align="right">4.84e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.547</td>
<td align="right">0.211</td>
<td align="right">0.242</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:27371789-27374743</td>
<td align="left">RP1-153G14.4</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">5.354000</td>
<td align="right">8.60e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.676</td>
<td align="right">0.142</td>
<td align="right">0.182</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:27418522-27440897</td>
<td align="left">ZNF184</td>
<td align="left">GTEx Caudate</td>
<td align="right">-6.325200</td>
<td align="right">2.53e-10</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.231</td>
<td align="right">0.114</td>
<td align="right">0.655</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:27418522-27440897</td>
<td align="left">ZNF184</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">-4.952200</td>
<td align="right">7.34e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.174</td>
<td align="right">0.185</td>
<td align="right">0.641</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:27840926-27841289</td>
<td align="left">HIST1H4L</td>
<td align="left">NTR Blood</td>
<td align="right">4.870800</td>
<td align="right">1.11e-06</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.162</td>
<td align="right">0.838</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:28058932-28061442</td>
<td align="left">ZSCAN12P1</td>
<td align="left">PsychENCODE</td>
<td align="right">6.268010</td>
<td align="right">3.66e-10</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.056</td>
<td align="right">0.626</td>
<td align="right">0.318</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:28058932-28061442</td>
<td align="left">ZSCAN12P1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-4.936930</td>
<td align="right">7.94e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.388</td>
<td align="right">0.365</td>
<td align="right">0.247</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:28083406-28084329</td>
<td align="left">RP1-265C24.5</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">5.532000</td>
<td align="right">3.16e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.009</td>
<td align="right">0.143</td>
<td align="right">0.847</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:28092338-28097860</td>
<td align="left">ZSCAN16</td>
<td align="left">YFS Blood</td>
<td align="right">-6.109000</td>
<td align="right">1.00e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.013</td>
<td align="right">0.054</td>
<td align="right">0.933</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-5.307800</td>
<td align="right">1.11e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.238</td>
<td align="right">0.147</td>
<td align="right">0.615</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">-6.017000</td>
<td align="right">1.77e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.080</td>
<td align="right">0.053</td>
<td align="right">0.866</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-6.159020</td>
<td align="right">7.32e-10</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.169</td>
<td align="right">0.191</td>
<td align="right">0.640</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:28227098-28228736</td>
<td align="left">NKAPL</td>
<td align="left">PsychENCODE</td>
<td align="right">5.002860</td>
<td align="right">5.65e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.627</td>
<td align="right">0.332</td>
<td align="right">0.041</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.094600</td>
<td align="right">3.49e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.031</td>
<td align="right">0.146</td>
<td align="right">0.822</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.062800</td>
<td align="right">4.13e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.756</td>
<td align="right">0.244</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">5.198000</td>
<td align="right">2.01e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.172</td>
<td align="right">0.174</td>
<td align="right">0.654</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Putamen</td>
<td align="right">5.739000</td>
<td align="right">9.52e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.018</td>
<td align="right">0.051</td>
<td align="right">0.931</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Thyroid</td>
<td align="right">5.338400</td>
<td align="right">9.38e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.858</td>
<td align="right">0.141</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.662330</td>
<td align="right">1.49e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.288</td>
<td align="right">0.107</td>
<td align="right">0.604</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:28249314-28270326</td>
<td align="left">PGBD1</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-6.313100</td>
<td align="right">2.74e-10</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.032</td>
<td align="right">0.017</td>
<td align="right">0.950</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:28292470-28324048</td>
<td align="left">ZSCAN31</td>
<td align="left">GTEx Amygdala</td>
<td align="right">-5.084150</td>
<td align="right">3.69e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.381</td>
<td align="right">0.404</td>
<td align="right">0.214</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Amygdala</td>
<td align="right">4.949900</td>
<td align="right">7.43e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.777</td>
<td align="right">0.111</td>
<td align="right">0.111</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Hippocampus</td>
<td align="right">4.951000</td>
<td align="right">7.37e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.389</td>
<td align="right">0.273</td>
<td align="right">0.338</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Thyroid</td>
<td align="right">6.093300</td>
<td align="right">1.11e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.086</td>
<td align="right">0.914</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Hypothalamus</td>
<td align="right">-5.777500</td>
<td align="right">7.58e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.062</td>
<td align="right">0.179</td>
<td align="right">0.758</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Putamen</td>
<td align="right">-4.894000</td>
<td align="right">9.90e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.239</td>
<td align="right">0.226</td>
<td align="right">0.535</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-4.953290</td>
<td align="right">7.30e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.002</td>
<td align="right">0.275</td>
<td align="right">0.723</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:30644166-30655672</td>
<td align="left">PPP1R18</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">4.910200</td>
<td align="right">9.10e-07</td>
<td align="right">0.106</td>
<td align="right">0.007</td>
<td align="right">0.130</td>
<td align="right">0.007</td>
<td align="right">0.750</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.299700</td>
<td align="right">1.16e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.999</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.067100</td>
<td align="right">4.04e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.999</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.936600</td>
<td align="right">7.95e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.999</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-5.299000</td>
<td align="right">1.16e-07</td>
<td align="right">0.010</td>
<td align="right">0.001</td>
<td align="right">0.012</td>
<td align="right">0.000</td>
<td align="right">0.976</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.253270</td>
<td align="right">1.49e-07</td>
<td align="right">0.016</td>
<td align="right">0.001</td>
<td align="right">0.020</td>
<td align="right">0.000</td>
<td align="right">0.963</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.557400</td>
<td align="right">2.74e-08</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.999</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:30881982-30894236</td>
<td align="left">VARS2</td>
<td align="left">GTEx Cortex</td>
<td align="right">5.922000</td>
<td align="right">3.18e-09</td>
<td align="right">0.105</td>
<td align="right">0.004</td>
<td align="right">0.158</td>
<td align="right">0.005</td>
<td align="right">0.727</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:30881982-30894236</td>
<td align="left">VARS2</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">6.323130</td>
<td align="right">2.56e-10</td>
<td align="right">0.005</td>
<td align="right">0.001</td>
<td align="right">0.007</td>
<td align="right">0.000</td>
<td align="right">0.986</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:31255287-31256741</td>
<td align="left">WASF5P</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.156240</td>
<td align="right">2.52e-07</td>
<td align="right">0.000</td>
<td align="right">0.046</td>
<td align="right">0.000</td>
<td align="right">0.091</td>
<td align="right">0.862</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:31368479-31445283</td>
<td align="left">HCP5</td>
<td align="left">GTEx Thyroid</td>
<td align="right">6.400800</td>
<td align="right">1.55e-10</td>
<td align="right">0.000</td>
<td align="right">0.008</td>
<td align="right">0.000</td>
<td align="right">0.015</td>
<td align="right">0.976</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:31462658-31478901</td>
<td align="left">MICB</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.557000</td>
<td align="right">2.74e-08</td>
<td align="right">0.000</td>
<td align="right">0.047</td>
<td align="right">0.000</td>
<td align="right">0.094</td>
<td align="right">0.859</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:31606805-31620482</td>
<td align="left">BAG6</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.580000</td>
<td align="right">2.40e-08</td>
<td align="right">0.008</td>
<td align="right">0.408</td>
<td align="right">0.006</td>
<td align="right">0.319</td>
<td align="right">0.259</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:31694815-31698357</td>
<td align="left">DDAH2</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">5.409500</td>
<td align="right">6.32e-08</td>
<td align="right">0.331</td>
<td align="right">0.042</td>
<td align="right">0.258</td>
<td align="right">0.033</td>
<td align="right">0.336</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:31694816-31698039</td>
<td align="left">DDAH2</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.344500</td>
<td align="right">9.07e-08</td>
<td align="right">0.000</td>
<td align="right">0.051</td>
<td align="right">0.000</td>
<td align="right">0.039</td>
<td align="right">0.909</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:99817347-99842082</td>
<td align="left">COQ3</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.146560</td>
<td align="right">2.65e-07</td>
<td align="right">0.324</td>
<td align="right">0.015</td>
<td align="right">0.598</td>
<td align="right">0.029</td>
<td align="right">0.034</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:105404922-105531207</td>
<td align="left">LIN28B</td>
<td align="left">CMC DLPFC</td>
<td align="right">-5.232050</td>
<td align="right">1.68e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.008</td>
<td align="right">0.990</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr6:105404923-105531207</td>
<td align="left">LIN28B</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.105689</td>
<td align="right">3.30e-07</td>
<td align="right">0.000</td>
<td align="right">0.005</td>
<td align="right">0.000</td>
<td align="right">0.052</td>
<td align="right">0.943</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr6:105584224-105617820</td>
<td align="left">BVES-AS1</td>
<td align="left">GTEx Amygdala</td>
<td align="right">-5.578300</td>
<td align="right">2.43e-08</td>
<td align="right">0.061</td>
<td align="right">0.007</td>
<td align="right">0.373</td>
<td align="right">0.045</td>
<td align="right">0.514</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.505026</td>
<td align="right">3.69e-08</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.003</td>
<td align="right">0.009</td>
<td align="right">0.987</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.790690</td>
<td align="right">7.01e-09</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.054</td>
<td align="right">0.945</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.531000</td>
<td align="right">3.18e-08</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.008</td>
<td align="right">0.991</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12276886</td>
<td align="left">TMEM106B</td>
<td align="left">YFS Blood</td>
<td align="right">5.373600</td>
<td align="right">7.72e-08</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.007</td>
<td align="right">0.993</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr7:24836158-25021253</td>
<td align="left">OSBPL3</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.622890</td>
<td align="right">1.88e-08</td>
<td align="right">0.090</td>
<td align="right">0.040</td>
<td align="right">0.062</td>
<td align="right">0.027</td>
<td align="right">0.780</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr8:52232136-52722005</td>
<td align="left">PXDNL</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.887460</td>
<td align="right">3.92e-09</td>
<td align="right">0.090</td>
<td align="right">0.019</td>
<td align="right">0.318</td>
<td align="right">0.065</td>
<td align="right">0.508</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr8:61297147-61429354</td>
<td align="left">RP11-163N6.2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.336530</td>
<td align="right">9.47e-08</td>
<td align="right">0.084</td>
<td align="right">0.162</td>
<td align="right">0.118</td>
<td align="right">0.228</td>
<td align="right">0.408</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr9:126605315-126605965</td>
<td align="left">PIGFP2</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.305600</td>
<td align="right">1.12e-07</td>
<td align="right">0.017</td>
<td align="right">0.004</td>
<td align="right">0.603</td>
<td align="right">0.126</td>
<td align="right">0.250</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr11:57067112-57092426</td>
<td align="left">TNKS1BP1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">4.922610</td>
<td align="right">8.54e-07</td>
<td align="right">0.080</td>
<td align="right">0.025</td>
<td align="right">0.107</td>
<td align="right">0.032</td>
<td align="right">0.756</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr11:57405497-57420263</td>
<td align="left">AP000662.4</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-4.980256</td>
<td align="right">6.35e-07</td>
<td align="right">0.000</td>
<td align="right">0.136</td>
<td align="right">0.000</td>
<td align="right">0.255</td>
<td align="right">0.610</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr11:57424488-57429340</td>
<td align="left">CLP1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.195860</td>
<td align="right">2.04e-07</td>
<td align="right">0.001</td>
<td align="right">0.008</td>
<td align="right">0.002</td>
<td align="right">0.015</td>
<td align="right">0.974</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr11:61535973-61560274</td>
<td align="left">TMEM258</td>
<td align="left">PsychENCODE</td>
<td align="right">5.021730</td>
<td align="right">5.12e-07</td>
<td align="right">0.000</td>
<td align="right">0.049</td>
<td align="right">0.000</td>
<td align="right">0.041</td>
<td align="right">0.910</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr11:113280318-113346111</td>
<td align="left">DRD2</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">-5.073787</td>
<td align="right">3.90e-07</td>
<td align="right">0.366</td>
<td align="right">0.032</td>
<td align="right">0.515</td>
<td align="right">0.045</td>
<td align="right">0.042</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr13:53602875-53626196</td>
<td align="left">OLFM4</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.091290</td>
<td align="right">3.56e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.865</td>
<td align="right">0.089</td>
<td align="right">0.046</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr14:42057064-42074059</td>
<td align="left">CTD-2298J14.2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.678860</td>
<td align="right">1.36e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.022</td>
<td align="right">0.978</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:42076773-42373752</td>
<td align="left">LRFN5</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">5.423400</td>
<td align="right">5.85e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.029</td>
<td align="right">0.971</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:42076773-42373752</td>
<td align="left">LRFN5</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.597540</td>
<td align="right">2.17e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.041</td>
<td align="right">0.959</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:59951161-59971429</td>
<td align="left">JKAMP</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.125100</td>
<td align="right">2.97e-07</td>
<td align="right">0.001</td>
<td align="right">0.004</td>
<td align="right">0.004</td>
<td align="right">0.022</td>
<td align="right">0.969</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:59971779-60043549</td>
<td align="left">CCDC175</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.478850</td>
<td align="right">4.28e-08</td>
<td align="right">0.000</td>
<td align="right">0.004</td>
<td align="right">0.000</td>
<td align="right">0.018</td>
<td align="right">0.979</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:60062693-60337557</td>
<td align="left">RTN1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-4.874920</td>
<td align="right">1.09e-06</td>
<td align="right">0.001</td>
<td align="right">0.006</td>
<td align="right">0.007</td>
<td align="right">0.033</td>
<td align="right">0.953</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:60062695-60337684</td>
<td align="left">RTN1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.348450</td>
<td align="right">8.87e-08</td>
<td align="right">0.000</td>
<td align="right">0.003</td>
<td align="right">0.000</td>
<td align="right">0.016</td>
<td align="right">0.981</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:64319682-64693151</td>
<td align="left">SYNE2</td>
<td align="left">NTR Blood</td>
<td align="right">5.609528</td>
<td align="right">2.03e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.016</td>
<td align="right">0.984</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:64550950-64770377</td>
<td align="left">ESR2</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.982300</td>
<td align="right">2.20e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.113</td>
<td align="right">0.026</td>
<td align="right">0.860</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:64550950-64770377</td>
<td align="left">ESR2</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-5.655371</td>
<td align="right">1.56e-08</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.014</td>
<td align="right">0.986</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:75120140-75179818</td>
<td align="left">AREL1</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.015110</td>
<td align="right">5.30e-07</td>
<td align="right">0.000</td>
<td align="right">0.002</td>
<td align="right">0.000</td>
<td align="right">0.216</td>
<td align="right">0.782</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:75319736-75330537</td>
<td align="left">PROX2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.758100</td>
<td align="right">8.51e-09</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.017</td>
<td align="right">0.020</td>
<td align="right">0.962</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:75348593-75370450</td>
<td align="left">DLST</td>
<td align="left">CMC DLPFC</td>
<td align="right">4.981400</td>
<td align="right">6.31e-07</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.047</td>
<td align="right">0.952</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:75348594-75370448</td>
<td align="left">DLST</td>
<td align="left">PsychENCODE</td>
<td align="right">5.089700</td>
<td align="right">3.59e-07</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.023</td>
<td align="right">0.977</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:75370656-75389188</td>
<td align="left">RPS6KL1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.023810</td>
<td align="right">5.07e-07</td>
<td align="right">0.003</td>
<td align="right">0.001</td>
<td align="right">0.205</td>
<td align="right">0.082</td>
<td align="right">0.708</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:75370657-75390099</td>
<td align="left">RPS6KL1</td>
<td align="left">PsychENCODE</td>
<td align="right">-4.952550</td>
<td align="right">7.32e-07</td>
<td align="right">0.002</td>
<td align="right">0.000</td>
<td align="right">0.176</td>
<td align="right">0.031</td>
<td align="right">0.791</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr14:103878456-103879098</td>
<td align="left">RP11-600F24.2</td>
<td align="left">PsychENCODE</td>
<td align="right">5.185660</td>
<td align="right">2.15e-07</td>
<td align="right">0.007</td>
<td align="right">0.002</td>
<td align="right">0.552</td>
<td align="right">0.202</td>
<td align="right">0.238</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:103985996-103989448</td>
<td align="left">CKB</td>
<td align="left">YFS Blood</td>
<td align="right">5.346000</td>
<td align="right">8.99e-08</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.005</td>
<td align="right">0.995</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:103995508-104003410</td>
<td align="left">TRMT61A</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.051300</td>
<td align="right">4.39e-07</td>
<td align="right">0.001</td>
<td align="right">0.004</td>
<td align="right">0.004</td>
<td align="right">0.030</td>
<td align="right">0.961</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:103995521-104003410</td>
<td align="left">TRMT61A</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">4.977593</td>
<td align="right">6.44e-07</td>
<td align="right">0.006</td>
<td align="right">0.010</td>
<td align="right">0.049</td>
<td align="right">0.081</td>
<td align="right">0.854</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:104019758-104028214</td>
<td align="left">RP11-894P9.2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.462560</td>
<td align="right">4.69e-08</td>
<td align="right">0.000</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.005</td>
<td align="right">0.994</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:104153913-104154464</td>
<td align="left">RP11-73M18.6</td>
<td align="left">PsychENCODE</td>
<td align="right">5.031320</td>
<td align="right">4.87e-07</td>
<td align="right">0.000</td>
<td align="right">0.005</td>
<td align="right">0.001</td>
<td align="right">0.413</td>
<td align="right">0.581</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr14:104160897-104161507</td>
<td align="left">RP11-73M18.7</td>
<td align="left">PsychENCODE</td>
<td align="right">4.856130</td>
<td align="right">1.20e-06</td>
<td align="right">0.000</td>
<td align="right">0.006</td>
<td align="right">0.000</td>
<td align="right">0.513</td>
<td align="right">0.480</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:104162690-104163500</td>
<td align="left">RP11-73M18.8</td>
<td align="left">GTEx Amygdala</td>
<td align="right">5.142000</td>
<td align="right">2.72e-07</td>
<td align="right">0.010</td>
<td align="right">0.002</td>
<td align="right">0.082</td>
<td align="right">0.019</td>
<td align="right">0.887</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:104177607-104179149</td>
<td align="left">AL049840.1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.029540</td>
<td align="right">4.92e-07</td>
<td align="right">0.001</td>
<td align="right">0.003</td>
<td align="right">0.008</td>
<td align="right">0.026</td>
<td align="right">0.962</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:104177607-104179149</td>
<td align="left">AL049840.1</td>
<td align="left">GTEx Cortex</td>
<td align="right">5.143620</td>
<td align="right">2.69e-07</td>
<td align="right">0.001</td>
<td align="right">0.002</td>
<td align="right">0.007</td>
<td align="right">0.012</td>
<td align="right">0.979</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:104179904-104180441</td>
<td align="left">RP11-73M18.9</td>
<td align="left">GTEx Cortex</td>
<td align="right">4.977330</td>
<td align="right">6.45e-07</td>
<td align="right">0.000</td>
<td align="right">0.002</td>
<td align="right">0.001</td>
<td align="right">0.013</td>
<td align="right">0.984</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:104179904-104180586</td>
<td align="left">RP11-73M18.9</td>
<td align="left">PsychENCODE</td>
<td align="right">4.830100</td>
<td align="right">1.36e-06</td>
<td align="right">0.001</td>
<td align="right">0.005</td>
<td align="right">0.049</td>
<td align="right">0.425</td>
<td align="right">0.520</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr16:72146056-72210777</td>
<td align="left">PMFBP1</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.160620</td>
<td align="right">2.46e-07</td>
<td align="right">0.013</td>
<td align="right">0.005</td>
<td align="right">0.174</td>
<td align="right">0.069</td>
<td align="right">0.738</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr17:27400528-27507430</td>
<td align="left">MYO18A</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">-5.128570</td>
<td align="right">2.92e-07</td>
<td align="right">0.002</td>
<td align="right">0.040</td>
<td align="right">0.001</td>
<td align="right">0.019</td>
<td align="right">0.937</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr17:27401933-27405875</td>
<td align="left">TIAF1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">-5.361200</td>
<td align="right">8.27e-08</td>
<td align="right">0.016</td>
<td align="right">0.111</td>
<td align="right">0.008</td>
<td align="right">0.055</td>
<td align="right">0.810</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr17:65520597-65521538</td>
<td align="left">CTD-2653B5.1</td>
<td align="left">PsychENCODE</td>
<td align="right">5.105730</td>
<td align="right">3.30e-07</td>
<td align="right">0.000</td>
<td align="right">0.441</td>
<td align="right">0.000</td>
<td align="right">0.028</td>
<td align="right">0.531</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr18:52385091-52562747</td>
<td align="left">RAB27B</td>
<td align="left">PsychENCODE</td>
<td align="right">5.012900</td>
<td align="right">5.36e-07</td>
<td align="right">0.000</td>
<td align="right">0.015</td>
<td align="right">0.000</td>
<td align="right">0.328</td>
<td align="right">0.657</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr18:52495707-52562747</td>
<td align="left">RAB27B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.843190</td>
<td align="right">1.28e-06</td>
<td align="right">0.000</td>
<td align="right">0.016</td>
<td align="right">0.000</td>
<td align="right">0.038</td>
<td align="right">0.945</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr20:47835831-47860614</td>
<td align="left">DDX27</td>
<td align="left">CMC DLPFC</td>
<td align="right">4.836260</td>
<td align="right">1.32e-06</td>
<td align="right">0.003</td>
<td align="right">0.067</td>
<td align="right">0.001</td>
<td align="right">0.029</td>
<td align="right">0.900</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr22:41165634-41215403</td>
<td align="left">SLC25A17</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">5.076990</td>
<td align="right">3.83e-07</td>
<td align="right">0.007</td>
<td align="right">0.001</td>
<td align="right">0.547</td>
<td align="right">0.097</td>
<td align="right">0.348</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr22:41165634-41215403</td>
<td align="left">SLC25A17</td>
<td align="left">GTEx Thyroid</td>
<td align="right">4.896100</td>
<td align="right">9.78e-07</td>
<td align="right">0.000</td>
<td align="right">0.008</td>
<td align="right">0.020</td>
<td align="right">0.671</td>
<td align="right">0.301</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr22:41253088-41351450</td>
<td align="left">XPNPEP3</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">4.951000</td>
<td align="right">7.38e-07</td>
<td align="right">0.009</td>
<td align="right">0.001</td>
<td align="right">0.731</td>
<td align="right">0.091</td>
<td align="right">0.168</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr22:41258260-41363888</td>
<td align="left">XPNPEP3</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.110000</td>
<td align="right">3.21e-07</td>
<td align="right">0.000</td>
<td align="right">0.004</td>
<td align="right">0.005</td>
<td align="right">0.358</td>
<td align="right">0.632</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr22:41487790-41576081</td>
<td align="left">EP300</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.493900</td>
<td align="right">3.93e-08</td>
<td align="right">0.001</td>
<td align="right">0.001</td>
<td align="right">0.061</td>
<td align="right">0.049</td>
<td align="right">0.888</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr22:41487790-41576081</td>
<td align="left">EP300</td>
<td align="left">YFS Blood</td>
<td align="right">5.059100</td>
<td align="right">4.21e-07</td>
<td align="right">0.000</td>
<td align="right">0.012</td>
<td align="right">0.000</td>
<td align="right">0.955</td>
<td align="right">0.033</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr22:41641614-41682216</td>
<td align="left">RANGAP1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.240100</td>
<td align="right">1.61e-07</td>
<td align="right">0.010</td>
<td align="right">0.000</td>
<td align="right">0.814</td>
<td align="right">0.028</td>
<td align="right">0.147</td>
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr22:41641615-41682255</td>
<td align="left">RANGAP1</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.575273</td>
<td align="right">2.47e-08</td>
<td align="right">0.000</td>
<td align="right">0.004</td>
<td align="right">0.000</td>
<td align="right">0.705</td>
<td align="right">0.290</td>
<td align="left">No</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr22:41697526-41756151</td>
<td align="left">ZC3H7B</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.729100</td>
<td align="right">1.01e-08</td>
<td align="right">0.001</td>
<td align="right">0.000</td>
<td align="right">0.105</td>
<td align="right">0.031</td>
<td align="right">0.862</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
</tbody>
</table>
</details>
<p></br></p>
<p>140 of the 176 significant features presented a low posterior probability of TWAS and GWAS associations resulting from distinct causal SNPs. This is a good index of colocalisation, but PP4, should also be considered due to the possibility of other models besides PP3 and PP4 to be the most probable. When considering features with high PP4 (&gt; 0.8), 97 of the 140 features with low PP3 also presented a high probability of GWAS and TWAS associations resulting from the same causal SNP. Therefore, based on both criteria applied, we considered 97 features as colocalised.</p>
<hr />
</div>
<div id="process-conditional-analysis-results" class="section level2">
<h2><span class="header-section-number">3.6</span> Process conditional analysis results</h2>
<details>
<p><summary>Organise coloc results</summary></p>
<pre class="r"><code># Read in the report files
library(data.table)
setwd(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Conditional&quot;)

# Read in the clean TWAS results
twas_sign &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig_CLEAN.txt&quot;)
twas_sign$PANEL_clean&lt;-gsub(&#39; $&#39;,&#39;&#39;,twas_sign$PANEL_clean)

# Read in all jointly significant associations
temp = list.files(pattern=glob2rx(&quot;*chr*.report&quot;))
report&lt;-do.call(rbind, lapply(temp, function(x) read.table(x, header=T,stringsAsFactors=F)))
report$JOINT.ID&lt;-NA
report$MARGIN.ID&lt;-NA
report$JOINT.N&lt;-NA
report$MARGIN.N&lt;-NA
report$loc&lt;-gsub(&#39;.*loc_&#39;,&#39;&#39;,report$FILE)
joint_res&lt;-NULL
margin_res&lt;-NULL

# Insert names of jointly significant genes
for(i in unique(report$CHR)){
  joint_i&lt;-read.table(paste0(&#39;test.cond.chr&#39;,i,&#39;.joint_included.dat&#39;), header=T,stringsAsFactors=F)
  margin_i&lt;-read.table(paste0(&#39;test.cond.chr&#39;,i,&#39;.joint_dropped.dat&#39;), header=T,stringsAsFactors=F)
  
  joint_i$path&lt;-gsub(&#39;/[^/]+$&#39;,&#39;&#39;,joint_i$FILE)
    joint_i$path&lt;-gsub(&#39;/[^/]+$&#39;,&#39;&#39;,joint_i$path)
    joint_i$WGT&lt;-NA
    for(j in 1:dim(joint_i)[1]){
      joint_i$WGT[j]&lt;-gsub(paste0(joint_i$path[j],&#39;/&#39;),&#39;&#39;,joint_i$FILE[j])
  }

  if(dim(margin_i)[1] &gt; 0){
    margin_i$path&lt;-gsub(&#39;/[^/]+$&#39;,&#39;&#39;,margin_i$FILE)
    margin_i$path&lt;-gsub(&#39;/[^/]+$&#39;,&#39;&#39;,margin_i$path)
    margin_i$WGT&lt;-NA
    for(j in 1:dim(margin_i)[1]){
      margin_i$WGT[j]&lt;-gsub(paste0(margin_i$path[j],&#39;/&#39;),&#39;&#39;,margin_i$FILE[j])
    }
  }

  temp = list.files(pattern=glob2rx(paste0(&quot;*chr&quot;,i,&quot;.loc*.genes&quot;)))

  for(k in 1:length(temp)){
    loc_k&lt;-read.table(paste0(&#39;test.cond.chr&#39;,i,&#39;.loc_&#39;,k,&#39;.genes&#39;), header=T, stringsAsFactors=F)
    
    loc_k$path&lt;-gsub(&#39;/[^/]+$&#39;,&#39;&#39;,loc_k$FILE)
    loc_k$path&lt;-gsub(&#39;/[^/]+$&#39;,&#39;&#39;,loc_k$path)
    loc_k$WGT&lt;-NA
    for(j in 1:dim(loc_k)[1]){
      loc_k$WGT[j]&lt;-gsub(paste0(loc_k$path[j],&#39;/&#39;),&#39;&#39;,loc_k$FILE[j])
    }

    loc_k$P0&lt;-NULL
    loc_k$P1&lt;-NULL
    
    loc_k&lt;-merge(loc_k, twas_sign[,c(&#39;P0&#39;,&#39;P1&#39;,&#39;WGT&#39;,&#39;PANEL_clean&#39;)], by=&#39;WGT&#39;)
    
    loc_k_joint&lt;-loc_k[(loc_k$WGT %in% joint_i$WGT),]
    joint_res&lt;-rbind(joint_res,loc_k_joint)
        
    if(dim(margin_i)[1] &gt; 0){
      loc_k_margin&lt;-loc_k[(loc_k$WGT %in% margin_i$WGT),]
      margin_res&lt;-rbind(margin_res,loc_k_margin)
    }
    
    g_list&lt;-NULL
    for(g in unique(loc_k_joint$ID)){
      g_list&lt;-c(g_list,paste0(g, &quot; (&quot;,paste(loc_k_joint$PANEL_clean[loc_k_joint$ID == g], collapse=&#39;, &#39;),&quot;)&quot;))
    }
    report[report$CHR == i &amp; report$loc == k,]$JOINT.ID&lt;-paste(g_list,collapse=&#39;, &#39;)

    if(dim(loc_k_margin)[1] &gt; 0){
      g_list&lt;-NULL
      for(g in unique(loc_k_margin$ID)){
        g_list&lt;-c(g_list,paste0(g, &quot; (&quot;,paste(unique(loc_k_margin$PANEL_clean[loc_k_margin$ID == g]), collapse=&#39;, &#39;),&quot;)&quot;))
      }
      report[report$CHR == i &amp; report$loc == k,]$MARGIN.ID&lt;-paste(g_list,collapse=&#39;, &#39;)
    } else {
      report[report$CHR == i &amp; report$loc == k,]$MARGIN.ID&lt;-&#39;-&#39;
    }
    
    report[report$CHR == i &amp; report$loc == k,]$JOINT.N&lt;-dim(loc_k_joint)[1]
    report[report$CHR == i &amp; report$loc == k,]$MARGIN.N&lt;-dim(loc_k_margin)[1]
  }
}

report$LOCUS&lt;-paste0(report$CHR,&#39;:&#39;,report$P0,&#39;:&#39;,report$P1)
report$BP&lt;-paste0(report$P0,&#39;-&#39;,report$P1)
report$VAR.EXP&lt;-paste0(report$VAR.EXP*100,&#39;%&#39;)

report&lt;-report[,c(&#39;CHR&#39;,&#39;P0&#39;,&#39;P1&#39;,&#39;BP&#39;,&#39;LOCUS&#39;,&quot;JOINT.N&quot;,&#39;MARGIN.N&#39;,&#39;BEST.TWAS.P&#39;,&#39;BEST.SNP.P&#39;,&#39;VAR.EXP&#39;,&#39;JOINT.ID&#39;,&#39;MARGIN.ID&#39;)]

report&lt;-report[order(report$CHR, report$P0),]

# Save full conditional results table
write.csv(report[,c(&quot;CHR&quot;,&quot;BP&quot;,&quot;JOINT.ID&quot;,&quot;MARGIN.ID&quot;,&quot;BEST.TWAS.P&quot;,&quot;BEST.SNP.P&quot;,&quot;VAR.EXP&quot;)],&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Conditional/MDD_TWAS_Conditional_table_full.csv&#39;, row.names=F, quote=T)

# Save brief conditional results table
write.csv(report[,c(&#39;CHR&#39;,&#39;BP&#39;,&#39;JOINT.ID&#39;,&#39;MARGIN.N&#39;,&#39;BEST.TWAS.P&#39;,&#39;BEST.SNP.P&#39;,&#39;VAR.EXP&#39;)],&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Conditional/MDD_TWAS_Conditional_table_brief.csv&#39;, row.names=F, quote=T)

# Combine gene results for marginal and joint genes
joint_res$Type&lt;-&#39;Joint&#39;
margin_res$Type&lt;-&#39;Marginal&#39;

gene_res&lt;-rbind(joint_res, margin_res)

# Check number of indepenent associations
dim(joint_res) # 50

# Check number of independent associations without genome-wide significant snp
dim(joint_res[2*pnorm(-abs(joint_res$BEST.GWAS.Z)) &gt; 5e-8,]) # 25

# Check number of independent associations with genome-wide significant snp but an r2 with predicted expression &lt;0.1
dim(joint_res[2*pnorm(-abs(joint_res$BEST.GWAS.Z)) &lt; 5e-8 &amp; joint_res$TOP.SNP.COR^2 &lt; 0.1,]) # 2

# Check number of independent novel associations
dim(joint_res[(2*pnorm(-abs(joint_res$BEST.GWAS.Z)) &lt; 5e-8 &amp; joint_res$TOP.SNP.COR^2 &lt; 0.1) | 2*pnorm(-abs(joint_res$BEST.GWAS.Z)) &gt; 5e-8,]) # 27

# Check number of novel associations
dim(gene_res[(2*pnorm(-abs(gene_res$BEST.GWAS.Z)) &lt; 5e-8 &amp; gene_res$TOP.SNP.COR^2 &lt; 0.1) | 2*pnorm(-abs(gene_res$BEST.GWAS.Z)) &gt; 5e-8,]) # 68

gene_res$Novel&lt;-&#39;No&#39;
gene_res$Novel[(2*pnorm(-abs(gene_res$BEST.GWAS.Z)) &lt; 5e-8 &amp; gene_res$TOP.SNP.COR^2 &lt; 0.1) | 2*pnorm(-abs(gene_res$BEST.GWAS.Z)) &gt; 5e-8]&lt;-&#39;Yes&#39;

gene_res$BP&lt;-paste0(gene_res$P0,&#39;-&#39;,gene_res$P1)
gene_res$BEST.GWAS.P&lt;-2*pnorm(-abs(gene_res$BEST.GWAS.Z))

gene_res&lt;-gene_res[order(gene_res$CHR, gene_res$P0),]

gene_res$Colocalised&lt;-F
gene_res$Colocalised[gene_res$COLOC.PP4 &gt;0.8]&lt;-T

# Check number of independent novel associations which colocalise for joint genes
joint_res$Colocalised&lt;-F
joint_res$Colocalised[joint_res$COLOC.PP4 &gt;0.8]&lt;-T

dim(joint_res[(2*pnorm(-abs(joint_res$BEST.GWAS.Z)) &lt; 5e-8 &amp; joint_res$TOP.SNP.COR^2 &lt; 0.1 &amp; joint_res$Colocalised == T) | (2*pnorm(-abs(joint_res$BEST.GWAS.Z)) &gt; 5e-8 &amp; joint_res$Colocalised == T),]) # 12

# Check number of novel associations which colocalise
dim(gene_res[(2*pnorm(-abs(gene_res$BEST.GWAS.Z)) &lt; 5e-8 &amp; gene_res$TOP.SNP.COR^2 &lt; 0.1 &amp; gene_res$Colocalised == T) | (2*pnorm(-abs(gene_res$BEST.GWAS.Z)) &gt; 5e-8 &amp; gene_res$Colocalised == T),]) # 45

gene_res&lt;-gene_res[,c(&#39;CHR&#39;,&#39;BP&#39;,&#39;ID&#39;,&#39;PANEL_clean&#39;,&#39;WGT&#39;,&#39;TWAS.P&#39;,&#39;BEST.GWAS.P&#39;,&#39;TOP.SNP.COR&#39;,&#39;Type&#39;,&#39;Novel&#39;,&#39;COLOC.PP3&#39;,&#39;COLOC.PP4&#39;,&#39;Colocalised&#39;)]

# Save table showing whether gene associations are novel
write.csv(gene_res,&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Conditional/MDD_TWAS_Conditional_table_novelty.csv&#39;, row.names=F, quote=T)</code></pre>
</details>
<details>
<p><summary>Show full conditional analysis table</summary></p>
<table>
<caption>MDD TWAS Full Conditional Results</caption>
<thead>
<tr class="header">
<th align="right">CHR</th>
<th align="left">BP</th>
<th align="left">Jointly sign. Features (SNP-weight set)</th>
<th align="left">Marginally sign. Features (SNP-weight set)</th>
<th align="right">Top TWAS p-value</th>
<th align="right">Top GWAS p-value</th>
<th align="left">Variance Explained</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="left">7413452-9875347</td>
<td align="left">RERE (YFS Blood)</td>
<td align="left">RP5-1115A15.1 (GTEx Thyroid, GTEx Whole Blood), RERE (GTEx Whole Blood)</td>
<td align="right">1.10e-07</td>
<td align="right">3.18e-08</td>
<td align="left">100%</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">35885799-37876701</td>
<td align="left">SNORA63 (GTEx Nucleus accumbens)</td>
<td align="left">-</td>
<td align="right">1.24e-06</td>
<td align="right">6.27e-08</td>
<td align="left">69.3%</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">71753372-73766162</td>
<td align="left">NEGR1 (GTEx Whole Blood)</td>
<td align="left">NEGR1 (GTEx Caudate, GTEx Putamen), RPL31P12 (GTEx Cerebellar Hemisphere, GTEx Cerebellum, PsychENCODE)</td>
<td align="right">1.94e-18</td>
<td align="right">4.54e-15</td>
<td align="left">97.8%</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">174891875-177103690</td>
<td align="left">RFWD2 (CMC DLPFC Splicing)</td>
<td align="left">RFWD2 (CMC DLPFC Splicing), RP11-318C24.2 (GTEx Thyroid)</td>
<td align="right">4.66e-07</td>
<td align="right">2.30e-07</td>
<td align="left">90%</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">180725304-182724241</td>
<td align="left">CACNA1E (CMC DLPFC Splicing)</td>
<td align="left">-</td>
<td align="right">6.06e-07</td>
<td align="right">1.08e-07</td>
<td align="left">64%</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">196478918-198741422</td>
<td align="left">DENND1B (CMC DLPFC Splicing)</td>
<td align="left">DENND1B (CMC DLPFC Splicing, CMC DLPFC)</td>
<td align="right">5.90e-08</td>
<td align="right">3.11e-08</td>
<td align="left">92.6%</td>
</tr>
<tr class="odd">
<td align="right">2</td>
<td align="left">57388379-59467945</td>
<td align="left">FANCL (CMC DLPFC Splicing, CMC DLPFC)</td>
<td align="left">-</td>
<td align="right">2.18e-07</td>
<td align="right">4.68e-09</td>
<td align="left">85%</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="left">196832647-199295649</td>
<td align="left">ANKRD44 (YFS Blood)</td>
<td align="left">SF3B1 (GTEx Hypothalamus)</td>
<td align="right">1.27e-08</td>
<td align="right">3.52e-07</td>
<td align="left">82.8%</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">43487406-45561063</td>
<td align="left">ZNF445 (CMC DLPFC)</td>
<td align="left">EP300 (GTEx Cerebellum, YFS Blood), XPNPEP3 (GTEx Frontal Cortex, CMC DLPFC), SLC25A17 (GTEx Nucleus accumbens, GTEx Thyroid), RANGAP1 (CMC DLPFC Splicing, PsychENCODE)</td>
<td align="right">3.34e-07</td>
<td align="right">6.34e-08</td>
<td align="left">74.7%</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="left">40937584-43090938</td>
<td align="left">SLC30A9 (GTEx Cortex), TMEM33 (PsychENCODE)</td>
<td align="left">SLC30A9 (GTEx Amygdala, GTEx ACC, GTEx Caudate, GTEx Hypothalamus, GTEx Nucleus accumbens, PsychENCODE), RP11-814H16.2 (GTEx Cerebellar Hemisphere), DCAF4L1 (GTEx Thyroid)</td>
<td align="right">7.72e-09</td>
<td align="right">3.59e-09</td>
<td align="left">92.3%</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">86565927-88989352</td>
<td align="left">TMEM161B-AS1 (GTEx Caudate, PsychENCODE), CTC-467M3.3 (GTEx Frontal Cortex), CTC-498M16.4 (GTEx Substantia nigra)</td>
<td align="left">TMEM161B-AS1 (GTEx Adrenal Gland, GTEx Amygdala, GTEx ACC, GTEx Cerebellar Hemisphere, GTEx Cerebellum, GTEx Cortex, GTEx Frontal Cortex, GTEx Hypothalamus, GTEx Nucleus accumbens, GTEx Putamen, GTEx Substantia nigra, GTEx Pituitary, GTEx Thyroid, GTEx Whole Blood), CTC-467M3.3 (GTEx ACC, GTEx Cerebellar Hemisphere, GTEx Cortex, PsychENCODE)</td>
<td align="right">1.33e-12</td>
<td align="right">1.07e-10</td>
<td align="left">-2250%</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">139030460-141219083</td>
<td align="left">PCDHA5 (GTEx Thyroid)</td>
<td align="left">PCDHA8 (GTEx Cerebellar Hemisphere), NDUFA2 (CMC DLPFC)</td>
<td align="right">6.55e-08</td>
<td align="right">1.37e-06</td>
<td align="left">87.7%</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">25193720-29216321</td>
<td align="left">ZNF184 (GTEx Caudate), PRSS16 (GTEx Cerebellar Hemisphere, GTEx Pituitary), ZSCAN9 (GTEx Cerebellum), ZSCAN23 (GTEx Hypothalamus), BTN3A2 (NTR Blood), ZSCAN12P1 (PsychENCODE)</td>
<td align="left">RP5-874C20.3 (GTEx Adrenal Gland, GTEx Cerebellum, GTEx Hippocampus, GTEx Putamen, GTEx Thyroid, GTEx Whole Blood), ZNF204P (GTEx Adrenal Gland), ZKSCAN3 (GTEx Amygdala, GTEx Hippocampus, GTEx Thyroid), ZSCAN31 (GTEx Amygdala), PGBD1 (GTEx Cerebellar Hemisphere), BTN3A2 (GTEx Cerebellar Hemisphere, GTEx Hippocampus, GTEx Pituitary, GTEx Thyroid, GTEx Whole Blood), PRSS16 (GTEx Cerebellum, GTEx Frontal Cortex, GTEx Whole Blood), HMGN4 (GTEx Cerebellum), ZSCAN9 (GTEx Hippocampus, GTEx Pituitary), RP1-265C24.5 (GTEx Hippocampus), RP1-153G14.4 (GTEx Hippocampus), ZNF184 (GTEx Hypothalamus), ZSCAN23 (GTEx Putamen, GTEx Pituitary), HIST1H4D (NTR Blood), HIST1H4L (NTR Blood), NKAPL (PsychENCODE), ZSCAN12P1 (GTEx Whole Blood), ZSCAN16 (YFS Blood)</td>
<td align="right">2.53e-10</td>
<td align="right">1.34e-10</td>
<td align="left">100%</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">30577966-32580366</td>
<td align="left">BAG6 (CMC DLPFC Splicing), MICB (GTEx Thyroid), HCP5 (GTEx Thyroid)</td>
<td align="left">PPP1R18 (GTEx Adrenal Gland), FLOT1 (GTEx Cerebellum, CMC DLPFC Splicing, GTEx Pituitary, GTEx Thyroid), VARS2 (GTEx Cortex, GTEx Whole Blood), DDAH2 (GTEx Frontal Cortex, CMC DLPFC), WASF5P (GTEx Pituitary)</td>
<td align="right">1.55e-10</td>
<td align="right">3.95e-08</td>
<td align="left">86%</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">98832858-100829135</td>
<td align="left">COQ3 (CMC DLPFC Splicing)</td>
<td align="left">-</td>
<td align="right">2.65e-07</td>
<td align="right">9.09e-08</td>
<td align="left">35.1%</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">104405706-106583999</td>
<td align="left">BVES-AS1 (GTEx Amygdala)</td>
<td align="left">LIN28B (CMC DLPFC, PsychENCODE)</td>
<td align="right">2.43e-08</td>
<td align="right">9.50e-08</td>
<td align="left">92.9%</td>
</tr>
<tr class="odd">
<td align="right">7</td>
<td align="left">11252396-13282905</td>
<td align="left">TMEM106B (PsychENCODE)</td>
<td align="left">TMEM106B (GTEx Adrenal Gland, GTEx Whole Blood, YFS Blood)</td>
<td align="right">7.01e-09</td>
<td align="right">2.55e-08</td>
<td align="left">100%</td>
</tr>
<tr class="even">
<td align="right">7</td>
<td align="left">24021857-26019767</td>
<td align="left">OSBPL3 (GTEx Pituitary)</td>
<td align="left">-</td>
<td align="right">1.88e-08</td>
<td align="right">6.49e-07</td>
<td align="left">77.7%</td>
</tr>
<tr class="odd">
<td align="right">8</td>
<td align="left">51238261-53720740</td>
<td align="left">PXDNL (CMC DLPFC)</td>
<td align="left">-</td>
<td align="right">3.92e-09</td>
<td align="right">1.34e-07</td>
<td align="left">83.8%</td>
</tr>
<tr class="even">
<td align="right">8</td>
<td align="left">60435234-62428932</td>
<td align="left">RP11-163N6.2 (GTEx Thyroid)</td>
<td align="left">-</td>
<td align="right">9.47e-08</td>
<td align="right">5.25e-07</td>
<td align="left">89.8%</td>
</tr>
<tr class="odd">
<td align="right">9</td>
<td align="left">125606617-127604411</td>
<td align="left">PIGFP2 (PsychENCODE)</td>
<td align="left">-</td>
<td align="right">1.12e-07</td>
<td align="right">2.73e-08</td>
<td align="left">63.8%</td>
</tr>
<tr class="even">
<td align="right">11</td>
<td align="left">56092913-58422547</td>
<td align="left">TNKS1BP1 (GTEx Adrenal Gland), CLP1 (GTEx Whole Blood)</td>
<td align="left">AP000662.4 (GTEx Thyroid)</td>
<td align="right">2.04e-07</td>
<td align="right">1.47e-07</td>
<td align="left">95.2%</td>
</tr>
<tr class="odd">
<td align="right">11</td>
<td align="left">60540194-62557903</td>
<td align="left">TMEM258 (PsychENCODE)</td>
<td align="left">-</td>
<td align="right">5.12e-07</td>
<td align="right">4.26e-07</td>
<td align="left">83.9%</td>
</tr>
<tr class="even">
<td align="right">11</td>
<td align="left">112346414-114345882</td>
<td align="left">DRD2 (GTEx Frontal Cortex)</td>
<td align="left">-</td>
<td align="right">3.90e-07</td>
<td align="right">4.90e-07</td>
<td align="left">0.414%</td>
</tr>
<tr class="odd">
<td align="right">13</td>
<td align="left">52652520-54625616</td>
<td align="left">OLFM4 (CMC DLPFC)</td>
<td align="left">-</td>
<td align="right">3.56e-07</td>
<td align="right">6.06e-19</td>
<td align="left">29.9%</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">41077086-43073683</td>
<td align="left">CTD-2298J14.2 (GTEx Thyroid)</td>
<td align="left">LRFN5 (GTEx Cerebellar Hemisphere, GTEx Cerebellum)</td>
<td align="right">1.36e-08</td>
<td align="right">2.57e-09</td>
<td align="left">88.1%</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">58952573-61334943</td>
<td align="left">CCDC175 (GTEx Thyroid)</td>
<td align="left">RTN1 (CMC DLPFC Splicing, GTEx Thyroid), JKAMP (GTEx Thyroid)</td>
<td align="right">4.28e-08</td>
<td align="right">2.18e-07</td>
<td align="left">82.3%</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">63322572-65770213</td>
<td align="left">ESR2 (GTEx Pituitary)</td>
<td align="left">SYNE2 (NTR Blood), ESR2 (GTEx Whole Blood)</td>
<td align="right">2.20e-09</td>
<td align="right">7.60e-10</td>
<td align="left">80%</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">74120633-76388050</td>
<td align="left">PROX2 (GTEx Thyroid)</td>
<td align="left">RPS6KL1 (CMC DLPFC Splicing, PsychENCODE), DLST (CMC DLPFC, PsychENCODE), AREL1 (PsychENCODE)</td>
<td align="right">8.51e-09</td>
<td align="right">6.71e-09</td>
<td align="left">93.5%</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">102878783-105180229</td>
<td align="left">RP11-894P9.2 (GTEx Thyroid)</td>
<td align="left">RP11-73M18.8 (GTEx Amygdala), AL049840.1 (GTEx Cerebellum, GTEx Cortex), RP11-73M18.9 (GTEx Cortex, PsychENCODE), TRMT61A (CMC DLPFC, GTEx Whole Blood), RP11-600F24.2 (PsychENCODE), RP11-73M18.7 (PsychENCODE), RP11-73M18.6 (PsychENCODE), CKB (YFS Blood)</td>
<td align="right">4.69e-08</td>
<td align="right">3.05e-09</td>
<td align="left">84.6%</td>
</tr>
<tr class="odd">
<td align="right">16</td>
<td align="left">71147494-73210261</td>
<td align="left">PMFBP1 (PsychENCODE)</td>
<td align="left">RP11-73M18.8 (GTEx Amygdala), AL049840.1 (GTEx Cerebellum, GTEx Cortex), RP11-73M18.9 (GTEx Cortex, PsychENCODE), TRMT61A (CMC DLPFC, GTEx Whole Blood), RP11-600F24.2 (PsychENCODE), RP11-73M18.7 (PsychENCODE), RP11-73M18.6 (PsychENCODE), CKB (YFS Blood)</td>
<td align="right">2.46e-07</td>
<td align="right">3.35e-08</td>
<td align="left">76.3%</td>
</tr>
<tr class="even">
<td align="right">17</td>
<td align="left">26406423-28478661</td>
<td align="left">TIAF1 (GTEx Adrenal Gland)</td>
<td align="left">MYO18A (GTEx Adrenal Gland)</td>
<td align="right">8.27e-08</td>
<td align="right">8.51e-09</td>
<td align="left">58.5%</td>
</tr>
<tr class="odd">
<td align="right">17</td>
<td align="left">64524284-66521332</td>
<td align="left">CTD-2653B5.1 (PsychENCODE)</td>
<td align="left">-</td>
<td align="right">3.30e-07</td>
<td align="right">5.39e-06</td>
<td align="left">25.8%</td>
</tr>
<tr class="even">
<td align="right">18</td>
<td align="left">51385406-53561919</td>
<td align="left">RAB27B (PsychENCODE)</td>
<td align="left">RAB27B (CMC DLPFC Splicing)</td>
<td align="right">5.36e-07</td>
<td align="right">3.62e-11</td>
<td align="left">14.6%</td>
</tr>
<tr class="odd">
<td align="right">20</td>
<td align="left">46838019-48853908</td>
<td align="left">DDX27 (CMC DLPFC)</td>
<td align="left">SF3B1 (GTEx Hypothalamus)</td>
<td align="right">1.32e-06</td>
<td align="right">3.54e-06</td>
<td align="left">91%</td>
</tr>
<tr class="even">
<td align="right">22</td>
<td align="left">40218102-42697216</td>
<td align="left">ZC3H7B (GTEx Cerebellum)</td>
<td align="left">EP300 (GTEx Cerebellum, YFS Blood), XPNPEP3 (GTEx Frontal Cortex, CMC DLPFC), SLC25A17 (GTEx Nucleus accumbens, GTEx Thyroid), RANGAP1 (CMC DLPFC Splicing, PsychENCODE)</td>
<td align="right">1.01e-08</td>
<td align="right">7.56e-09</td>
<td align="left">95.5%</td>
</tr>
</tbody>
</table>
</details>
<details>
<p><summary>Show brief conditional analysis table</summary></p>
<table>
<caption>MDD TWAS Brief Conditional Results</caption>
<thead>
<tr class="header">
<th align="right">CHR</th>
<th align="left">BP</th>
<th align="left">Jointly sign. Features (SNP-weight set)</th>
<th align="right">N Marginal</th>
<th align="right">Top TWAS p-value</th>
<th align="right">Top GWAS p-value</th>
<th align="left">Variance Explained</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="left">7413452-9875347</td>
<td align="left">RERE (YFS Blood)</td>
<td align="right">3</td>
<td align="right">1.10e-07</td>
<td align="right">3.18e-08</td>
<td align="left">100%</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">35885799-37876701</td>
<td align="left">SNORA63 (GTEx Nucleus accumbens)</td>
<td align="right">0</td>
<td align="right">1.24e-06</td>
<td align="right">6.27e-08</td>
<td align="left">69.3%</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">71753372-73766162</td>
<td align="left">NEGR1 (GTEx Whole Blood)</td>
<td align="right">5</td>
<td align="right">1.94e-18</td>
<td align="right">4.54e-15</td>
<td align="left">97.8%</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">174891875-177103690</td>
<td align="left">RFWD2 (CMC DLPFC Splicing)</td>
<td align="right">3</td>
<td align="right">4.66e-07</td>
<td align="right">2.30e-07</td>
<td align="left">90%</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">180725304-182724241</td>
<td align="left">CACNA1E (CMC DLPFC Splicing)</td>
<td align="right">0</td>
<td align="right">6.06e-07</td>
<td align="right">1.08e-07</td>
<td align="left">64%</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">196478918-198741422</td>
<td align="left">DENND1B (CMC DLPFC Splicing)</td>
<td align="right">2</td>
<td align="right">5.90e-08</td>
<td align="right">3.11e-08</td>
<td align="left">92.6%</td>
</tr>
<tr class="odd">
<td align="right">2</td>
<td align="left">57388379-59467945</td>
<td align="left">FANCL (CMC DLPFC Splicing, CMC DLPFC)</td>
<td align="right">0</td>
<td align="right">2.18e-07</td>
<td align="right">4.68e-09</td>
<td align="left">85%</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="left">196832647-199295649</td>
<td align="left">ANKRD44 (YFS Blood)</td>
<td align="right">1</td>
<td align="right">1.27e-08</td>
<td align="right">3.52e-07</td>
<td align="left">82.8%</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">43487406-45561063</td>
<td align="left">ZNF445 (CMC DLPFC)</td>
<td align="right">8</td>
<td align="right">3.34e-07</td>
<td align="right">6.34e-08</td>
<td align="left">74.7%</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="left">40937584-43090938</td>
<td align="left">SLC30A9 (GTEx Cortex), TMEM33 (PsychENCODE)</td>
<td align="right">8</td>
<td align="right">7.72e-09</td>
<td align="right">3.59e-09</td>
<td align="left">92.3%</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">86565927-88989352</td>
<td align="left">TMEM161B-AS1 (GTEx Caudate, PsychENCODE), CTC-467M3.3 (GTEx Frontal Cortex), CTC-498M16.4 (GTEx Substantia nigra)</td>
<td align="right">18</td>
<td align="right">1.33e-12</td>
<td align="right">1.07e-10</td>
<td align="left">-2250%</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">139030460-141219083</td>
<td align="left">PCDHA5 (GTEx Thyroid)</td>
<td align="right">2</td>
<td align="right">6.55e-08</td>
<td align="right">1.37e-06</td>
<td align="left">87.7%</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">25193720-29216321</td>
<td align="left">ZNF184 (GTEx Caudate), PRSS16 (GTEx Cerebellar Hemisphere, GTEx Pituitary), ZSCAN9 (GTEx Cerebellum), ZSCAN23 (GTEx Hypothalamus), BTN3A2 (NTR Blood), ZSCAN12P1 (PsychENCODE)</td>
<td align="right">33</td>
<td align="right">2.53e-10</td>
<td align="right">1.34e-10</td>
<td align="left">100%</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">30577966-32580366</td>
<td align="left">BAG6 (CMC DLPFC Splicing), MICB (GTEx Thyroid), HCP5 (GTEx Thyroid)</td>
<td align="right">12</td>
<td align="right">1.55e-10</td>
<td align="right">3.95e-08</td>
<td align="left">86%</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">98832858-100829135</td>
<td align="left">COQ3 (CMC DLPFC Splicing)</td>
<td align="right">0</td>
<td align="right">2.65e-07</td>
<td align="right">9.09e-08</td>
<td align="left">35.1%</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">104405706-106583999</td>
<td align="left">BVES-AS1 (GTEx Amygdala)</td>
<td align="right">2</td>
<td align="right">2.43e-08</td>
<td align="right">9.50e-08</td>
<td align="left">92.9%</td>
</tr>
<tr class="odd">
<td align="right">7</td>
<td align="left">11252396-13282905</td>
<td align="left">TMEM106B (PsychENCODE)</td>
<td align="right">3</td>
<td align="right">7.01e-09</td>
<td align="right">2.55e-08</td>
<td align="left">100%</td>
</tr>
<tr class="even">
<td align="right">7</td>
<td align="left">24021857-26019767</td>
<td align="left">OSBPL3 (GTEx Pituitary)</td>
<td align="right">0</td>
<td align="right">1.88e-08</td>
<td align="right">6.49e-07</td>
<td align="left">77.7%</td>
</tr>
<tr class="odd">
<td align="right">8</td>
<td align="left">51238261-53720740</td>
<td align="left">PXDNL (CMC DLPFC)</td>
<td align="right">0</td>
<td align="right">3.92e-09</td>
<td align="right">1.34e-07</td>
<td align="left">83.8%</td>
</tr>
<tr class="even">
<td align="right">8</td>
<td align="left">60435234-62428932</td>
<td align="left">RP11-163N6.2 (GTEx Thyroid)</td>
<td align="right">0</td>
<td align="right">9.47e-08</td>
<td align="right">5.25e-07</td>
<td align="left">89.8%</td>
</tr>
<tr class="odd">
<td align="right">9</td>
<td align="left">125606617-127604411</td>
<td align="left">PIGFP2 (PsychENCODE)</td>
<td align="right">0</td>
<td align="right">1.12e-07</td>
<td align="right">2.73e-08</td>
<td align="left">63.8%</td>
</tr>
<tr class="even">
<td align="right">11</td>
<td align="left">56092913-58422547</td>
<td align="left">TNKS1BP1 (GTEx Adrenal Gland), CLP1 (GTEx Whole Blood)</td>
<td align="right">1</td>
<td align="right">2.04e-07</td>
<td align="right">1.47e-07</td>
<td align="left">95.2%</td>
</tr>
<tr class="odd">
<td align="right">11</td>
<td align="left">60540194-62557903</td>
<td align="left">TMEM258 (PsychENCODE)</td>
<td align="right">0</td>
<td align="right">5.12e-07</td>
<td align="right">4.26e-07</td>
<td align="left">83.9%</td>
</tr>
<tr class="even">
<td align="right">11</td>
<td align="left">112346414-114345882</td>
<td align="left">DRD2 (GTEx Frontal Cortex)</td>
<td align="right">0</td>
<td align="right">3.90e-07</td>
<td align="right">4.90e-07</td>
<td align="left">0.414%</td>
</tr>
<tr class="odd">
<td align="right">13</td>
<td align="left">52652520-54625616</td>
<td align="left">OLFM4 (CMC DLPFC)</td>
<td align="right">0</td>
<td align="right">3.56e-07</td>
<td align="right">6.06e-19</td>
<td align="left">29.9%</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">41077086-43073683</td>
<td align="left">CTD-2298J14.2 (GTEx Thyroid)</td>
<td align="right">2</td>
<td align="right">1.36e-08</td>
<td align="right">2.57e-09</td>
<td align="left">88.1%</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">58952573-61334943</td>
<td align="left">CCDC175 (GTEx Thyroid)</td>
<td align="right">3</td>
<td align="right">4.28e-08</td>
<td align="right">2.18e-07</td>
<td align="left">82.3%</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">63322572-65770213</td>
<td align="left">ESR2 (GTEx Pituitary)</td>
<td align="right">2</td>
<td align="right">2.20e-09</td>
<td align="right">7.60e-10</td>
<td align="left">80%</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">74120633-76388050</td>
<td align="left">PROX2 (GTEx Thyroid)</td>
<td align="right">5</td>
<td align="right">8.51e-09</td>
<td align="right">6.71e-09</td>
<td align="left">93.5%</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">102878783-105180229</td>
<td align="left">RP11-894P9.2 (GTEx Thyroid)</td>
<td align="right">11</td>
<td align="right">4.69e-08</td>
<td align="right">3.05e-09</td>
<td align="left">84.6%</td>
</tr>
<tr class="odd">
<td align="right">16</td>
<td align="left">71147494-73210261</td>
<td align="left">PMFBP1 (PsychENCODE)</td>
<td align="right">11</td>
<td align="right">2.46e-07</td>
<td align="right">3.35e-08</td>
<td align="left">76.3%</td>
</tr>
<tr class="even">
<td align="right">17</td>
<td align="left">26406423-28478661</td>
<td align="left">TIAF1 (GTEx Adrenal Gland)</td>
<td align="right">1</td>
<td align="right">8.27e-08</td>
<td align="right">8.51e-09</td>
<td align="left">58.5%</td>
</tr>
<tr class="odd">
<td align="right">17</td>
<td align="left">64524284-66521332</td>
<td align="left">CTD-2653B5.1 (PsychENCODE)</td>
<td align="right">0</td>
<td align="right">3.30e-07</td>
<td align="right">5.39e-06</td>
<td align="left">25.8%</td>
</tr>
<tr class="even">
<td align="right">18</td>
<td align="left">51385406-53561919</td>
<td align="left">RAB27B (PsychENCODE)</td>
<td align="right">1</td>
<td align="right">5.36e-07</td>
<td align="right">3.62e-11</td>
<td align="left">14.6%</td>
</tr>
<tr class="odd">
<td align="right">20</td>
<td align="left">46838019-48853908</td>
<td align="left">DDX27 (CMC DLPFC)</td>
<td align="right">1</td>
<td align="right">1.32e-06</td>
<td align="right">3.54e-06</td>
<td align="left">91%</td>
</tr>
<tr class="even">
<td align="right">22</td>
<td align="left">40218102-42697216</td>
<td align="left">ZC3H7B (GTEx Cerebellum)</td>
<td align="right">8</td>
<td align="right">1.01e-08</td>
<td align="right">7.56e-09</td>
<td align="left">95.5%</td>
</tr>
</tbody>
</table>
</details>
<details>
<p><summary>Show novelty table</summary></p>
<table>
<caption>MDD TWAS Results Novelty compared to GWAS</caption>
<thead>
<tr class="header">
<th align="right">CHR</th>
<th align="left">BP</th>
<th align="left">ID</th>
<th align="left">PANEL</th>
<th align="left">WGT</th>
<th align="right">TWAS.P</th>
<th align="left">BEST.GWAS.P</th>
<th align="right">TOP.SNP.COR</th>
<th align="left">TYPE</th>
<th align="left">NOVEL</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="left">8412457-8877702</td>
<td align="left">RERE</td>
<td align="left">YFS Blood</td>
<td align="left">YFS.BLOOD.RNAARR/YFS.RERE.wgt.RDat</td>
<td align="right">1.10e-07</td>
<td align="left">3.59e-08</td>
<td align="right">-0.97</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">8412457-8877702</td>
<td align="left">RERE</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000142599.13.wgt.RDat</td>
<td align="right">3.47e-07</td>
<td align="left">3.59e-08</td>
<td align="right">-0.98</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">8484705-8494898</td>
<td align="left">RP5-1115A15.1</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000232912.1.wgt.RDat</td>
<td align="right">2.28e-07</td>
<td align="left">3.59e-08</td>
<td align="right">-0.94</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">8484705-8494898</td>
<td align="left">RP5-1115A15.1</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000232912.1.wgt.RDat</td>
<td align="right">1.14e-06</td>
<td align="left">3.59e-08</td>
<td align="right">-0.75</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">36884051-36884179</td>
<td align="left">SNORA63</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="left">Brain_Nucleus_accumbens_basal_ganglia/Brain_Nucleus_accumbens_basal_ganglia.ENSG00000201448.1.wgt.RDat</td>
<td align="right">1.24e-06</td>
<td align="left">7.45e-08</td>
<td align="right">0.64</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000172260.9.wgt.RDat</td>
<td align="right">1.94e-18</td>
<td align="left">5.72e-15</td>
<td align="right">-0.82</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Caudate</td>
<td align="left">Brain_Caudate_basal_ganglia/Brain_Caudate_basal_ganglia.ENSG00000172260.9.wgt.RDat</td>
<td align="right">7.47e-09</td>
<td align="left">5.72e-15</td>
<td align="right">-0.69</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">71861623-72748417</td>
<td align="left">NEGR1</td>
<td align="left">GTEx Putamen</td>
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000172260.9.wgt.RDat</td>
<td align="right">2.88e-08</td>
<td align="left">5.72e-15</td>
<td align="right">-0.82</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000227207.2.wgt.RDat</td>
<td align="right">6.94e-15</td>
<td align="left">5.72e-15</td>
<td align="right">0.99</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000227207.2.wgt.RDat</td>
<td align="right">1.27e-14</td>
<td align="left">5.72e-15</td>
<td align="right">1.00</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">72767155-72767512</td>
<td align="left">RPL31P12</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000227207.wgt.RDat</td>
<td align="right">9.73e-15</td>
<td align="left">4.54e-15</td>
<td align="right">0.99</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">175873898-175889649</td>
<td align="left">RP11-318C24.2</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000227740.1.wgt.RDat</td>
<td align="right">4.97e-07</td>
<td align="left">2.34e-07</td>
<td align="right">0.88</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:176085817:176104146:clu_42334.wgt.RDat</td>
<td align="right">4.66e-07</td>
<td align="left">2.34e-07</td>
<td align="right">-0.86</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:176085817:176102983:clu_42334.wgt.RDat</td>
<td align="right">7.10e-07</td>
<td align="left">2.34e-07</td>
<td align="right">0.93</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">175913966-176176370</td>
<td align="left">RFWD2</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:176103036:176104146:clu_42334.wgt.RDat</td>
<td align="right">5.56e-07</td>
<td align="left">2.34e-07</td>
<td align="right">0.84</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">181452685-181775921</td>
<td align="left">CACNA1E</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:181724533:181725092:clu_42439.wgt.RDat</td>
<td align="right">6.06e-07</td>
<td align="left">1.10e-07</td>
<td align="right">-0.52</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:197684204:197704716:clu_42580.wgt.RDat</td>
<td align="right">5.90e-08</td>
<td align="left">8.80e-08</td>
<td align="right">0.90</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">1</td>
<td align="left">197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:197684204:197741998:clu_42580.wgt.RDat</td>
<td align="right">5.22e-07</td>
<td align="left">8.80e-08</td>
<td align="right">-0.89</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">1</td>
<td align="left">197473878-197744623</td>
<td align="left">DENND1B</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.DENND1B.wgt.RDat</td>
<td align="right">1.24e-06</td>
<td align="left">8.80e-08</td>
<td align="right">-0.78</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="left">58386377-58468515</td>
<td align="left">FANCL</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr2:58388773:58390001:clu_36265.wgt.RDat</td>
<td align="right">9.71e-07</td>
<td align="left">4.63e-09</td>
<td align="right">0.52</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">2</td>
<td align="left">58386377-58468515</td>
<td align="left">FANCL</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.FANCL.wgt.RDat</td>
<td align="right">2.18e-07</td>
<td align="left">4.63e-09</td>
<td align="right">-0.64</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="left">197831741-198175897</td>
<td align="left">ANKRD44</td>
<td align="left">YFS Blood</td>
<td align="left">YFS.BLOOD.RNAARR/YFS.ANKRD44.wgt.RDat</td>
<td align="right">1.27e-08</td>
<td align="left">7.81e-07</td>
<td align="right">-0.60</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">2</td>
<td align="left">198254508-198299815</td>
<td align="left">SF3B1</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000115524.11.wgt.RDat</td>
<td align="right">1.84e-07</td>
<td align="left">4.90e-07</td>
<td align="right">0.81</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">44481261-44561226</td>
<td align="left">ZNF445</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.ZNF445.wgt.RDat</td>
<td align="right">3.34e-07</td>
<td align="left">9.82e-08</td>
<td align="right">-0.66</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">4</td>
<td align="left">41937137-41962589</td>
<td align="left">TMEM33</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000109133.wgt.RDat</td>
<td align="right">1.32e-06</td>
<td align="left">2.27e-08</td>
<td align="right">-0.42</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="left">41983713-41988476</td>
<td align="left">DCAF4L1</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000182308.5.wgt.RDat</td>
<td align="right">2.93e-07</td>
<td align="left">3.64e-09</td>
<td align="right">0.64</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">4</td>
<td align="left">41990758-41991254</td>
<td align="left">RP11-814H16.2</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000272862.1.wgt.RDat</td>
<td align="right">5.45e-07</td>
<td align="left">3.64e-09</td>
<td align="right">-0.74</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="left">41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Cortex</td>
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000014824.9.wgt.RDat</td>
<td align="right">7.72e-09</td>
<td align="left">3.64e-09</td>
<td align="right">0.73</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">4</td>
<td align="left">41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Amygdala</td>
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000014824.9.wgt.RDat</td>
<td align="right">1.49e-07</td>
<td align="left">3.64e-09</td>
<td align="right">0.79</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="left">41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx ACC</td>
<td align="left">Brain_Anterior_cingulate_cortex_BA24/Brain_Anterior_cingulate_cortex_BA24.ENSG00000014824.9.wgt.RDat</td>
<td align="right">5.68e-07</td>
<td align="left">3.64e-09</td>
<td align="right">0.81</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">4</td>
<td align="left">41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Caudate</td>
<td align="left">Brain_Caudate_basal_ganglia/Brain_Caudate_basal_ganglia.ENSG00000014824.9.wgt.RDat</td>
<td align="right">1.21e-06</td>
<td align="left">3.64e-09</td>
<td align="right">0.78</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="left">41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000014824.9.wgt.RDat</td>
<td align="right">3.67e-07</td>
<td align="left">3.64e-09</td>
<td align="right">0.83</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">4</td>
<td align="left">41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="left">Brain_Nucleus_accumbens_basal_ganglia/Brain_Nucleus_accumbens_basal_ganglia.ENSG00000014824.9.wgt.RDat</td>
<td align="right">2.11e-08</td>
<td align="left">3.64e-09</td>
<td align="right">0.82</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="left">41992489-42092474</td>
<td align="left">SLC30A9</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000014824.wgt.RDat</td>
<td align="right">1.45e-07</td>
<td align="left">2.27e-08</td>
<td align="right">0.76</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87564712-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000247828.wgt.RDat</td>
<td align="right">1.12e-09</td>
<td align="left">1.88e-10</td>
<td align="right">0.57</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Caudate</td>
<td align="left">Brain_Caudate_basal_ganglia/Brain_Caudate_basal_ganglia.ENSG00000247828.3.wgt.RDat</td>
<td align="right">3.34e-10</td>
<td align="left">1.66e-10</td>
<td align="right">0.67</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000247828.3.wgt.RDat</td>
<td align="right">8.32e-08</td>
<td align="left">1.66e-10</td>
<td align="right">0.52</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Amygdala</td>
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000247828.3.wgt.RDat</td>
<td align="right">9.45e-10</td>
<td align="left">1.66e-10</td>
<td align="right">0.58</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx ACC</td>
<td align="left">Brain_Anterior_cingulate_cortex_BA24/Brain_Anterior_cingulate_cortex_BA24.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.15e-10</td>
<td align="left">1.66e-10</td>
<td align="right">0.69</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.84e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.61</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.42e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.58</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Cortex</td>
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.73e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.61</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.82e-11</td>
<td align="left">1.66e-10</td>
<td align="right">0.70</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000247828.3.wgt.RDat</td>
<td align="right">4.21e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.57</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="left">Brain_Nucleus_accumbens_basal_ganglia/Brain_Nucleus_accumbens_basal_ganglia.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.85e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.61</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Putamen</td>
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.87e-10</td>
<td align="left">1.66e-10</td>
<td align="right">0.65</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Substantia nigra</td>
<td align="left">Brain_Substantia_nigra/Brain_Substantia_nigra.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.38e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.60</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000247828.3.wgt.RDat</td>
<td align="right">1.46e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.58</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000247828.3.wgt.RDat</td>
<td align="right">3.87e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.61</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87564888-87732502</td>
<td align="left">TMEM161B-AS1</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000247828.3.wgt.RDat</td>
<td align="right">3.27e-08</td>
<td align="left">1.66e-10</td>
<td align="right">0.53</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87729709-87794514</td>
<td align="left">CTC-498M16.4</td>
<td align="left">GTEx Substantia nigra</td>
<td align="left">Brain_Substantia_nigra/Brain_Substantia_nigra.ENSG00000271904.1.wgt.RDat</td>
<td align="right">6.53e-08</td>
<td align="left">1.66e-10</td>
<td align="right">0.82</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000250377.1.wgt.RDat</td>
<td align="right">1.33e-12</td>
<td align="left">1.66e-10</td>
<td align="right">-0.45</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx ACC</td>
<td align="left">Brain_Anterior_cingulate_cortex_BA24/Brain_Anterior_cingulate_cortex_BA24.ENSG00000250377.1.wgt.RDat</td>
<td align="right">6.11e-09</td>
<td align="left">1.66e-10</td>
<td align="right">-0.67</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000250377.1.wgt.RDat</td>
<td align="right">4.60e-09</td>
<td align="left">1.66e-10</td>
<td align="right">-0.30</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">GTEx Cortex</td>
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000250377.1.wgt.RDat</td>
<td align="right">7.47e-11</td>
<td align="left">1.66e-10</td>
<td align="right">-0.41</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">87988462-87989789</td>
<td align="left">CTC-467M3.3</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000250377.wgt.RDat</td>
<td align="right">1.07e-09</td>
<td align="left">1.88e-10</td>
<td align="right">-0.12</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">140024947-140027370</td>
<td align="left">NDUFA2</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.NDUFA2.wgt.RDat</td>
<td align="right">2.10e-07</td>
<td align="left">1.37e-06</td>
<td align="right">-0.86</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">5</td>
<td align="left">140201222-140203811</td>
<td align="left">PCDHA5</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000204965.4.wgt.RDat</td>
<td align="right">6.55e-08</td>
<td align="left">1.37e-06</td>
<td align="right">0.66</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="left">140220907-140223351</td>
<td align="left">PCDHA8</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000204962.4.wgt.RDat</td>
<td align="right">6.36e-07</td>
<td align="left">1.37e-06</td>
<td align="right">0.86</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">26188921-26189323</td>
<td align="left">HIST1H4D</td>
<td align="left">NTR Blood</td>
<td align="left">NTR.BLOOD.RNAARR/NTR.HIST1H4D.wgt.RDat</td>
<td align="right">6.11e-07</td>
<td align="left">2.15e-10</td>
<td align="right">0.42</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">26365386-26378540</td>
<td align="left">BTN3A2</td>
<td align="left">NTR Blood</td>
<td align="left">NTR.BLOOD.RNAARR/NTR.BTN3A2.wgt.RDat</td>
<td align="right">1.00e-07</td>
<td align="left">2.15e-10</td>
<td align="right">-0.61</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000186470.9.wgt.RDat</td>
<td align="right">2.12e-07</td>
<td align="left">2.15e-10</td>
<td align="right">-0.61</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000186470.9.wgt.RDat</td>
<td align="right">6.96e-07</td>
<td align="left">2.15e-10</td>
<td align="right">-0.62</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000186470.9.wgt.RDat</td>
<td align="right">3.66e-09</td>
<td align="left">2.15e-10</td>
<td align="right">-0.64</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000186470.9.wgt.RDat</td>
<td align="right">4.22e-08</td>
<td align="left">2.15e-10</td>
<td align="right">-0.61</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">26365387-26378546</td>
<td align="left">BTN3A2</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000186470.9.wgt.RDat</td>
<td align="right">3.64e-07</td>
<td align="left">2.15e-10</td>
<td align="right">-0.61</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">26538633-26546482</td>
<td align="left">HMGN4</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000182952.4.wgt.RDat</td>
<td align="right">6.84e-08</td>
<td align="left">2.15e-10</td>
<td align="right">-0.32</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000112812.11.wgt.RDat</td>
<td align="right">1.00e-06</td>
<td align="left">2.79e-10</td>
<td align="right">0.56</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000112812.11.wgt.RDat</td>
<td align="right">3.30e-09</td>
<td align="left">2.79e-10</td>
<td align="right">0.30</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000112812.11.wgt.RDat</td>
<td align="right">7.50e-07</td>
<td align="left">2.79e-10</td>
<td align="right">0.55</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000112812.11.wgt.RDat</td>
<td align="right">4.54e-07</td>
<td align="left">2.79e-10</td>
<td align="right">0.48</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">27215480-27224250</td>
<td align="left">PRSS16</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000112812.11.wgt.RDat</td>
<td align="right">9.51e-08</td>
<td align="left">2.79e-10</td>
<td align="right">0.47</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">27325604-27339304</td>
<td align="left">ZNF204P</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000204789.3.wgt.RDat</td>
<td align="right">4.84e-07</td>
<td align="left">2.79e-10</td>
<td align="right">0.60</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">27371789-27374743</td>
<td align="left">RP1-153G14.4</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000271755.1.wgt.RDat</td>
<td align="right">8.60e-08</td>
<td align="left">2.79e-10</td>
<td align="right">-0.28</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">27418522-27440897</td>
<td align="left">ZNF184</td>
<td align="left">GTEx Caudate</td>
<td align="left">Brain_Caudate_basal_ganglia/Brain_Caudate_basal_ganglia.ENSG00000096654.11.wgt.RDat</td>
<td align="right">2.53e-10</td>
<td align="left">2.79e-10</td>
<td align="right">0.53</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">27418522-27440897</td>
<td align="left">ZNF184</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000096654.11.wgt.RDat</td>
<td align="right">7.34e-07</td>
<td align="left">2.79e-10</td>
<td align="right">0.66</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">27840926-27841289</td>
<td align="left">HIST1H4L</td>
<td align="left">NTR Blood</td>
<td align="left">NTR.BLOOD.RNAARR/NTR.HIST1H4L.wgt.RDat</td>
<td align="right">1.11e-06</td>
<td align="left">1.66e-10</td>
<td align="right">-0.78</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28058932-28061442</td>
<td align="left">ZSCAN12P1</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000219891.wgt.RDat</td>
<td align="right">3.66e-10</td>
<td align="left">1.36e-10</td>
<td align="right">-0.49</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28058932-28061442</td>
<td align="left">ZSCAN12P1</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000219891.2.wgt.RDat</td>
<td align="right">7.94e-07</td>
<td align="left">1.66e-10</td>
<td align="right">0.64</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28083406-28084329</td>
<td align="left">RP1-265C24.5</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000219392.1.wgt.RDat</td>
<td align="right">3.16e-08</td>
<td align="left">1.66e-10</td>
<td align="right">-0.73</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28092338-28097860</td>
<td align="left">ZSCAN16</td>
<td align="left">YFS Blood</td>
<td align="left">YFS.BLOOD.RNAARR/YFS.ZSCAN16.wgt.RDat</td>
<td align="right">1.00e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.80</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000137185.7.wgt.RDat</td>
<td align="right">1.11e-07</td>
<td align="left">1.66e-10</td>
<td align="right">0.73</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000137185.7.wgt.RDat</td>
<td align="right">1.77e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.65</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28192664-28201260</td>
<td align="left">ZSCAN9</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000137185.7.wgt.RDat</td>
<td align="right">7.32e-10</td>
<td align="left">1.66e-10</td>
<td align="right">0.70</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28227098-28228736</td>
<td align="left">NKAPL</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000189134.wgt.RDat</td>
<td align="right">5.65e-07</td>
<td align="left">1.36e-10</td>
<td align="right">-0.45</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000197062.7.wgt.RDat</td>
<td align="right">3.49e-07</td>
<td align="left">1.66e-10</td>
<td align="right">-0.72</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000197062.7.wgt.RDat</td>
<td align="right">4.13e-07</td>
<td align="left">1.66e-10</td>
<td align="right">-0.80</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000197062.7.wgt.RDat</td>
<td align="right">2.01e-07</td>
<td align="left">1.66e-10</td>
<td align="right">-0.69</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Putamen</td>
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000197062.7.wgt.RDat</td>
<td align="right">9.52e-09</td>
<td align="left">1.66e-10</td>
<td align="right">-0.84</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000197062.7.wgt.RDat</td>
<td align="right">9.38e-08</td>
<td align="left">1.66e-10</td>
<td align="right">-0.82</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28234788-28245974</td>
<td align="left">RP5-874C20.3</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000197062.7.wgt.RDat</td>
<td align="right">1.49e-08</td>
<td align="left">1.66e-10</td>
<td align="right">-0.77</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28249314-28270326</td>
<td align="left">PGBD1</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000137338.4.wgt.RDat</td>
<td align="right">2.74e-10</td>
<td align="left">1.66e-10</td>
<td align="right">0.99</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28292470-28324048</td>
<td align="left">ZSCAN31</td>
<td align="left">GTEx Amygdala</td>
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000235109.3.wgt.RDat</td>
<td align="right">3.69e-07</td>
<td align="left">1.66e-10</td>
<td align="right">0.57</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Amygdala</td>
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000189298.9.wgt.RDat</td>
<td align="right">7.43e-07</td>
<td align="left">1.66e-10</td>
<td align="right">-0.51</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000189298.9.wgt.RDat</td>
<td align="right">7.37e-07</td>
<td align="left">1.66e-10</td>
<td align="right">-0.74</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28317691-28336947</td>
<td align="left">ZKSCAN3</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000189298.9.wgt.RDat</td>
<td align="right">1.11e-09</td>
<td align="left">1.66e-10</td>
<td align="right">-0.78</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000187987.5.wgt.RDat</td>
<td align="right">7.58e-09</td>
<td align="left">1.66e-10</td>
<td align="right">0.74</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Putamen</td>
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000187987.5.wgt.RDat</td>
<td align="right">9.90e-07</td>
<td align="left">1.66e-10</td>
<td align="right">0.70</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">28399707-28411279</td>
<td align="left">ZSCAN23</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000187987.5.wgt.RDat</td>
<td align="right">7.30e-07</td>
<td align="left">1.66e-10</td>
<td align="right">0.72</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">30644166-30655672</td>
<td align="left">PPP1R18</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000146112.7.wgt.RDat</td>
<td align="right">9.10e-07</td>
<td align="left">1.16e-07</td>
<td align="right">-0.02</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:30708575:30709391:clu_26190.wgt.RDat</td>
<td align="right">1.16e-07</td>
<td align="left">1.16e-07</td>
<td align="right">0.43</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:30708575:30709924:clu_26190.wgt.RDat</td>
<td align="right">4.04e-07</td>
<td align="left">1.16e-07</td>
<td align="right">0.43</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:30709110:30709391:clu_26190.wgt.RDat</td>
<td align="right">7.95e-07</td>
<td align="left">1.16e-07</td>
<td align="right">-0.40</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000137312.10.wgt.RDat</td>
<td align="right">1.16e-07</td>
<td align="left">1.16e-07</td>
<td align="right">0.42</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000137312.10.wgt.RDat</td>
<td align="right">1.49e-07</td>
<td align="left">1.16e-07</td>
<td align="right">0.41</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000137312.10.wgt.RDat</td>
<td align="right">2.74e-08</td>
<td align="left">1.16e-07</td>
<td align="right">0.44</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">30881982-30894236</td>
<td align="left">VARS2</td>
<td align="left">GTEx Cortex</td>
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000137411.12.wgt.RDat</td>
<td align="right">3.18e-09</td>
<td align="left">1.16e-07</td>
<td align="right">-0.66</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">30881982-30894236</td>
<td align="left">VARS2</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000137411.12.wgt.RDat</td>
<td align="right">2.56e-10</td>
<td align="left">1.16e-07</td>
<td align="right">-0.60</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">31255287-31256741</td>
<td align="left">WASF5P</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000231402.1.wgt.RDat</td>
<td align="right">2.52e-07</td>
<td align="left">1.16e-07</td>
<td align="right">0.54</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">31368479-31445283</td>
<td align="left">HCP5</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000206337.6.wgt.RDat</td>
<td align="right">1.55e-10</td>
<td align="left">1.16e-07</td>
<td align="right">-0.58</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">31462658-31478901</td>
<td align="left">MICB</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000204516.5.wgt.RDat</td>
<td align="right">2.74e-08</td>
<td align="left">1.16e-07</td>
<td align="right">0.20</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">31606805-31620482</td>
<td align="left">BAG6</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:31619553:31620177:clu_26248.wgt.RDat</td>
<td align="right">2.40e-08</td>
<td align="left">5.44e-07</td>
<td align="right">0.40</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">31694815-31698357</td>
<td align="left">DDAH2</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000213722.4.wgt.RDat</td>
<td align="right">6.32e-08</td>
<td align="left">5.44e-07</td>
<td align="right">-0.30</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">31694816-31698039</td>
<td align="left">DDAH2</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.DDAH2.wgt.RDat</td>
<td align="right">9.07e-08</td>
<td align="left">5.44e-07</td>
<td align="right">-0.46</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">99817347-99842082</td>
<td align="left">COQ3</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:99831006:99831574:clu_27287.wgt.RDat</td>
<td align="right">2.65e-07</td>
<td align="left">1.70e-07</td>
<td align="right">0.22</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">105404922-105531207</td>
<td align="left">LIN28B</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.LIN28B.wgt.RDat</td>
<td align="right">1.68e-07</td>
<td align="left">1.16e-07</td>
<td align="right">0.97</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">6</td>
<td align="left">105404923-105531207</td>
<td align="left">LIN28B</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000187772.wgt.RDat</td>
<td align="right">3.30e-07</td>
<td align="left">9.29e-08</td>
<td align="right">0.73</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="left">105584224-105617820</td>
<td align="left">BVES-AS1</td>
<td align="left">GTEx Amygdala</td>
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000203808.6.wgt.RDat</td>
<td align="right">2.43e-08</td>
<td align="left">1.16e-07</td>
<td align="right">0.81</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">7</td>
<td align="left">12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000106460.wgt.RDat</td>
<td align="right">7.01e-09</td>
<td align="left">5.33e-08</td>
<td align="right">-0.95</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">7</td>
<td align="left">12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000106460.14.wgt.RDat</td>
<td align="right">3.69e-08</td>
<td align="left">5.96e-08</td>
<td align="right">0.98</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">7</td>
<td align="left">12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000106460.14.wgt.RDat</td>
<td align="right">3.18e-08</td>
<td align="left">5.96e-08</td>
<td align="right">0.82</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">7</td>
<td align="left">12250867-12276886</td>
<td align="left">TMEM106B</td>
<td align="left">YFS Blood</td>
<td align="left">YFS.BLOOD.RNAARR/YFS.TMEM106B.wgt.RDat</td>
<td align="right">7.72e-08</td>
<td align="left">5.96e-08</td>
<td align="right">0.95</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">7</td>
<td align="left">24836158-25021253</td>
<td align="left">OSBPL3</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000070882.8.wgt.RDat</td>
<td align="right">1.88e-08</td>
<td align="left">7.05e-07</td>
<td align="right">-0.53</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">8</td>
<td align="left">52232136-52722005</td>
<td align="left">PXDNL</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.PXDNL.wgt.RDat</td>
<td align="right">3.92e-09</td>
<td align="left">1.52e-07</td>
<td align="right">0.61</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">8</td>
<td align="left">61297147-61429354</td>
<td align="left">RP11-163N6.2</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000251396.2.wgt.RDat</td>
<td align="right">9.47e-08</td>
<td align="left">5.17e-07</td>
<td align="right">-0.74</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">9</td>
<td align="left">126605315-126605965</td>
<td align="left">PIGFP2</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000234881.wgt.RDat</td>
<td align="right">1.12e-07</td>
<td align="left">2.70e-08</td>
<td align="right">-0.50</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">11</td>
<td align="left">57067112-57092426</td>
<td align="left">TNKS1BP1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000149115.9.wgt.RDat</td>
<td align="right">8.54e-07</td>
<td align="left">6.70e-07</td>
<td align="right">0.49</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">11</td>
<td align="left">57405497-57420263</td>
<td align="left">AP000662.4</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000254602.1.wgt.RDat</td>
<td align="right">6.35e-07</td>
<td align="left">6.70e-07</td>
<td align="right">-0.62</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">11</td>
<td align="left">57424488-57429340</td>
<td align="left">CLP1</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000172409.5.wgt.RDat</td>
<td align="right">2.04e-07</td>
<td align="left">6.70e-07</td>
<td align="right">0.88</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">11</td>
<td align="left">61535973-61560274</td>
<td align="left">TMEM258</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000134825.wgt.RDat</td>
<td align="right">5.12e-07</td>
<td align="left">4.19e-07</td>
<td align="right">0.73</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">11</td>
<td align="left">113280318-113346111</td>
<td align="left">DRD2</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000149295.9.wgt.RDat</td>
<td align="right">3.90e-07</td>
<td align="left">4.90e-07</td>
<td align="right">0.00</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">13</td>
<td align="left">53602875-53626196</td>
<td align="left">OLFM4</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.OLFM4.wgt.RDat</td>
<td align="right">3.56e-07</td>
<td align="left">6.11e-19</td>
<td align="right">0.42</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">42057064-42074059</td>
<td align="left">CTD-2298J14.2</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000258636.1.wgt.RDat</td>
<td align="right">1.36e-08</td>
<td align="left">2.85e-09</td>
<td align="right">-0.87</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">42076773-42373752</td>
<td align="left">LRFN5</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000165379.9.wgt.RDat</td>
<td align="right">5.85e-08</td>
<td align="left">2.85e-09</td>
<td align="right">0.78</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">42076773-42373752</td>
<td align="left">LRFN5</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000165379.9.wgt.RDat</td>
<td align="right">2.17e-08</td>
<td align="left">2.85e-09</td>
<td align="right">0.89</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">59951161-59971429</td>
<td align="left">JKAMP</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000050130.13.wgt.RDat</td>
<td align="right">2.97e-07</td>
<td align="left">3.77e-07</td>
<td align="right">-0.62</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">59971779-60043549</td>
<td align="left">CCDC175</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000151838.7.wgt.RDat</td>
<td align="right">4.28e-08</td>
<td align="left">3.77e-07</td>
<td align="right">-0.64</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">60062693-60337557</td>
<td align="left">RTN1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr14:60074210:60097164:clu_16682.wgt.RDat</td>
<td align="right">1.09e-06</td>
<td align="left">3.77e-07</td>
<td align="right">-0.85</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">60062695-60337684</td>
<td align="left">RTN1</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000139970.12.wgt.RDat</td>
<td align="right">8.87e-08</td>
<td align="left">3.77e-07</td>
<td align="right">-0.77</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">64319682-64693151</td>
<td align="left">SYNE2</td>
<td align="left">NTR Blood</td>
<td align="left">NTR.BLOOD.RNAARR/NTR.SYNE2.wgt.RDat</td>
<td align="right">2.03e-08</td>
<td align="left">7.75e-10</td>
<td align="right">-0.94</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">64550950-64770377</td>
<td align="left">ESR2</td>
<td align="left">GTEx Pituitary</td>
<td align="left">Pituitary/Pituitary.ENSG00000140009.14.wgt.RDat</td>
<td align="right">2.20e-09</td>
<td align="left">7.75e-10</td>
<td align="right">0.70</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">64550950-64770377</td>
<td align="left">ESR2</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000140009.14.wgt.RDat</td>
<td align="right">1.56e-08</td>
<td align="left">7.75e-10</td>
<td align="right">0.91</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">75120140-75179818</td>
<td align="left">AREL1</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000119682.wgt.RDat</td>
<td align="right">5.30e-07</td>
<td align="left">6.63e-09</td>
<td align="right">-0.77</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">75319736-75330537</td>
<td align="left">PROX2</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000119608.8.wgt.RDat</td>
<td align="right">8.51e-09</td>
<td align="left">1.13e-08</td>
<td align="right">-0.89</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">75348593-75370450</td>
<td align="left">DLST</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.DLST.wgt.RDat</td>
<td align="right">6.31e-07</td>
<td align="left">1.13e-08</td>
<td align="right">0.89</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">75348594-75370448</td>
<td align="left">DLST</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000119689.wgt.RDat</td>
<td align="right">3.59e-07</td>
<td align="left">6.63e-09</td>
<td align="right">0.80</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">75370656-75389188</td>
<td align="left">RPS6KL1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr14:75375893:75377951:clu_16995.wgt.RDat</td>
<td align="right">5.07e-07</td>
<td align="left">1.13e-08</td>
<td align="right">-0.57</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">75370657-75390099</td>
<td align="left">RPS6KL1</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000198208.wgt.RDat</td>
<td align="right">7.32e-07</td>
<td align="left">6.63e-09</td>
<td align="right">-0.87</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">103878456-103879098</td>
<td align="left">RP11-600F24.2</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000244691.wgt.RDat</td>
<td align="right">2.15e-07</td>
<td align="left">3.03e-09</td>
<td align="right">-0.64</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">103985996-103989448</td>
<td align="left">CKB</td>
<td align="left">YFS Blood</td>
<td align="left">YFS.BLOOD.RNAARR/YFS.CKB.wgt.RDat</td>
<td align="right">8.99e-08</td>
<td align="left">7.87e-08</td>
<td align="right">-0.97</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">103995508-104003410</td>
<td align="left">TRMT61A</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.TRMT61A.wgt.RDat</td>
<td align="right">4.39e-07</td>
<td align="left">7.87e-08</td>
<td align="right">-0.57</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">103995521-104003410</td>
<td align="left">TRMT61A</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">Whole_Blood/Whole_Blood.ENSG00000166166.8.wgt.RDat</td>
<td align="right">6.44e-07</td>
<td align="left">7.87e-08</td>
<td align="right">-0.64</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">104019758-104028214</td>
<td align="left">RP11-894P9.2</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000258851.1.wgt.RDat</td>
<td align="right">4.69e-08</td>
<td align="left">7.87e-08</td>
<td align="right">0.94</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">104153913-104154464</td>
<td align="left">RP11-73M18.6</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000270108.wgt.RDat</td>
<td align="right">4.87e-07</td>
<td align="left">3.03e-09</td>
<td align="right">-0.63</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">104160897-104161507</td>
<td align="left">RP11-73M18.7</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000269940.wgt.RDat</td>
<td align="right">1.20e-06</td>
<td align="left">3.03e-09</td>
<td align="right">-0.65</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">104162690-104163500</td>
<td align="left">RP11-73M18.8</td>
<td align="left">GTEx Amygdala</td>
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000269958.1.wgt.RDat</td>
<td align="right">2.72e-07</td>
<td align="left">7.87e-08</td>
<td align="right">-0.70</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">104177607-104179149</td>
<td align="left">AL049840.1</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000224997.1.wgt.RDat</td>
<td align="right">4.92e-07</td>
<td align="left">7.87e-08</td>
<td align="right">-0.60</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">104177607-104179149</td>
<td align="left">AL049840.1</td>
<td align="left">GTEx Cortex</td>
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000224997.1.wgt.RDat</td>
<td align="right">2.69e-07</td>
<td align="left">7.87e-08</td>
<td align="right">-0.72</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">14</td>
<td align="left">104179904-104180441</td>
<td align="left">RP11-73M18.9</td>
<td align="left">GTEx Cortex</td>
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000269963.1.wgt.RDat</td>
<td align="right">6.45e-07</td>
<td align="left">7.87e-08</td>
<td align="right">-0.69</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">14</td>
<td align="left">104179904-104180586</td>
<td align="left">RP11-73M18.9</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000269963.wgt.RDat</td>
<td align="right">1.36e-06</td>
<td align="left">3.03e-09</td>
<td align="right">-0.62</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">16</td>
<td align="left">72146056-72210777</td>
<td align="left">PMFBP1</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000118557.wgt.RDat</td>
<td align="right">2.46e-07</td>
<td align="left">4.76e-08</td>
<td align="right">-0.70</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">17</td>
<td align="left">27400528-27507430</td>
<td align="left">MYO18A</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000196535.10.wgt.RDat</td>
<td align="right">2.92e-07</td>
<td align="left">4.19e-07</td>
<td align="right">-0.62</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">17</td>
<td align="left">27401933-27405875</td>
<td align="left">TIAF1</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000221995.4.wgt.RDat</td>
<td align="right">8.27e-08</td>
<td align="left">4.19e-07</td>
<td align="right">-0.46</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">17</td>
<td align="left">65520597-65521538</td>
<td align="left">CTD-2653B5.1</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000264754.wgt.RDat</td>
<td align="right">3.30e-07</td>
<td align="left">2.92e-05</td>
<td align="right">0.13</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">18</td>
<td align="left">52385091-52562747</td>
<td align="left">RAB27B</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000041353.wgt.RDat</td>
<td align="right">5.36e-07</td>
<td align="left">7.47e-09</td>
<td align="right">0.11</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">18</td>
<td align="left">52495707-52562747</td>
<td align="left">RAB27B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr18:52385372:52544798:clu_20555.wgt.RDat</td>
<td align="right">1.28e-06</td>
<td align="left">3.06e-07</td>
<td align="right">0.11</td>
<td align="left">Marginal</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="right">20</td>
<td align="left">47835831-47860614</td>
<td align="left">DDX27</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.DDX27.wgt.RDat</td>
<td align="right">1.32e-06</td>
<td align="left">3.48e-06</td>
<td align="right">-0.78</td>
<td align="left">Joint</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="right">22</td>
<td align="left">41165634-41215403</td>
<td align="left">SLC25A17</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="left">Brain_Nucleus_accumbens_basal_ganglia/Brain_Nucleus_accumbens_basal_ganglia.ENSG00000100372.10.wgt.RDat</td>
<td align="right">3.83e-07</td>
<td align="left">1.00e-08</td>
<td align="right">0.74</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">22</td>
<td align="left">41165634-41215403</td>
<td align="left">SLC25A17</td>
<td align="left">GTEx Thyroid</td>
<td align="left">Thyroid/Thyroid.ENSG00000100372.10.wgt.RDat</td>
<td align="right">9.78e-07</td>
<td align="left">1.00e-08</td>
<td align="right">0.47</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">22</td>
<td align="left">41253088-41351450</td>
<td align="left">XPNPEP3</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000196236.8.wgt.RDat</td>
<td align="right">7.38e-07</td>
<td align="left">1.00e-08</td>
<td align="right">0.45</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">22</td>
<td align="left">41258260-41363888</td>
<td align="left">XPNPEP3</td>
<td align="left">CMC DLPFC</td>
<td align="left">CMC.BRAIN.RNASEQ/CMC.XPNPEP3.wgt.RDat</td>
<td align="right">3.21e-07</td>
<td align="left">1.00e-08</td>
<td align="right">0.64</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">22</td>
<td align="left">41487790-41576081</td>
<td align="left">EP300</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000100393.9.wgt.RDat</td>
<td align="right">3.93e-08</td>
<td align="left">1.00e-08</td>
<td align="right">0.85</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">22</td>
<td align="left">41487790-41576081</td>
<td align="left">EP300</td>
<td align="left">YFS Blood</td>
<td align="left">YFS.BLOOD.RNAARR/YFS.EP300.wgt.RDat</td>
<td align="right">4.21e-07</td>
<td align="left">1.00e-08</td>
<td align="right">0.61</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">22</td>
<td align="left">41641614-41682216</td>
<td align="left">RANGAP1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr22:41657584:41664101:clu_21526.wgt.RDat</td>
<td align="right">1.61e-07</td>
<td align="left">1.00e-08</td>
<td align="right">0.57</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="right">22</td>
<td align="left">41641615-41682255</td>
<td align="left">RANGAP1</td>
<td align="left">PsychENCODE</td>
<td align="left">PEC_TWAS_weights/ENSG00000100401.wgt.RDat</td>
<td align="right">2.47e-08</td>
<td align="left">7.47e-09</td>
<td align="right">-0.81</td>
<td align="left">Marginal</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="right">22</td>
<td align="left">41697526-41756151</td>
<td align="left">ZC3H7B</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000100403.10.wgt.RDat</td>
<td align="right">1.01e-08</td>
<td align="left">1.00e-08</td>
<td align="right">0.93</td>
<td align="left">Joint</td>
<td align="left">No</td>
</tr>
</tbody>
</table>
</details>
<hr />
</div>
<div id="focus" class="section level2">
<h2><span class="header-section-number">3.7</span> FOCUS</h2>
<details>
<p><summary>Merge the relevent FOCUS databases</summary></p>
<pre class="bash"><code># Create a list of databases to be merged.
cat &lt;&lt; &#39;EOF&#39; &gt; /scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/MDD_TWAS_db/MDD_TWAS_db_list.txt
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Adrenal_Gland.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Amygdala.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Anterior_cingulate_cortex_BA24.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Caudate_basal_ganglia.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Cerebellar_Hemisphere.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Cerebellum.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Cortex.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Frontal_Cortex_BA9.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Hippocampus.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Hypothalamus.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Nucleus_accumbens_basal_ganglia.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Putamen_basal_ganglia.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Spinal_cord_cervical_c-1.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Brain_Substantia_nigra.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/CMC.BRAIN.RNASEQ.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/CMC.BRAIN.RNASEQ_SPLICING.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/NTR.BLOOD.RNAARR.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/PEC_TWAS_weights.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Pituitary.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Thyroid.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/Whole_Blood.db
/scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/SNP-weights/YFS.BLOOD.RNAARR.db
EOF

####
# Merge the databases
####
# I have written a script to do this in R
/users/k1806347/brc_scratch/Software/Rscript.sh /scratch/users/k1806347/Software/MyGit/FOCUS_db_merger/FOCUS_db_merger.r \
--dbs /scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/MDD_TWAS_db/MDD_TWAS_db_list.txt \
--out /scratch/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/MDD_TWAS_db/MDD_TWAS

# This approach is giving strange results.
# Delete the output</code></pre>
</details>
<details>
<p><summary>Create FOCUS database for MDD TWAS</summary></p>
<pre class="bash"><code># Merging the databases causes some strange warnings
# Use the standard approach for the time being

########
# Import the FUSION SNP-weights to FOCUS format
########

# Import with all tissues in the MDD TWAS combined
# A shell script to do this has been written
mkdir -p /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/MDD_TWAS_db

# Fusion weights
sbatch -p brc,shared --mem=10G /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/MDD_TWAS_db/create_db_fusion.sh

# Onyl continue once the FUSION SNP-weights are in the database
# Psych ENCODE weights
sbatch -p brc,shared --mem=10G /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/MDD_TWAS_db/create_db_psychENCODE.sh</code></pre>
</details>
<details>
<p><summary>Run FOCUS</summary></p>
<pre class="bash"><code>########
# Finemap TWAS associations
########

# Use a threshold that will run FOCUS for all TWAS significant loci (max GWAS.P = 5e-6)
for chr in $(seq 1 22); do
sbatch -p brc,shared --mem=10G /users/k1806347/brc_scratch/Software/focus.sh finemap /users/k1806347/brc_scratch/Data/GWAS_sumstats/Lorenza/DEPR01.focus.sumstats.gz  /scratch/groups/biomarkers-brc-mh/Reference_data/1KG_Phase3/PLINK/EUR/EUR_phase3.MAF_001.chr${chr} /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FOCUS/MDD_TWAS_db/MDD_TWAS.db --chr ${chr} --p-threshold 5e-6 --plot --out /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/FOCUS/MDD.FOCUS.MDD_TWAS_db.chr${chr}
done
</code></pre>
</details>
<details>
<p><summary>Process the FOCUS results</summary></p>
<pre class="r"><code>library(data.table)

fusion &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig_CLEAN.txt&quot;)

focus.files&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/FOCUS/&#39;, pattern=glob2rx(&quot;MDD.FOCUS.MDD_TWAS_db.chr*.focus.tsv&quot;))
length(focus.files)
focus&lt;-NULL
for(i in focus.files){
focus&lt;-rbind(focus,fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/FOCUS/&#39;,i)))
}

# Update the feature IDs with gene names
pos&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos&#39;)
focus&lt;-merge(focus, pos[,c(&#39;WGT&#39;,&#39;ID&#39;)], by.x=&#39;mol_name&#39;, by.y=&#39;WGT&#39;,all.x=T)
focus&lt;-focus[order(focus$chrom, focus$region, 1-focus$pip),]

# I noticed a bug in the output where features that should be in the 90% credible set are not
focus_bug&lt;-NULL
for(i in unique(focus$region)){
    focus_temp&lt;-focus[focus$region == i,]
    if(sum(focus_temp$in_cred_set) == 0 &amp; max(focus_temp$pip) != focus_temp$pip[focus_temp$ens_gene_id == &#39;NULL.MODEL&#39;]){
    print(head(focus_temp))
        focus_bug&lt;-rbind(focus_bug, focus_temp)
    }
}

# This shows several features should be in the credible set
focus$in_cred_set[focus$ID == &#39;OLFM4&#39; &amp; focus$tissue == &#39;cmc.brain.rnaseq&#39; &amp; focus$region == &#39;13:53339645-13:54682393&#39;] &lt;- 1
focus$in_cred_set[focus$ID == &#39;ENSG00000229267&#39; &amp; focus$tissue == &#39;pec_twas_weights&#39; &amp; focus$region == &#39;2:214014511-2:215573795&#39;] &lt;- 1
focus$in_cred_set[focus$ID == &#39;COQ3&#39; &amp; focus$tissue == &#39;cmc.brain.rnaseq_splicing&#39; &amp; focus$region == &#39;6:97842747-6:100629728&#39;] &lt;- 1
focus$in_cred_set[focus$ID == &#39;OSBPL3&#39; &amp; focus$tissue == &#39;pituitary&#39; &amp; focus$region == &#39;7:23471523-7:25077097&#39;] &lt;- 1
focus$in_cred_set[focus$ID == &#39;PXDNL&#39; &amp; focus$tissue == &#39;cmc.brain.rnaseq&#39; &amp; focus$region == &#39;8:50082470-8:53302930&#39;] &lt;- 1

# Update tissue for psychencode features
focus_psychencode&lt;-focus[focus$tissue == &#39;pec_twas_weights&#39;,]
focus_fusion&lt;-focus[focus$tissue != &#39;pec_twas_weights&#39;,]
focus_psychencode$tissue&lt;-&#39;psychencode&#39;

# Update PsychENCODE gene IDs from ensembl to gene names
library(biomaRt)
ensembl = useEnsembl(biomart=&quot;ensembl&quot;, dataset=&quot;hsapiens_gene_ensembl&quot;, GRCh=37)
listAttributes(ensembl)
Genes&lt;-getBM(attributes=c(&#39;ensembl_gene_id&#39;,&#39;external_gene_name&#39;), mart = ensembl)

focus_psychencode&lt;-merge(focus_psychencode, Genes, by.x=&#39;ID&#39;, by.y=&#39;ensembl_gene_id&#39;)
focus_psychencode$ID&lt;-focus_psychencode$external_gene_name
focus_psychencode$external_gene_name&lt;-NULL
focus_psychencode&lt;-focus_psychencode[,names(focus),with=F]
focus&lt;-rbind(focus_fusion,focus_psychencode)

fusion_focus&lt;-merge(fusion, focus[,c(&#39;mol_name&#39;,&#39;tissue&#39;,&#39;twas_z&#39;,&#39;pip&#39;,&#39;in_cred_set&#39;,&#39;region&#39;),with=F], by.x=c(&#39;WGT&#39;), by.y=c(&#39;mol_name&#39;), all.x=T)
fusion_focus&lt;-fusion_focus[,c(&#39;WGT&#39;,&#39;CHR&#39;,&#39;P0&#39;,&#39;P1&#39;,&#39;PANEL_clean_short&#39;,&#39;ID&#39;,&#39;TWAS.Z&#39;,&#39;TWAS.P&#39;,&#39;twas_z&#39;,&#39;in_cred_set&#39;,&#39;pip&#39;,&#39;region&#39;),with=F]
names(fusion_focus)&lt;-c(&#39;WGT&#39;,&#39;CHR&#39;,&#39;P0&#39;,&#39;P1&#39;,&#39;SNP-weight Set&#39;,&#39;ID&#39;,&#39;TWAS.Z&#39;,&#39;TWAS.P&#39;,&#39;FOCUS_twas_z&#39;,&#39;FOCUS_in_cred_set&#39;,&#39;FOCUS_pip&#39;,&#39;FOCUS_region&#39;)
fusion_focus&lt;-fusion_focus[order(fusion_focus$CHR, fusion_focus$P0),]
fusion_focus$Location&lt;-paste0(&#39;chr&#39;,fusion_focus$CHR,&#39;:&#39;,fusion_focus$P0,&#39;-&#39;,fusion_focus$P1)   
fusion_focus&lt;-fusion_focus[,c(&#39;Location&#39;,&#39;SNP-weight Set&#39;,&#39;ID&#39;,&#39;TWAS.Z&#39;,&#39;TWAS.P&#39;,&#39;FOCUS_twas_z&#39;,&#39;FOCUS_in_cred_set&#39;,&#39;FOCUS_pip&#39;,&#39;FOCUS_region&#39;),with=F]

write.csv(fusion_focus,&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/FOCUS/MDD_TWAS_sig_FOCUS_results.csv&#39;, row.names=F, quote=F)
write.csv(focus,&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/FOCUS/MDD_TWAS_FOCUS_results.csv&#39;, row.names=F, quote=F)</code></pre>
</details>
<details>
<p><summary>Combine FOCUS results with novelty table</summary></p>
<pre class="r"><code>library(data.table)

focus&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/FOCUS/MDD_TWAS_FOCUS_results.csv&#39;)
fusion&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Conditional/MDD_TWAS_Conditional_table_novelty.csv&#39;)

fusion_focus&lt;-merge(fusion, focus[,c(&#39;mol_name&#39;,&#39;tissue&#39;,&#39;twas_z&#39;,&#39;pip&#39;,&#39;in_cred_set&#39;,&#39;region&#39;),with=F], by.x=c(&#39;WGT&#39;), by.y=c(&#39;mol_name&#39;), all.x=T)

fusion_focus&lt;-fusion_focus[,c(&#39;WGT&#39;,&#39;CHR&#39;,&#39;P0&#39;,&#39;P1&#39;,&#39;PANEL_clean&#39;,&#39;ID&#39;,&#39;TWAS.Z&#39;,&#39;TWAS.P&#39;,&#39;Novel&#39;,&#39;Colocalised&#39;,&#39;in_cred_set&#39;,&#39;pip&#39;,&#39;region&#39;),with=F]
names(fusion_focus)&lt;-c(&#39;WGT&#39;,&#39;CHR&#39;,&#39;P0&#39;,&#39;P1&#39;,&#39;SNP-weight Set&#39;,&#39;ID&#39;,&#39;TWAS.Z&#39;,&#39;TWAS.P&#39;,&#39;Novel&#39;,&#39;Colocalised&#39;,&#39;FOCUS_in_cred_set&#39;,&#39;FOCUS_pip&#39;,&#39;FOCUS_region&#39;)
fusion_focus&lt;-fusion_focus[order(fusion_focus$CHR, fusion_focus$P0),]
fusion_focus$Location&lt;-paste0(&#39;chr&#39;,fusion_focus$CHR,&#39;:&#39;,fusion_focus$P0,&#39;-&#39;,fusion_focus$P1)   

# Remove the MHC region
fusion_focus_noMHC&lt;-fusion_focus[!(fusion_focus$CHR == 6 &amp; fusion_focus$P1 &gt; 26e6 &amp; fusion_focus$P0 &lt; 34e6),]

# Subset those which are high confidence
fusion_focus_highConf&lt;-fusion_focus_noMHC[fusion_focus_noMHC$Colocalised == T &amp; fusion_focus_noMHC$FOCUS_pip &gt; 0.5 &amp; fusion_focus_noMHC$TWAS.P &lt; 3.685926e-08,]

# One high confidence gene is novel (TMEM106B)

# Subset transcriptom-wide significant that colocalised and pip &gt; 0.5
fusion_focus_TWsig&lt;-fusion_focus_noMHC[fusion_focus_noMHC$TWAS.P &lt; 1.368572e-06,]

sum(duplicated(fusion_focus_TWsig$WGT)) # 2
fusion_focus_TWsig[(fusion_focus_TWsig$WGT %in% fusion_focus_TWsig$WGT[duplicated(fusion_focus_TWsig$WGT)]),]
# The FANCL feature is duplicated due to FOCUS defining two nearby regions of association. A splice variant of FANCL is in the credible set for one region but not the other.

# Subset transcriptom-wide significant that colocalised and pip &gt; 0.5
fusion_focus_TWsig&lt;-fusion_focus_noMHC[fusion_focus_noMHC$Colocalised == T &amp; fusion_focus_noMHC$FOCUS_pip &gt; 0.5 &amp; fusion_focus_noMHC$TWAS.P &lt; 1.368572e-06,]

write.csv(fusion_focus,&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/FOCUS/MDD_TWAS_sig_FOCUS_results.csv&#39;, row.names=F, quote=F)
write.csv(focus,&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/FOCUS/MDD_TWAS_FOCUS_results.csv&#39;, row.names=F, quote=F)</code></pre>
</details>
<details>
<p><summary>Show FOCUS and other combined results</summary></p>
<table>
<caption>MDD TWAS Results with FOCUS</caption>
<thead>
<tr class="header">
<th align="left">WGT</th>
<th align="right">CHR</th>
<th align="right">P0</th>
<th align="right">P1</th>
<th align="left">SNP.weight.Set</th>
<th align="left">ID</th>
<th align="right">TWAS.Z</th>
<th align="right">TWAS.P</th>
<th align="left">Novel</th>
<th align="left">Colocalised</th>
<th align="right">FOCUS_in_cred_set</th>
<th align="right">FOCUS_pip</th>
<th align="left">FOCUS_region</th>
<th align="left">Location</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000142599.13.wgt.RDat</td>
<td align="right">1</td>
<td align="right">8412457</td>
<td align="right">8877702</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">RERE</td>
<td align="right">-5.095707</td>
<td align="right">3.47e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.110000</td>
<td align="left">1:7247335-1:9365199</td>
<td align="left">chr1:8412457-8877702</td>
</tr>
<tr class="even">
<td align="left">YFS.BLOOD.RNAARR/YFS.RERE.wgt.RDat</td>
<td align="right">1</td>
<td align="right">8412457</td>
<td align="right">8877702</td>
<td align="left">YFS Blood</td>
<td align="left">RERE</td>
<td align="right">-5.310078</td>
<td align="right">1.10e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.504000</td>
<td align="left">1:7247335-1:9365199</td>
<td align="left">chr1:8412457-8877702</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000232912.1.wgt.RDat</td>
<td align="right">1</td>
<td align="right">8484705</td>
<td align="right">8494898</td>
<td align="left">GTEx Thyroid</td>
<td align="left">RP5-1115A15.1</td>
<td align="right">-5.175240</td>
<td align="right">2.28e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.269000</td>
<td align="left">1:7247335-1:9365199</td>
<td align="left">chr1:8484705-8494898</td>
</tr>
<tr class="even">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000232912.1.wgt.RDat</td>
<td align="right">1</td>
<td align="right">8484705</td>
<td align="right">8494898</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">RP5-1115A15.1</td>
<td align="right">-4.866386</td>
<td align="right">1.14e-06</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.078700</td>
<td align="left">1:7247335-1:9365199</td>
<td align="left">chr1:8484705-8494898</td>
</tr>
<tr class="odd">
<td align="left">Brain_Nucleus_accumbens_basal_ganglia/Brain_Nucleus_accumbens_basal_ganglia.ENSG00000201448.1.wgt.RDat</td>
<td align="right">1</td>
<td align="right">36884051</td>
<td align="right">36884179</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="left">SNORA63</td>
<td align="right">4.848870</td>
<td align="right">1.24e-06</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.702000</td>
<td align="left">1:34799758-1:37549183</td>
<td align="left">chr1:36884051-36884179</td>
</tr>
<tr class="even">
<td align="left">Brain_Caudate_basal_ganglia/Brain_Caudate_basal_ganglia.ENSG00000172260.9.wgt.RDat</td>
<td align="right">1</td>
<td align="right">71861623</td>
<td align="right">72748417</td>
<td align="left">GTEx Caudate</td>
<td align="left">NEGR1</td>
<td align="right">5.780100</td>
<td align="right">7.47e-09</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000217</td>
<td align="left">1:71684606-1:74326484</td>
<td align="left">chr1:71861623-72748417</td>
</tr>
<tr class="odd">
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000172260.9.wgt.RDat</td>
<td align="right">1</td>
<td align="right">71861623</td>
<td align="right">72748417</td>
<td align="left">GTEx Putamen</td>
<td align="left">NEGR1</td>
<td align="right">5.548510</td>
<td align="right">2.88e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000186</td>
<td align="left">1:71684606-1:74326484</td>
<td align="left">chr1:71861623-72748417</td>
</tr>
<tr class="even">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000172260.9.wgt.RDat</td>
<td align="right">1</td>
<td align="right">71861623</td>
<td align="right">72748417</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">NEGR1</td>
<td align="right">8.760622</td>
<td align="right">1.94e-18</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">1.000000</td>
<td align="left">1:71684606-1:74326484</td>
<td align="left">chr1:71861623-72748417</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000227207.2.wgt.RDat</td>
<td align="right">1</td>
<td align="right">72767155</td>
<td align="right">72767512</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">RPL31P12</td>
<td align="right">-7.785520</td>
<td align="right">6.94e-15</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000325</td>
<td align="left">1:71684606-1:74326484</td>
<td align="left">chr1:72767155-72767512</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000227207.2.wgt.RDat</td>
<td align="right">1</td>
<td align="right">72767155</td>
<td align="right">72767512</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">RPL31P12</td>
<td align="right">-7.708820</td>
<td align="right">1.27e-14</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000410</td>
<td align="left">1:71684606-1:74326484</td>
<td align="left">chr1:72767155-72767512</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000227207.wgt.RDat</td>
<td align="right">1</td>
<td align="right">72767155</td>
<td align="right">72767512</td>
<td align="left">PsychENCODE</td>
<td align="left">RPL31P12</td>
<td align="right">-7.742756</td>
<td align="right">9.73e-15</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000648</td>
<td align="left">1:71684606-1:74326484</td>
<td align="left">chr1:72767155-72767512</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000227740.1.wgt.RDat</td>
<td align="right">1</td>
<td align="right">175873898</td>
<td align="right">175889649</td>
<td align="left">GTEx Thyroid</td>
<td align="left">RP11-318C24.2</td>
<td align="right">-5.027510</td>
<td align="right">4.97e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.286000</td>
<td align="left">1:175089768-1:177433331</td>
<td align="left">chr1:175873898-175889649</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:176085817:176102983:clu_42334.wgt.RDat</td>
<td align="right">1</td>
<td align="right">175913966</td>
<td align="right">176176370</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">RFWD2</td>
<td align="right">-4.958690</td>
<td align="right">7.10e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.184000</td>
<td align="left">1:175089768-1:177433331</td>
<td align="left">chr1:175913966-176176370</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:176085817:176104146:clu_42334.wgt.RDat</td>
<td align="right">1</td>
<td align="right">175913966</td>
<td align="right">176176370</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">RFWD2</td>
<td align="right">5.039850</td>
<td align="right">4.66e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.208000</td>
<td align="left">1:175089768-1:177433331</td>
<td align="left">chr1:175913966-176176370</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:176103036:176104146:clu_42334.wgt.RDat</td>
<td align="right">1</td>
<td align="right">175913966</td>
<td align="right">176176370</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">RFWD2</td>
<td align="right">-5.005960</td>
<td align="right">5.56e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.188000</td>
<td align="left">1:175089768-1:177433331</td>
<td align="left">chr1:175913966-176176370</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:181724533:181725092:clu_42439.wgt.RDat</td>
<td align="right">1</td>
<td align="right">181452685</td>
<td align="right">181775921</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">CACNA1E</td>
<td align="right">-4.989390</td>
<td align="right">6.06e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.518000</td>
<td align="left">1:181144121-1:182755356</td>
<td align="left">chr1:181452685-181775921</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ/CMC.DENND1B.wgt.RDat</td>
<td align="right">1</td>
<td align="right">197473878</td>
<td align="right">197744623</td>
<td align="left">CMC DLPFC</td>
<td align="left">DENND1B</td>
<td align="right">4.848374</td>
<td align="right">1.24e-06</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.725000</td>
<td align="left">1:197311514-1:199239815</td>
<td align="left">chr1:197473878-197744623</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:197684204:197704716:clu_42580.wgt.RDat</td>
<td align="right">1</td>
<td align="right">197473878</td>
<td align="right">197744623</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">DENND1B</td>
<td align="right">-5.421950</td>
<td align="right">5.90e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.165000</td>
<td align="left">1:197311514-1:199239815</td>
<td align="left">chr1:197473878-197744623</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr1:197684204:197741998:clu_42580.wgt.RDat</td>
<td align="right">1</td>
<td align="right">197473878</td>
<td align="right">197744623</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">DENND1B</td>
<td align="right">5.018050</td>
<td align="right">5.22e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.100000</td>
<td align="left">1:197311514-1:199239815</td>
<td align="left">chr1:197473878-197744623</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ/CMC.FANCL.wgt.RDat</td>
<td align="right">2</td>
<td align="right">58386377</td>
<td align="right">58468515</td>
<td align="left">CMC DLPFC</td>
<td align="left">FANCL</td>
<td align="right">-5.183180</td>
<td align="right">2.18e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.007500</td>
<td align="left">2:57429100-2:58296890</td>
<td align="left">chr2:58386377-58468515</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ/CMC.FANCL.wgt.RDat</td>
<td align="right">2</td>
<td align="right">58386377</td>
<td align="right">58468515</td>
<td align="left">CMC DLPFC</td>
<td align="left">FANCL</td>
<td align="right">-5.183180</td>
<td align="right">2.18e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.398000</td>
<td align="left">2:58297664-2:60292000</td>
<td align="left">chr2:58386377-58468515</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr2:58388773:58390001:clu_36265.wgt.RDat</td>
<td align="right">2</td>
<td align="right">58386377</td>
<td align="right">58468515</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">FANCL</td>
<td align="right">4.897476</td>
<td align="right">9.71e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.800000</td>
<td align="left">2:57429100-2:58296890</td>
<td align="left">chr2:58386377-58468515</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr2:58388773:58390001:clu_36265.wgt.RDat</td>
<td align="right">2</td>
<td align="right">58386377</td>
<td align="right">58468515</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">FANCL</td>
<td align="right">4.897476</td>
<td align="right">9.71e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000688</td>
<td align="left">2:58297664-2:60292000</td>
<td align="left">chr2:58386377-58468515</td>
</tr>
<tr class="even">
<td align="left">YFS.BLOOD.RNAARR/YFS.ANKRD44.wgt.RDat</td>
<td align="right">2</td>
<td align="right">197831741</td>
<td align="right">198175897</td>
<td align="left">YFS Blood</td>
<td align="left">ANKRD44</td>
<td align="right">-5.690140</td>
<td align="right">1.27e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.308000</td>
<td align="left">2:198078110-2:199311125</td>
<td align="left">chr2:197831741-198175897</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000115524.11.wgt.RDat</td>
<td align="right">2</td>
<td align="right">198254508</td>
<td align="right">198299815</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">SF3B1</td>
<td align="right">5.214900</td>
<td align="right">1.84e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.126000</td>
<td align="left">2:198078110-2:199311125</td>
<td align="left">chr2:198254508-198299815</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ/CMC.ZNF445.wgt.RDat</td>
<td align="right">3</td>
<td align="right">44481261</td>
<td align="right">44561226</td>
<td align="left">CMC DLPFC</td>
<td align="left">ZNF445</td>
<td align="right">-5.103280</td>
<td align="right">3.34e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.308000</td>
<td align="left">3:42540002-3:45165153</td>
<td align="left">chr3:44481261-44561226</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000109133.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41937137</td>
<td align="right">41962589</td>
<td align="left">PsychENCODE</td>
<td align="left">TMEM33</td>
<td align="right">4.837418</td>
<td align="right">1.32e-06</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.009820</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41937137-41962589</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000182308.5.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41983713</td>
<td align="right">41988476</td>
<td align="left">GTEx Thyroid</td>
<td align="left">DCAF4L1</td>
<td align="right">-5.128000</td>
<td align="right">2.93e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.010600</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41983713-41988476</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000272862.1.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41990758</td>
<td align="right">41991254</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">RP11-814H16.2</td>
<td align="right">5.009600</td>
<td align="right">5.45e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.004000</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41990758-41991254</td>
</tr>
<tr class="even">
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000014824.9.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41992489</td>
<td align="right">42092474</td>
<td align="left">GTEx Amygdala</td>
<td align="left">SLC30A9</td>
<td align="right">-5.253400</td>
<td align="right">1.49e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.034900</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41992489-42092474</td>
</tr>
<tr class="odd">
<td align="left">Brain_Anterior_cingulate_cortex_BA24/Brain_Anterior_cingulate_cortex_BA24.ENSG00000014824.9.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41992489</td>
<td align="right">42092474</td>
<td align="left">GTEx ACC</td>
<td align="left">SLC30A9</td>
<td align="right">-5.001690</td>
<td align="right">5.68e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.052500</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41992489-42092474</td>
</tr>
<tr class="even">
<td align="left">Brain_Caudate_basal_ganglia/Brain_Caudate_basal_ganglia.ENSG00000014824.9.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41992489</td>
<td align="right">42092474</td>
<td align="left">GTEx Caudate</td>
<td align="left">SLC30A9</td>
<td align="right">-4.854800</td>
<td align="right">1.21e-06</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.006000</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41992489-42092474</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000014824.9.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41992489</td>
<td align="right">42092474</td>
<td align="left">GTEx Cortex</td>
<td align="left">SLC30A9</td>
<td align="right">-5.774530</td>
<td align="right">7.72e-09</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.534000</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41992489-42092474</td>
</tr>
<tr class="even">
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000014824.9.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41992489</td>
<td align="right">42092474</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">SLC30A9</td>
<td align="right">-5.085140</td>
<td align="right">3.67e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.029800</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41992489-42092474</td>
</tr>
<tr class="odd">
<td align="left">Brain_Nucleus_accumbens_basal_ganglia/Brain_Nucleus_accumbens_basal_ganglia.ENSG00000014824.9.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41992489</td>
<td align="right">42092474</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="left">SLC30A9</td>
<td align="right">-5.602700</td>
<td align="right">2.11e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.271000</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41992489-42092474</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000014824.wgt.RDat</td>
<td align="right">4</td>
<td align="right">41992489</td>
<td align="right">42092474</td>
<td align="left">PsychENCODE</td>
<td align="left">SLC30A9</td>
<td align="right">-5.259200</td>
<td align="right">1.45e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.002660</td>
<td align="left">4:40202609-4:42213058</td>
<td align="left">chr4:41992489-42092474</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000247828.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564712</td>
<td align="right">87732502</td>
<td align="left">PsychENCODE</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.091010</td>
<td align="right">1.12e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.003040</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564712-87732502</td>
</tr>
<tr class="even">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">5.360090</td>
<td align="right">8.32e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.008360</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="odd">
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Amygdala</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.118500</td>
<td align="right">9.45e-10</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.117000</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="even">
<td align="left">Brain_Anterior_cingulate_cortex_BA24/Brain_Anterior_cingulate_cortex_BA24.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx ACC</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.445500</td>
<td align="right">1.15e-10</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.136000</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="odd">
<td align="left">Brain_Caudate_basal_ganglia/Brain_Caudate_basal_ganglia.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Caudate</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.282167</td>
<td align="right">3.34e-10</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.082400</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.011700</td>
<td align="right">1.84e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.025000</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.053050</td>
<td align="right">1.42e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.088000</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="even">
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Cortex</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.021420</td>
<td align="right">1.73e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.048200</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="odd">
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.720000</td>
<td align="right">1.82e-11</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.126000</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="even">
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">5.875800</td>
<td align="right">4.21e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.057200</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="odd">
<td align="left">Brain_Nucleus_accumbens_basal_ganglia/Brain_Nucleus_accumbens_basal_ganglia.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.010490</td>
<td align="right">1.85e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.056900</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="even">
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Putamen</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.372050</td>
<td align="right">1.87e-10</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.059300</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="odd">
<td align="left">Brain_Substantia_nigra/Brain_Substantia_nigra.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Substantia nigra</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.057270</td>
<td align="right">1.38e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.049800</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Pituitary</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">6.048500</td>
<td align="right">1.46e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.070200</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Thyroid</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">5.889760</td>
<td align="right">3.87e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.035900</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="even">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000247828.3.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87564888</td>
<td align="right">87732502</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">TMEM161B-AS1</td>
<td align="right">5.526440</td>
<td align="right">3.27e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.008200</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87564888-87732502</td>
</tr>
<tr class="odd">
<td align="left">Brain_Substantia_nigra/Brain_Substantia_nigra.ENSG00000271904.1.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87729709</td>
<td align="right">87794514</td>
<td align="left">GTEx Substantia nigra</td>
<td align="left">CTC-498M16.4</td>
<td align="right">5.403610</td>
<td align="right">6.53e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.004490</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87729709-87794514</td>
</tr>
<tr class="even">
<td align="left">Brain_Anterior_cingulate_cortex_BA24/Brain_Anterior_cingulate_cortex_BA24.ENSG00000250377.1.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87988462</td>
<td align="right">87989789</td>
<td align="left">GTEx ACC</td>
<td align="left">CTC-467M3.3</td>
<td align="right">-5.813700</td>
<td align="right">6.11e-09</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000351</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87988462-87989789</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000250377.1.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87988462</td>
<td align="right">87989789</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">CTC-467M3.3</td>
<td align="right">-5.861000</td>
<td align="right">4.60e-09</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.135000</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87988462-87989789</td>
</tr>
<tr class="even">
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000250377.1.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87988462</td>
<td align="right">87989789</td>
<td align="left">GTEx Cortex</td>
<td align="left">CTC-467M3.3</td>
<td align="right">-6.510990</td>
<td align="right">7.47e-11</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.004320</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87988462-87989789</td>
</tr>
<tr class="odd">
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000250377.1.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87988462</td>
<td align="right">87989789</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">CTC-467M3.3</td>
<td align="right">-7.091600</td>
<td align="right">1.33e-12</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.839000</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87988462-87989789</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000250377.wgt.RDat</td>
<td align="right">5</td>
<td align="right">87988462</td>
<td align="right">87989789</td>
<td align="left">PsychENCODE</td>
<td align="left">CTC-467M3.3</td>
<td align="right">-6.097890</td>
<td align="right">1.07e-09</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.021700</td>
<td align="left">5:87390031-5:88891530</td>
<td align="left">chr5:87988462-87989789</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ/CMC.NDUFA2.wgt.RDat</td>
<td align="right">5</td>
<td align="right">140024947</td>
<td align="right">140027370</td>
<td align="left">CMC DLPFC</td>
<td align="left">NDUFA2</td>
<td align="right">5.190020</td>
<td align="right">2.10e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.326000</td>
<td align="left">5:139265072-5:140645971</td>
<td align="left">chr5:140024947-140027370</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000204965.4.wgt.RDat</td>
<td align="right">5</td>
<td align="right">140201222</td>
<td align="right">140203811</td>
<td align="left">GTEx Thyroid</td>
<td align="left">PCDHA5</td>
<td align="right">-5.402970</td>
<td align="right">6.55e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.011400</td>
<td align="left">5:139265072-5:140645971</td>
<td align="left">chr5:140201222-140203811</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000204962.4.wgt.RDat</td>
<td align="right">5</td>
<td align="right">140220907</td>
<td align="right">140223351</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">PCDHA8</td>
<td align="right">-4.980100</td>
<td align="right">6.36e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.087600</td>
<td align="left">5:139265072-5:140645971</td>
<td align="left">chr5:140220907-140223351</td>
</tr>
<tr class="even">
<td align="left">NTR.BLOOD.RNAARR/NTR.HIST1H4D.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26188921</td>
<td align="right">26189323</td>
<td align="left">NTR Blood</td>
<td align="left">HIST1H4D</td>
<td align="right">-4.987600</td>
<td align="right">6.11e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.187000</td>
<td align="left">6:25684587-6:26789628</td>
<td align="left">chr6:26188921-26189323</td>
</tr>
<tr class="odd">
<td align="left">NTR.BLOOD.RNAARR/NTR.BTN3A2.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26365386</td>
<td align="right">26378540</td>
<td align="left">NTR Blood</td>
<td align="left">BTN3A2</td>
<td align="right">5.326600</td>
<td align="right">1.00e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.038900</td>
<td align="left">6:25684587-6:26789628</td>
<td align="left">chr6:26365386-26378540</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000186470.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26365387</td>
<td align="right">26378546</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">BTN3A2</td>
<td align="right">5.188200</td>
<td align="right">2.12e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.016800</td>
<td align="left">6:25684587-6:26789628</td>
<td align="left">chr6:26365387-26378546</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000186470.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26365387</td>
<td align="right">26378546</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">BTN3A2</td>
<td align="right">4.963000</td>
<td align="right">6.96e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.008270</td>
<td align="left">6:25684587-6:26789628</td>
<td align="left">chr6:26365387-26378546</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000186470.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26365387</td>
<td align="right">26378546</td>
<td align="left">GTEx Pituitary</td>
<td align="left">BTN3A2</td>
<td align="right">5.898930</td>
<td align="right">3.66e-09</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.277000</td>
<td align="left">6:25684587-6:26789628</td>
<td align="left">chr6:26365387-26378546</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000186470.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26365387</td>
<td align="right">26378546</td>
<td align="left">GTEx Thyroid</td>
<td align="left">BTN3A2</td>
<td align="right">5.481600</td>
<td align="right">4.22e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.069000</td>
<td align="left">6:25684587-6:26789628</td>
<td align="left">chr6:26365387-26378546</td>
</tr>
<tr class="even">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000186470.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26365387</td>
<td align="right">26378546</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">BTN3A2</td>
<td align="right">5.086960</td>
<td align="right">3.64e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.037800</td>
<td align="left">6:25684587-6:26789628</td>
<td align="left">chr6:26365387-26378546</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000182952.4.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26538633</td>
<td align="right">26546482</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">HMGN4</td>
<td align="right">5.395400</td>
<td align="right">6.84e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.043100</td>
<td align="left">6:25684587-6:26789628</td>
<td align="left">chr6:26538633-26546482</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000182952.4.wgt.RDat</td>
<td align="right">6</td>
<td align="right">26538633</td>
<td align="right">26546482</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">HMGN4</td>
<td align="right">5.395400</td>
<td align="right">6.84e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000403</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:26538633-26546482</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000112812.11.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27215480</td>
<td align="right">27224250</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">PRSS16</td>
<td align="right">-4.891200</td>
<td align="right">1.00e-06</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.001250</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27215480-27224250</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000112812.11.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27215480</td>
<td align="right">27224250</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">PRSS16</td>
<td align="right">-4.947900</td>
<td align="right">7.50e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.001470</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27215480-27224250</td>
</tr>
<tr class="odd">
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000112812.11.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27215480</td>
<td align="right">27224250</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">PRSS16</td>
<td align="right">-5.045000</td>
<td align="right">4.54e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000813</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27215480-27224250</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000112812.11.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27215480</td>
<td align="right">27224250</td>
<td align="left">GTEx Pituitary</td>
<td align="left">PRSS16</td>
<td align="right">-5.916080</td>
<td align="right">3.30e-09</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.077000</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27215480-27224250</td>
</tr>
<tr class="odd">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000112812.11.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27215480</td>
<td align="right">27224250</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">PRSS16</td>
<td align="right">-5.335920</td>
<td align="right">9.51e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.006580</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27215480-27224250</td>
</tr>
<tr class="even">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000204789.3.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27325604</td>
<td align="right">27339304</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">ZNF204P</td>
<td align="right">-5.032700</td>
<td align="right">4.84e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.008260</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27325604-27339304</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000271755.1.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27371789</td>
<td align="right">27374743</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">RP1-153G14.4</td>
<td align="right">5.354000</td>
<td align="right">8.60e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.018300</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27371789-27374743</td>
</tr>
<tr class="even">
<td align="left">Brain_Caudate_basal_ganglia/Brain_Caudate_basal_ganglia.ENSG00000096654.11.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27418522</td>
<td align="right">27440897</td>
<td align="left">GTEx Caudate</td>
<td align="left">ZNF184</td>
<td align="right">-6.325200</td>
<td align="right">2.53e-10</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.111000</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27418522-27440897</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000096654.11.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27418522</td>
<td align="right">27440897</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">ZNF184</td>
<td align="right">-4.952200</td>
<td align="right">7.34e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000884</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27418522-27440897</td>
</tr>
<tr class="even">
<td align="left">NTR.BLOOD.RNAARR/NTR.HIST1H4L.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27840926</td>
<td align="right">27841289</td>
<td align="left">NTR Blood</td>
<td align="left">HIST1H4L</td>
<td align="right">4.870800</td>
<td align="right">1.11e-06</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000344</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:27840926-27841289</td>
</tr>
<tr class="odd">
<td align="left">NTR.BLOOD.RNAARR/NTR.HIST1H4L.wgt.RDat</td>
<td align="right">6</td>
<td align="right">27840926</td>
<td align="right">27841289</td>
<td align="left">NTR Blood</td>
<td align="left">HIST1H4L</td>
<td align="right">4.870800</td>
<td align="right">1.11e-06</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.003600</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:27840926-27841289</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000219891.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28058932</td>
<td align="right">28061442</td>
<td align="left">PsychENCODE</td>
<td align="left">ZSCAN12P1</td>
<td align="right">6.268010</td>
<td align="right">3.66e-10</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.018100</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28058932-28061442</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000219891.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28058932</td>
<td align="right">28061442</td>
<td align="left">PsychENCODE</td>
<td align="left">ZSCAN12P1</td>
<td align="right">6.268010</td>
<td align="right">3.66e-10</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.303000</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28058932-28061442</td>
</tr>
<tr class="even">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000219891.2.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28058932</td>
<td align="right">28061442</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">ZSCAN12P1</td>
<td align="right">-4.936930</td>
<td align="right">7.94e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000506</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28058932-28061442</td>
</tr>
<tr class="odd">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000219891.2.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28058932</td>
<td align="right">28061442</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">ZSCAN12P1</td>
<td align="right">-4.936930</td>
<td align="right">7.94e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.003560</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28058932-28061442</td>
</tr>
<tr class="even">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000219392.1.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28083406</td>
<td align="right">28084329</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">RP1-265C24.5</td>
<td align="right">5.532000</td>
<td align="right">3.16e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.007530</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28083406-28084329</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000219392.1.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28083406</td>
<td align="right">28084329</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">RP1-265C24.5</td>
<td align="right">5.532000</td>
<td align="right">3.16e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.007950</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28083406-28084329</td>
</tr>
<tr class="even">
<td align="left">YFS.BLOOD.RNAARR/YFS.ZSCAN16.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28092338</td>
<td align="right">28097860</td>
<td align="left">YFS Blood</td>
<td align="left">ZSCAN16</td>
<td align="right">-6.109000</td>
<td align="right">1.00e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.052900</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28092338-28097860</td>
</tr>
<tr class="odd">
<td align="left">YFS.BLOOD.RNAARR/YFS.ZSCAN16.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28092338</td>
<td align="right">28097860</td>
<td align="left">YFS Blood</td>
<td align="left">ZSCAN16</td>
<td align="right">-6.109000</td>
<td align="right">1.00e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.087200</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28092338-28097860</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000137185.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28192664</td>
<td align="right">28201260</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">ZSCAN9</td>
<td align="right">-5.307800</td>
<td align="right">1.11e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.011800</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28192664-28201260</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000137185.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28192664</td>
<td align="right">28201260</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">ZSCAN9</td>
<td align="right">-5.307800</td>
<td align="right">1.11e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000608</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28192664-28201260</td>
</tr>
<tr class="even">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000137185.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28192664</td>
<td align="right">28201260</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">ZSCAN9</td>
<td align="right">-6.017000</td>
<td align="right">1.77e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.014300</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28192664-28201260</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000137185.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28192664</td>
<td align="right">28201260</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">ZSCAN9</td>
<td align="right">-6.017000</td>
<td align="right">1.77e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000726</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28192664-28201260</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000137185.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28192664</td>
<td align="right">28201260</td>
<td align="left">GTEx Pituitary</td>
<td align="left">ZSCAN9</td>
<td align="right">-6.159020</td>
<td align="right">7.32e-10</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000231</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28192664-28201260</td>
</tr>
<tr class="odd">
<td align="left">Pituitary/Pituitary.ENSG00000137185.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28192664</td>
<td align="right">28201260</td>
<td align="left">GTEx Pituitary</td>
<td align="left">ZSCAN9</td>
<td align="right">-6.159020</td>
<td align="right">7.32e-10</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.057800</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28192664-28201260</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000189134.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28227098</td>
<td align="right">28228736</td>
<td align="left">PsychENCODE</td>
<td align="left">NKAPL</td>
<td align="right">5.002860</td>
<td align="right">5.65e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000507</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28227098-28228736</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000189134.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28227098</td>
<td align="right">28228736</td>
<td align="left">PsychENCODE</td>
<td align="left">NKAPL</td>
<td align="right">5.002860</td>
<td align="right">5.65e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.021200</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28227098-28228736</td>
</tr>
<tr class="even">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.094600</td>
<td align="right">3.49e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.005860</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="odd">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.094600</td>
<td align="right">3.49e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.003240</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.062800</td>
<td align="right">4.13e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000540</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.062800</td>
<td align="right">4.13e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.025800</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="even">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.198000</td>
<td align="right">2.01e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.546000</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.198000</td>
<td align="right">2.01e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.018300</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="even">
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Putamen</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.739000</td>
<td align="right">9.52e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.005060</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="odd">
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Putamen</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.739000</td>
<td align="right">9.52e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.033100</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Thyroid</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.338400</td>
<td align="right">9.38e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.010900</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Thyroid</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.338400</td>
<td align="right">9.38e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.003450</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="even">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.662330</td>
<td align="right">1.49e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.001660</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="odd">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000197062.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28234788</td>
<td align="right">28245974</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">RP5-874C20.3</td>
<td align="right">5.662330</td>
<td align="right">1.49e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.030100</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28234788-28245974</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000137338.4.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28249314</td>
<td align="right">28270326</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">PGBD1</td>
<td align="right">-6.313100</td>
<td align="right">2.74e-10</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.140000</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28249314-28270326</td>
</tr>
<tr class="odd">
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000235109.3.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28292470</td>
<td align="right">28324048</td>
<td align="left">GTEx Amygdala</td>
<td align="left">ZSCAN31</td>
<td align="right">-5.084150</td>
<td align="right">3.69e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000740</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28292470-28324048</td>
</tr>
<tr class="even">
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000235109.3.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28292470</td>
<td align="right">28324048</td>
<td align="left">GTEx Amygdala</td>
<td align="left">ZSCAN31</td>
<td align="right">-5.084150</td>
<td align="right">3.69e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.004050</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28292470-28324048</td>
</tr>
<tr class="odd">
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000189298.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28317691</td>
<td align="right">28336947</td>
<td align="left">GTEx Amygdala</td>
<td align="left">ZKSCAN3</td>
<td align="right">4.949900</td>
<td align="right">7.43e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.001330</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28317691-28336947</td>
</tr>
<tr class="even">
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000189298.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28317691</td>
<td align="right">28336947</td>
<td align="left">GTEx Amygdala</td>
<td align="left">ZKSCAN3</td>
<td align="right">4.949900</td>
<td align="right">7.43e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.056200</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28317691-28336947</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000189298.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28317691</td>
<td align="right">28336947</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">ZKSCAN3</td>
<td align="right">4.951000</td>
<td align="right">7.37e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000232</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28317691-28336947</td>
</tr>
<tr class="even">
<td align="left">Brain_Hippocampus/Brain_Hippocampus.ENSG00000189298.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28317691</td>
<td align="right">28336947</td>
<td align="left">GTEx Hippocampus</td>
<td align="left">ZKSCAN3</td>
<td align="right">4.951000</td>
<td align="right">7.37e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.004150</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28317691-28336947</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000189298.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28317691</td>
<td align="right">28336947</td>
<td align="left">GTEx Thyroid</td>
<td align="left">ZKSCAN3</td>
<td align="right">6.093300</td>
<td align="right">1.11e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.008910</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28317691-28336947</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000189298.9.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28317691</td>
<td align="right">28336947</td>
<td align="left">GTEx Thyroid</td>
<td align="left">ZKSCAN3</td>
<td align="right">6.093300</td>
<td align="right">1.11e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.037500</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28317691-28336947</td>
</tr>
<tr class="odd">
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000187987.5.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28399707</td>
<td align="right">28411279</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">ZSCAN23</td>
<td align="right">-5.777500</td>
<td align="right">7.58e-09</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.006160</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28399707-28411279</td>
</tr>
<tr class="even">
<td align="left">Brain_Hypothalamus/Brain_Hypothalamus.ENSG00000187987.5.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28399707</td>
<td align="right">28411279</td>
<td align="left">GTEx Hypothalamus</td>
<td align="left">ZSCAN23</td>
<td align="right">-5.777500</td>
<td align="right">7.58e-09</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.067400</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28399707-28411279</td>
</tr>
<tr class="odd">
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000187987.5.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28399707</td>
<td align="right">28411279</td>
<td align="left">GTEx Putamen</td>
<td align="left">ZSCAN23</td>
<td align="right">-4.894000</td>
<td align="right">9.90e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.002130</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28399707-28411279</td>
</tr>
<tr class="even">
<td align="left">Brain_Putamen_basal_ganglia/Brain_Putamen_basal_ganglia.ENSG00000187987.5.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28399707</td>
<td align="right">28411279</td>
<td align="left">GTEx Putamen</td>
<td align="left">ZSCAN23</td>
<td align="right">-4.894000</td>
<td align="right">9.90e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.011300</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28399707-28411279</td>
</tr>
<tr class="odd">
<td align="left">Pituitary/Pituitary.ENSG00000187987.5.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28399707</td>
<td align="right">28411279</td>
<td align="left">GTEx Pituitary</td>
<td align="left">ZSCAN23</td>
<td align="right">-4.953290</td>
<td align="right">7.30e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.001390</td>
<td align="left">6:26791421-6:28017629</td>
<td align="left">chr6:28399707-28411279</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000187987.5.wgt.RDat</td>
<td align="right">6</td>
<td align="right">28399707</td>
<td align="right">28411279</td>
<td align="left">GTEx Pituitary</td>
<td align="left">ZSCAN23</td>
<td align="right">-4.953290</td>
<td align="right">7.30e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.017500</td>
<td align="left">6:28018353-6:28917525</td>
<td align="left">chr6:28399707-28411279</td>
</tr>
<tr class="odd">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000146112.7.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30644166</td>
<td align="right">30655672</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">PPP1R18</td>
<td align="right">4.910200</td>
<td align="right">9.10e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.009620</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30644166-30655672</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:30708575:30709391:clu_26190.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30695485</td>
<td align="right">30710682</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">FLOT1</td>
<td align="right">-5.299700</td>
<td align="right">1.16e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.001230</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30695485-30710682</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:30708575:30709924:clu_26190.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30695485</td>
<td align="right">30710682</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">FLOT1</td>
<td align="right">-5.067100</td>
<td align="right">4.04e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000653</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30695485-30710682</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:30709110:30709391:clu_26190.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30695485</td>
<td align="right">30710682</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">FLOT1</td>
<td align="right">4.936600</td>
<td align="right">7.95e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000801</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30695485-30710682</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000137312.10.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30695486</td>
<td align="right">30710510</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">FLOT1</td>
<td align="right">-5.299000</td>
<td align="right">1.16e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.001200</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30695486-30710510</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000137312.10.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30695486</td>
<td align="right">30710510</td>
<td align="left">GTEx Pituitary</td>
<td align="left">FLOT1</td>
<td align="right">-5.253270</td>
<td align="right">1.49e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.001010</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30695486-30710510</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000137312.10.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30695486</td>
<td align="right">30710510</td>
<td align="left">GTEx Thyroid</td>
<td align="left">FLOT1</td>
<td align="right">-5.557400</td>
<td align="right">2.74e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.002400</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30695486-30710510</td>
</tr>
<tr class="even">
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000137411.12.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30881982</td>
<td align="right">30894236</td>
<td align="left">GTEx Cortex</td>
<td align="left">VARS2</td>
<td align="right">5.922000</td>
<td align="right">3.18e-09</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.030600</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30881982-30894236</td>
</tr>
<tr class="odd">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000137411.12.wgt.RDat</td>
<td align="right">6</td>
<td align="right">30881982</td>
<td align="right">30894236</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">VARS2</td>
<td align="right">6.323130</td>
<td align="right">2.56e-10</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.184000</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:30881982-30894236</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000231402.1.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31255287</td>
<td align="right">31256741</td>
<td align="left">GTEx Pituitary</td>
<td align="left">WASF5P</td>
<td align="right">-5.156240</td>
<td align="right">2.52e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.003470</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:31255287-31256741</td>
</tr>
<tr class="odd">
<td align="left">Pituitary/Pituitary.ENSG00000231402.1.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31255287</td>
<td align="right">31256741</td>
<td align="left">GTEx Pituitary</td>
<td align="left">WASF5P</td>
<td align="right">-5.156240</td>
<td align="right">2.52e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.009760</td>
<td align="left">6:31571218-6:32682664</td>
<td align="left">chr6:31255287-31256741</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000206337.6.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31368479</td>
<td align="right">31445283</td>
<td align="left">GTEx Thyroid</td>
<td align="left">HCP5</td>
<td align="right">6.400800</td>
<td align="right">1.55e-10</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.777000</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:31368479-31445283</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000206337.6.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31368479</td>
<td align="right">31445283</td>
<td align="left">GTEx Thyroid</td>
<td align="left">HCP5</td>
<td align="right">6.400800</td>
<td align="right">1.55e-10</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.061900</td>
<td align="left">6:31571218-6:32682664</td>
<td align="left">chr6:31368479-31445283</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000204516.5.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31462658</td>
<td align="right">31478901</td>
<td align="left">GTEx Thyroid</td>
<td align="left">MICB</td>
<td align="right">-5.557000</td>
<td align="right">2.74e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.091400</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:31462658-31478901</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000204516.5.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31462658</td>
<td align="right">31478901</td>
<td align="left">GTEx Thyroid</td>
<td align="left">MICB</td>
<td align="right">-5.557000</td>
<td align="right">2.74e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.000364</td>
<td align="left">6:31571218-6:32682664</td>
<td align="left">chr6:31462658-31478901</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:31619553:31620177:clu_26248.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31606805</td>
<td align="right">31620482</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">BAG6</td>
<td align="right">-5.580000</td>
<td align="right">2.40e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.000540</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:31606805-31620482</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:31619553:31620177:clu_26248.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31606805</td>
<td align="right">31620482</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">BAG6</td>
<td align="right">-5.580000</td>
<td align="right">2.40e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.221000</td>
<td align="left">6:31571218-6:32682664</td>
<td align="left">chr6:31606805-31620482</td>
</tr>
<tr class="even">
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000213722.4.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31694815</td>
<td align="right">31698357</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">DDAH2</td>
<td align="right">5.409500</td>
<td align="right">6.32e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.006230</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:31694815-31698357</td>
</tr>
<tr class="odd">
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000213722.4.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31694815</td>
<td align="right">31698357</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">DDAH2</td>
<td align="right">5.409500</td>
<td align="right">6.32e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.012700</td>
<td align="left">6:31571218-6:32682664</td>
<td align="left">chr6:31694815-31698357</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ/CMC.DDAH2.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31694816</td>
<td align="right">31698039</td>
<td align="left">CMC DLPFC</td>
<td align="left">DDAH2</td>
<td align="right">5.344500</td>
<td align="right">9.07e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.003160</td>
<td align="left">6:30798168-6:31571218</td>
<td align="left">chr6:31694816-31698039</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ/CMC.DDAH2.wgt.RDat</td>
<td align="right">6</td>
<td align="right">31694816</td>
<td align="right">31698039</td>
<td align="left">CMC DLPFC</td>
<td align="left">DDAH2</td>
<td align="right">5.344500</td>
<td align="right">9.07e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.016200</td>
<td align="left">6:31571218-6:32682664</td>
<td align="left">chr6:31694816-31698039</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr6:99831006:99831574:clu_27287.wgt.RDat</td>
<td align="right">6</td>
<td align="right">99817347</td>
<td align="right">99842082</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">COQ3</td>
<td align="right">5.146560</td>
<td align="right">2.65e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.974000</td>
<td align="left">6:97842747-6:100629728</td>
<td align="left">chr6:99817347-99842082</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ/CMC.LIN28B.wgt.RDat</td>
<td align="right">6</td>
<td align="right">105404922</td>
<td align="right">105531207</td>
<td align="left">CMC DLPFC</td>
<td align="left">LIN28B</td>
<td align="right">-5.232050</td>
<td align="right">1.68e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.127000</td>
<td align="left">6:103983395-6:106056733</td>
<td align="left">chr6:105404922-105531207</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000187772.wgt.RDat</td>
<td align="right">6</td>
<td align="right">105404923</td>
<td align="right">105531207</td>
<td align="left">PsychENCODE</td>
<td align="left">LIN28B</td>
<td align="right">-5.105689</td>
<td align="right">3.30e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.092400</td>
<td align="left">6:103983395-6:106056733</td>
<td align="left">chr6:105404923-105531207</td>
</tr>
<tr class="odd">
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000203808.6.wgt.RDat</td>
<td align="right">6</td>
<td align="right">105584224</td>
<td align="right">105617820</td>
<td align="left">GTEx Amygdala</td>
<td align="left">BVES-AS1</td>
<td align="right">-5.578300</td>
<td align="right">2.43e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.742000</td>
<td align="left">6:103983395-6:106056733</td>
<td align="left">chr6:105584224-105617820</td>
</tr>
<tr class="even">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000106460.14.wgt.RDat</td>
<td align="right">7</td>
<td align="right">12250867</td>
<td align="right">12282993</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">TMEM106B</td>
<td align="right">5.505026</td>
<td align="right">3.69e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.062900</td>
<td align="left">7:11299198-7:12635461</td>
<td align="left">chr7:12250867-12282993</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000106460.wgt.RDat</td>
<td align="right">7</td>
<td align="right">12250867</td>
<td align="right">12282993</td>
<td align="left">PsychENCODE</td>
<td align="left">TMEM106B</td>
<td align="right">-5.790690</td>
<td align="right">7.01e-09</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.259000</td>
<td align="left">7:11299198-7:12635461</td>
<td align="left">chr7:12250867-12282993</td>
</tr>
<tr class="even">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000106460.14.wgt.RDat</td>
<td align="right">7</td>
<td align="right">12250867</td>
<td align="right">12282993</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">TMEM106B</td>
<td align="right">5.531000</td>
<td align="right">3.18e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.614000</td>
<td align="left">7:11299198-7:12635461</td>
<td align="left">chr7:12250867-12282993</td>
</tr>
<tr class="odd">
<td align="left">YFS.BLOOD.RNAARR/YFS.TMEM106B.wgt.RDat</td>
<td align="right">7</td>
<td align="right">12250867</td>
<td align="right">12276886</td>
<td align="left">YFS Blood</td>
<td align="left">TMEM106B</td>
<td align="right">5.373600</td>
<td align="right">7.72e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.057400</td>
<td align="left">7:11299198-7:12635461</td>
<td align="left">chr7:12250867-12276886</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000070882.8.wgt.RDat</td>
<td align="right">7</td>
<td align="right">24836158</td>
<td align="right">25021253</td>
<td align="left">GTEx Pituitary</td>
<td align="left">OSBPL3</td>
<td align="right">-5.622890</td>
<td align="right">1.88e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.983000</td>
<td align="left">7:23471523-7:25077097</td>
<td align="left">chr7:24836158-25021253</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ/CMC.PXDNL.wgt.RDat</td>
<td align="right">8</td>
<td align="right">52232136</td>
<td align="right">52722005</td>
<td align="left">CMC DLPFC</td>
<td align="left">PXDNL</td>
<td align="right">5.887460</td>
<td align="right">3.92e-09</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.998000</td>
<td align="left">8:50082470-8:53302930</td>
<td align="left">chr8:52232136-52722005</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000251396.2.wgt.RDat</td>
<td align="right">8</td>
<td align="right">61297147</td>
<td align="right">61429354</td>
<td align="left">GTEx Thyroid</td>
<td align="left">RP11-163N6.2</td>
<td align="right">-5.336530</td>
<td align="right">9.47e-08</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.923000</td>
<td align="left">8:59728421-8:62328644</td>
<td align="left">chr8:61297147-61429354</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000234881.wgt.RDat</td>
<td align="right">9</td>
<td align="right">126605315</td>
<td align="right">126605965</td>
<td align="left">PsychENCODE</td>
<td align="left">PIGFP2</td>
<td align="right">-5.305600</td>
<td align="right">1.12e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.944000</td>
<td align="left">9:124871322-9:126971353</td>
<td align="left">chr9:126605315-126605965</td>
</tr>
<tr class="even">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000149115.9.wgt.RDat</td>
<td align="right">11</td>
<td align="right">57067112</td>
<td align="right">57092426</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">TNKS1BP1</td>
<td align="right">4.922610</td>
<td align="right">8.54e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.214000</td>
<td align="left">11:55082693-11:58457495</td>
<td align="left">chr11:57067112-57092426</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000254602.1.wgt.RDat</td>
<td align="right">11</td>
<td align="right">57405497</td>
<td align="right">57420263</td>
<td align="left">GTEx Thyroid</td>
<td align="left">AP000662.4</td>
<td align="right">-4.980256</td>
<td align="right">6.35e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.205000</td>
<td align="left">11:55082693-11:58457495</td>
<td align="left">chr11:57405497-57420263</td>
</tr>
<tr class="even">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000172409.5.wgt.RDat</td>
<td align="right">11</td>
<td align="right">57424488</td>
<td align="right">57429340</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">CLP1</td>
<td align="right">5.195860</td>
<td align="right">2.04e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.449000</td>
<td align="left">11:55082693-11:58457495</td>
<td align="left">chr11:57424488-57429340</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000134825.wgt.RDat</td>
<td align="right">11</td>
<td align="right">61535973</td>
<td align="right">61560274</td>
<td align="left">PsychENCODE</td>
<td align="left">TMEM258</td>
<td align="right">5.021730</td>
<td align="right">5.12e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.223000</td>
<td align="left">11:58780549-11:62223771</td>
<td align="left">chr11:61535973-61560274</td>
</tr>
<tr class="even">
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000149295.9.wgt.RDat</td>
<td align="right">11</td>
<td align="right">113280318</td>
<td align="right">113346111</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">DRD2</td>
<td align="right">-5.073787</td>
<td align="right">3.90e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.967000</td>
<td align="left">11:112459488-11:114256749</td>
<td align="left">chr11:113280318-113346111</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ/CMC.OLFM4.wgt.RDat</td>
<td align="right">13</td>
<td align="right">53602875</td>
<td align="right">53626196</td>
<td align="left">CMC DLPFC</td>
<td align="left">OLFM4</td>
<td align="right">5.091290</td>
<td align="right">3.56e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.992000</td>
<td align="left">13:53339645-13:54682393</td>
<td align="left">chr13:53602875-53626196</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000258636.1.wgt.RDat</td>
<td align="right">14</td>
<td align="right">42057064</td>
<td align="right">42074059</td>
<td align="left">GTEx Thyroid</td>
<td align="left">CTD-2298J14.2</td>
<td align="right">-5.678860</td>
<td align="right">1.36e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.402000</td>
<td align="left">14:41615719-14:43137256</td>
<td align="left">chr14:42057064-42074059</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellar_Hemisphere/Brain_Cerebellar_Hemisphere.ENSG00000165379.9.wgt.RDat</td>
<td align="right">14</td>
<td align="right">42076773</td>
<td align="right">42373752</td>
<td align="left">GTEx Cerebellar Hemisphere</td>
<td align="left">LRFN5</td>
<td align="right">5.423400</td>
<td align="right">5.85e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.093300</td>
<td align="left">14:41615719-14:43137256</td>
<td align="left">chr14:42076773-42373752</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000165379.9.wgt.RDat</td>
<td align="right">14</td>
<td align="right">42076773</td>
<td align="right">42373752</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">LRFN5</td>
<td align="right">5.597540</td>
<td align="right">2.17e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.504000</td>
<td align="left">14:41615719-14:43137256</td>
<td align="left">chr14:42076773-42373752</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000050130.13.wgt.RDat</td>
<td align="right">14</td>
<td align="right">59951161</td>
<td align="right">59971429</td>
<td align="left">GTEx Thyroid</td>
<td align="left">JKAMP</td>
<td align="right">-5.125100</td>
<td align="right">2.97e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.139000</td>
<td align="left">14:59448336-14:61680201</td>
<td align="left">chr14:59951161-59971429</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000151838.7.wgt.RDat</td>
<td align="right">14</td>
<td align="right">59971779</td>
<td align="right">60043549</td>
<td align="left">GTEx Thyroid</td>
<td align="left">CCDC175</td>
<td align="right">-5.478850</td>
<td align="right">4.28e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.609000</td>
<td align="left">14:59448336-14:61680201</td>
<td align="left">chr14:59971779-60043549</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr14:60074210:60097164:clu_16682.wgt.RDat</td>
<td align="right">14</td>
<td align="right">60062693</td>
<td align="right">60337557</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">RTN1</td>
<td align="right">-4.874920</td>
<td align="right">1.09e-06</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.039700</td>
<td align="left">14:59448336-14:61680201</td>
<td align="left">chr14:60062693-60337557</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000139970.12.wgt.RDat</td>
<td align="right">14</td>
<td align="right">60062695</td>
<td align="right">60337684</td>
<td align="left">GTEx Thyroid</td>
<td align="left">RTN1</td>
<td align="right">-5.348450</td>
<td align="right">8.87e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.189000</td>
<td align="left">14:59448336-14:61680201</td>
<td align="left">chr14:60062695-60337684</td>
</tr>
<tr class="odd">
<td align="left">NTR.BLOOD.RNAARR/NTR.SYNE2.wgt.RDat</td>
<td align="right">14</td>
<td align="right">64319682</td>
<td align="right">64693151</td>
<td align="left">NTR Blood</td>
<td align="left">SYNE2</td>
<td align="right">5.609528</td>
<td align="right">2.03e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.253000</td>
<td align="left">14:63790015-14:65220117</td>
<td align="left">chr14:64319682-64693151</td>
</tr>
<tr class="even">
<td align="left">Pituitary/Pituitary.ENSG00000140009.14.wgt.RDat</td>
<td align="right">14</td>
<td align="right">64550950</td>
<td align="right">64770377</td>
<td align="left">GTEx Pituitary</td>
<td align="left">ESR2</td>
<td align="right">-5.982300</td>
<td align="right">2.20e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.593000</td>
<td align="left">14:63790015-14:65220117</td>
<td align="left">chr14:64550950-64770377</td>
</tr>
<tr class="odd">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000140009.14.wgt.RDat</td>
<td align="right">14</td>
<td align="right">64550950</td>
<td align="right">64770377</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">ESR2</td>
<td align="right">-5.655371</td>
<td align="right">1.56e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">0</td>
<td align="right">0.157000</td>
<td align="left">14:63790015-14:65220117</td>
<td align="left">chr14:64550950-64770377</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000119682.wgt.RDat</td>
<td align="right">14</td>
<td align="right">75120140</td>
<td align="right">75179818</td>
<td align="left">PsychENCODE</td>
<td align="left">AREL1</td>
<td align="right">-5.015110</td>
<td align="right">5.30e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.018200</td>
<td align="left">14:72889754-14:76444767</td>
<td align="left">chr14:75120140-75179818</td>
</tr>
<tr class="odd">
<td align="left">Thyroid/Thyroid.ENSG00000119608.8.wgt.RDat</td>
<td align="right">14</td>
<td align="right">75319736</td>
<td align="right">75330537</td>
<td align="left">GTEx Thyroid</td>
<td align="left">PROX2</td>
<td align="right">-5.758100</td>
<td align="right">8.51e-09</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.718000</td>
<td align="left">14:72889754-14:76444767</td>
<td align="left">chr14:75319736-75330537</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ/CMC.DLST.wgt.RDat</td>
<td align="right">14</td>
<td align="right">75348593</td>
<td align="right">75370450</td>
<td align="left">CMC DLPFC</td>
<td align="left">DLST</td>
<td align="right">4.981400</td>
<td align="right">6.31e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.019200</td>
<td align="left">14:72889754-14:76444767</td>
<td align="left">chr14:75348593-75370450</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000119689.wgt.RDat</td>
<td align="right">14</td>
<td align="right">75348594</td>
<td align="right">75370448</td>
<td align="left">PsychENCODE</td>
<td align="left">DLST</td>
<td align="right">5.089700</td>
<td align="right">3.59e-07</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.210000</td>
<td align="left">14:72889754-14:76444767</td>
<td align="left">chr14:75348594-75370448</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr14:75375893:75377951:clu_16995.wgt.RDat</td>
<td align="right">14</td>
<td align="right">75370656</td>
<td align="right">75389188</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">RPS6KL1</td>
<td align="right">-5.023810</td>
<td align="right">5.07e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.032000</td>
<td align="left">14:72889754-14:76444767</td>
<td align="left">chr14:75370656-75389188</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000198208.wgt.RDat</td>
<td align="right">14</td>
<td align="right">75370657</td>
<td align="right">75390099</td>
<td align="left">PsychENCODE</td>
<td align="left">RPS6KL1</td>
<td align="right">-4.952550</td>
<td align="right">7.32e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">0</td>
<td align="right">0.012900</td>
<td align="left">14:72889754-14:76444767</td>
<td align="left">chr14:75370657-75390099</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000244691.wgt.RDat</td>
<td align="right">14</td>
<td align="right">103878456</td>
<td align="right">103879098</td>
<td align="left">PsychENCODE</td>
<td align="left">RP11-600F24.2</td>
<td align="right">5.185660</td>
<td align="right">2.15e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.039200</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:103878456-103879098</td>
</tr>
<tr class="odd">
<td align="left">YFS.BLOOD.RNAARR/YFS.CKB.wgt.RDat</td>
<td align="right">14</td>
<td align="right">103985996</td>
<td align="right">103989448</td>
<td align="left">YFS Blood</td>
<td align="left">CKB</td>
<td align="right">5.346000</td>
<td align="right">8.99e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.145000</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:103985996-103989448</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ/CMC.TRMT61A.wgt.RDat</td>
<td align="right">14</td>
<td align="right">103995508</td>
<td align="right">104003410</td>
<td align="left">CMC DLPFC</td>
<td align="left">TRMT61A</td>
<td align="right">5.051300</td>
<td align="right">4.39e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.039000</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:103995508-104003410</td>
</tr>
<tr class="odd">
<td align="left">Whole_Blood/Whole_Blood.ENSG00000166166.8.wgt.RDat</td>
<td align="right">14</td>
<td align="right">103995521</td>
<td align="right">104003410</td>
<td align="left">GTEx Whole Blood</td>
<td align="left">TRMT61A</td>
<td align="right">4.977593</td>
<td align="right">6.44e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.025300</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:103995521-104003410</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000258851.1.wgt.RDat</td>
<td align="right">14</td>
<td align="right">104019758</td>
<td align="right">104028214</td>
<td align="left">GTEx Thyroid</td>
<td align="left">RP11-894P9.2</td>
<td align="right">-5.462560</td>
<td align="right">4.69e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.232000</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:104019758-104028214</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000270108.wgt.RDat</td>
<td align="right">14</td>
<td align="right">104153913</td>
<td align="right">104154464</td>
<td align="left">PsychENCODE</td>
<td align="left">RP11-73M18.6</td>
<td align="right">5.031320</td>
<td align="right">4.87e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.180000</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:104153913-104154464</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000269940.wgt.RDat</td>
<td align="right">14</td>
<td align="right">104160897</td>
<td align="right">104161507</td>
<td align="left">PsychENCODE</td>
<td align="left">RP11-73M18.7</td>
<td align="right">4.856130</td>
<td align="right">1.20e-06</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.015900</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:104160897-104161507</td>
</tr>
<tr class="odd">
<td align="left">Brain_Amygdala/Brain_Amygdala.ENSG00000269958.1.wgt.RDat</td>
<td align="right">14</td>
<td align="right">104162690</td>
<td align="right">104163500</td>
<td align="left">GTEx Amygdala</td>
<td align="left">RP11-73M18.8</td>
<td align="right">5.142000</td>
<td align="right">2.72e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.077900</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:104162690-104163500</td>
</tr>
<tr class="even">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000224997.1.wgt.RDat</td>
<td align="right">14</td>
<td align="right">104177607</td>
<td align="right">104179149</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">AL049840.1</td>
<td align="right">5.029540</td>
<td align="right">4.92e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.031600</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:104177607-104179149</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000224997.1.wgt.RDat</td>
<td align="right">14</td>
<td align="right">104177607</td>
<td align="right">104179149</td>
<td align="left">GTEx Cortex</td>
<td align="left">AL049840.1</td>
<td align="right">5.143620</td>
<td align="right">2.69e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.034400</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:104177607-104179149</td>
</tr>
<tr class="even">
<td align="left">Brain_Cortex/Brain_Cortex.ENSG00000269963.1.wgt.RDat</td>
<td align="right">14</td>
<td align="right">104179904</td>
<td align="right">104180441</td>
<td align="left">GTEx Cortex</td>
<td align="left">RP11-73M18.9</td>
<td align="right">4.977330</td>
<td align="right">6.45e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.032000</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:104179904-104180441</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000269963.wgt.RDat</td>
<td align="right">14</td>
<td align="right">104179904</td>
<td align="right">104180586</td>
<td align="left">PsychENCODE</td>
<td align="left">RP11-73M18.9</td>
<td align="right">4.830100</td>
<td align="right">1.36e-06</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.011500</td>
<td align="left">14:103012102-14:105001723</td>
<td align="left">chr14:104179904-104180586</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000118557.wgt.RDat</td>
<td align="right">16</td>
<td align="right">72146056</td>
<td align="right">72210777</td>
<td align="left">PsychENCODE</td>
<td align="left">PMFBP1</td>
<td align="right">-5.160620</td>
<td align="right">2.46e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.964000</td>
<td align="left">16:71054116-16:72935146</td>
<td align="left">chr16:72146056-72210777</td>
</tr>
<tr class="odd">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000196535.10.wgt.RDat</td>
<td align="right">17</td>
<td align="right">27400528</td>
<td align="right">27507430</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">MYO18A</td>
<td align="right">-5.128570</td>
<td align="right">2.92e-07</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.184000</td>
<td align="left">17:27334244-17:29786491</td>
<td align="left">chr17:27400528-27507430</td>
</tr>
<tr class="even">
<td align="left">Adrenal_Gland/Adrenal_Gland.ENSG00000221995.4.wgt.RDat</td>
<td align="right">17</td>
<td align="right">27401933</td>
<td align="right">27405875</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="left">TIAF1</td>
<td align="right">-5.361200</td>
<td align="right">8.27e-08</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.746000</td>
<td align="left">17:27334244-17:29786491</td>
<td align="left">chr17:27401933-27405875</td>
</tr>
<tr class="odd">
<td align="left">PEC_TWAS_weights/ENSG00000264754.wgt.RDat</td>
<td align="right">17</td>
<td align="right">65520597</td>
<td align="right">65521538</td>
<td align="left">PsychENCODE</td>
<td align="left">CTD-2653B5.1</td>
<td align="right">5.105730</td>
<td align="right">3.30e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="left">NA</td>
<td align="left">chr17:65520597-65521538</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000041353.wgt.RDat</td>
<td align="right">18</td>
<td align="right">52385091</td>
<td align="right">52562747</td>
<td align="left">PsychENCODE</td>
<td align="left">RAB27B</td>
<td align="right">5.012900</td>
<td align="right">5.36e-07</td>
<td align="left">Yes</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.469000</td>
<td align="left">18:51554430-18:55213838</td>
<td align="left">chr18:52385091-52562747</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr18:52385372:52544798:clu_20555.wgt.RDat</td>
<td align="right">18</td>
<td align="right">52495707</td>
<td align="right">52562747</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">RAB27B</td>
<td align="right">4.843190</td>
<td align="right">1.28e-06</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.122000</td>
<td align="left">18:51554430-18:55213838</td>
<td align="left">chr18:52495707-52562747</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ/CMC.DDX27.wgt.RDat</td>
<td align="right">20</td>
<td align="right">47835831</td>
<td align="right">47860614</td>
<td align="left">CMC DLPFC</td>
<td align="left">DDX27</td>
<td align="right">4.836260</td>
<td align="right">1.32e-06</td>
<td align="left">Yes</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.353000</td>
<td align="left">20:47199980-20:49239658</td>
<td align="left">chr20:47835831-47860614</td>
</tr>
<tr class="odd">
<td align="left">Brain_Nucleus_accumbens_basal_ganglia/Brain_Nucleus_accumbens_basal_ganglia.ENSG00000100372.10.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41165634</td>
<td align="right">41215403</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="left">SLC25A17</td>
<td align="right">5.076990</td>
<td align="right">3.83e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.017600</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41165634-41215403</td>
</tr>
<tr class="even">
<td align="left">Thyroid/Thyroid.ENSG00000100372.10.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41165634</td>
<td align="right">41215403</td>
<td align="left">GTEx Thyroid</td>
<td align="left">SLC25A17</td>
<td align="right">4.896100</td>
<td align="right">9.78e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.018500</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41165634-41215403</td>
</tr>
<tr class="odd">
<td align="left">Brain_Frontal_Cortex_BA9/Brain_Frontal_Cortex_BA9.ENSG00000196236.8.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41253088</td>
<td align="right">41351450</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="left">XPNPEP3</td>
<td align="right">4.951000</td>
<td align="right">7.38e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.003980</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41253088-41351450</td>
</tr>
<tr class="even">
<td align="left">CMC.BRAIN.RNASEQ/CMC.XPNPEP3.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41258260</td>
<td align="right">41363888</td>
<td align="left">CMC DLPFC</td>
<td align="left">XPNPEP3</td>
<td align="right">5.110000</td>
<td align="right">3.21e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.022800</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41258260-41363888</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000100393.9.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41487790</td>
<td align="right">41576081</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">EP300</td>
<td align="right">5.493900</td>
<td align="right">3.93e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.319000</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41487790-41576081</td>
</tr>
<tr class="even">
<td align="left">YFS.BLOOD.RNAARR/YFS.EP300.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41487790</td>
<td align="right">41576081</td>
<td align="left">YFS Blood</td>
<td align="left">EP300</td>
<td align="right">5.059100</td>
<td align="right">4.21e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.058800</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41487790-41576081</td>
</tr>
<tr class="odd">
<td align="left">CMC.BRAIN.RNASEQ_SPLICING/CMC.chr22:41657584:41664101:clu_21526.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41641614</td>
<td align="right">41682216</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="left">RANGAP1</td>
<td align="right">5.240100</td>
<td align="right">1.61e-07</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.150000</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41641614-41682216</td>
</tr>
<tr class="even">
<td align="left">PEC_TWAS_weights/ENSG00000100401.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41641615</td>
<td align="right">41682255</td>
<td align="left">PsychENCODE</td>
<td align="left">RANGAP1</td>
<td align="right">-5.575273</td>
<td align="right">2.47e-08</td>
<td align="left">No</td>
<td align="left">FALSE</td>
<td align="right">1</td>
<td align="right">0.011900</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41641615-41682255</td>
</tr>
<tr class="odd">
<td align="left">Brain_Cerebellum/Brain_Cerebellum.ENSG00000100403.10.wgt.RDat</td>
<td align="right">22</td>
<td align="right">41697526</td>
<td align="right">41756151</td>
<td align="left">GTEx Cerebellum</td>
<td align="left">ZC3H7B</td>
<td align="right">5.729100</td>
<td align="right">1.01e-08</td>
<td align="left">No</td>
<td align="left">TRUE</td>
<td align="right">1</td>
<td align="right">0.382000</td>
<td align="left">22:40545828-22:42690262</td>
<td align="left">chr22:41697526-41756151</td>
</tr>
</tbody>
</table>
</details>
<hr />
</div>
<div id="twas-gsea" class="section level2">
<h2><span class="header-section-number">3.8</span> TWAS-GSEA</h2>
<details>
<p><summary>Combine the predicted expression files for FUSION and PsychENCODE</summary></p>
<pre class="r"><code>library(data.table)

FUSION&lt;-fread(cmd=&#39;zcat /scratch/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Predicted_expression/FUSION_1KG/FUSION_1KG_Expr_AllSets.csv.gz&#39;)
PsychENCODE&lt;-fread(cmd=&#39;zcat /scratch/groups/biomarkers-brc-mh/TWAS_resource/PsychEncode/Predicted_expression/FeaturePredictions.csv.gz&#39;)

both&lt;-merge(FUSION, PsychENCODE, by=c(&#39;FID&#39;,&#39;IID&#39;))

fwrite(both, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv&#39;, row.names=F, quote=F)</code></pre>
<pre class="bash"><code>gzip /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv</code></pre>
</details>
<details>
<p><summary>TWAS-GSEA: All tissues</summary></p>
<pre class="bash"><code># Using TWAS from all PANELs, removing duplicate genes.
sbatch -p brc,shared --mem=60G -n 3 /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --gmt_file /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/candidate.gmt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --n_cores 3 \
  --self_contained F \
  --min_r2 0.05 \
  --competitive T \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/MDD_TWAS_GSEA_Candidate_Wray

# hypothesis-free analysis
sbatch -p brc,shared --mem=60G -n 3 /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --gmt_file /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/hypofree.gmt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --self_contained F \
  --min_r2 0.05 \
  --n_cores 3 \
  --competitive T \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/MDD_TWAS_GSEA_Hypo_free

# brainspan
sbatch -p brc,shared --mem=60G -n 3 /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --prop_file /mnt/lustre/users/k1806347/Data/Gene_properties/Gusev_DPFC_BRAINSPAN/DFC_RIN_CLEANED.DE_ZScores.symbol.txt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --self_contained F \
  --min_r2 0.05 \
  --linear_p_thresh 1 \
  --n_cores 3 \
  --competitive T \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/MDD_TWAS_GSEA_brainspan
</code></pre>
</details>
<details>
<p><summary>TWAS-GSEA: Tissue groups</summary></p>
<pre class="r"><code>library(data.table)
res&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt&#39;)
Brain_res&lt;-res[grepl(&#39;Brain|BRAIN|PsychENCODE&#39;, res$PANEL),]
HPA_res&lt;-res[grepl(&#39;Adrenal|Pituitary|Hypothalamus&#39;, res$PANEL),]
HPT_res&lt;-res[grepl(&#39;Thyroid|Pituitary|Hypothalamus&#39;, res$PANEL),]
BLOOD_res&lt;-res[grepl(&#39;BLOOD&#39;, res$PANEL),]

write.table(Brain_res, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanelSet/MDD_TWAS_BRAIN.GW&#39;, row.names=F, col.names=T, quote=F)
write.table(HPA_res, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanelSet/MDD_TWAS_HPA.GW&#39;, row.names=F, col.names=T, quote=F)
write.table(HPT_res, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanelSet/MDD_TWAS_HPT.GW&#39;, row.names=F, col.names=T, quote=F)
write.table(BLOOD_res, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanelSet/MDD_TWAS_BLOOD.GW&#39;, row.names=F, col.names=T, quote=F)</code></pre>
<pre class="bash"><code># Candidate
for set in $(echo BRAIN HPA HPT BLOOD);do
sbatch -p brc,shared --mem=20G /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanelSet/MDD_TWAS_${set}.GW \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --gmt_file /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/candidate.gmt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --n_cores 1 \
  --self_contained F \
  --min_r2 0.05 \
  --competitive T \
  --linear_p_thresh 1 \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet/${set}_GSEA_Candidate_Wray
done

# Hypo-free
for set in $(echo BRAIN HPA HPT BLOOD);do
sbatch -p brc,shared --mem=20G /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanelSet/MDD_TWAS_${set}.GW \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --gmt_file /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/hypofree.gmt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --n_cores 1 \
  --self_contained F \
  --min_r2 0.05 \
  --competitive T \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet/${set}_GSEA_Hypo_free
done

# brainspan
for set in $(echo BRAIN HPA HPT BLOOD);do
sbatch -p brc,shared --mem=20G -n 1 /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanelSet/MDD_TWAS_${set}.GW \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --prop_file /mnt/lustre/users/k1806347/Data/Gene_properties/Gusev_DPFC_BRAINSPAN/DFC_RIN_CLEANED.DE_ZScores.symbol.txt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --self_contained F \
  --min_r2 0.05 \
  --n_cores 3 \
  --linear_p_thresh 1 \
  --competitive T \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet/${set}_brainspan

done
</code></pre>
</details>
<details>
<p><summary>TWAS-GSEA: Tissue-specific</summary></p>
<pre class="r"><code>library(data.table)
res&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues.txt&#39;)

for(i in unique(res$PANEL)){
  write.table(res[res$PANEL == i,], paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanel/MDD_TWAS_&#39;,i,&#39;.GW&#39;), row.names=F, col.names=T, quote=F)
}</code></pre>
<pre class="bash"><code>for tissue in $(cat /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWASweights_list_withPsychENCODE.txt); do

sbatch -p brc,shared --mem=20G /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanel/MDD_TWAS_${tissue}.GW \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --gmt_file /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/candidate.gmt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --n_cores 1 \
  --self_contained F \
  --linear_p_thresh 1 \
  --competitive T \
  --min_r2 0.05 \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/${tissue}_GSEA_Candidate_Wray

done

for tissue in $(cat /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWASweights_list_withPsychENCODE.txt); do

sbatch -p brc,shared --mem=20G /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanel/MDD_TWAS_${tissue}.GW \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --gmt_file /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/hypofree.gmt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --self_contained F \
  --competitive T \
  --min_r2 0.05 \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/${tissue}_GSEA_Hypo_free

done

# brainspan
for tissue in $(cat /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWASweights_list_withPsychENCODE.txt); do

sbatch -p brc,shared --mem=20G -n 1 /users/k1806347/brc_scratch/Software/Rscript.sh /mnt/lustre/groups/biomarkers-brc-mh/TWAS_resource/FUSION/Scripts/Git/opain/TWAS-GSEA/TWAS-GSEA.V1.2.R \
  --twas_results /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/ByPanel/MDD_TWAS_${tissue}.GW \
  --pos /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/MDD_TWAS.pos \
  --prop_file /mnt/lustre/users/k1806347/Data/Gene_properties/Gusev_DPFC_BRAINSPAN/DFC_RIN_CLEANED.DE_ZScores.symbol.txt \
  --qqplot F \
  --use_alt_id ID \
  --expression_ref /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/FUSION_PsychENCODE_FeaturePredictions.csv.gz \
  --self_contained F \
  --min_r2 0.05 \
  --linear_p_thresh 1 \
  --n_cores 3 \
  --competitive T \
  --covar GeneLength,NSNP \
  --output /users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/${tissue}_brainspan

done

# Note. The individual tissue analyses return more significant findings. Instead of aggregating tissues, perhaps meta-analysis of per tissue results would be more effective.</code></pre>
</details>
<details>
<p><summary>Tabulate the results</summary></p>
<pre class="r"><code>library(data.table)

#####
# AllTissue results
#####
##
# Candidate
##

res&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/MDD_TWAS_GSEA_Candidate_Wray.competitive.txt&#39;)

res&lt;-res[res$P.CORR &lt; 0.05,]
res&lt;-data.frame( GeneSet=gsub(&#39;\\.getlink.*&#39;,&#39;&#39;,res$GeneSet),
                            PMID=gsub(&#39;.*\\.&#39;,&#39;&#39;,res$GeneSet),
                            res[,c(&#39;Estimate&#39;,&#39;SE&#39;,&#39;T&#39;,&#39;N_Mem_Avail&#39;,&#39;N_Mem&#39;,&#39;P&#39;,&#39;P.CORR&#39;),with=F])

write.csv(res, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/AllTissue_GSEA_Candidate_Wray_latest.competitive.Significant.csv&#39;, col.names=T, row.names=F, quote=F)

##
# Hpothesis free
##

res&lt;-fread(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/MDD_TWAS_GSEA_Hypo_free.competitive.txt&#39;)

res&lt;-res[res$P.CORR &lt; 0.05,]
res&lt;-data.frame( GeneSet=gsub(&#39;\\.getlink.*&#39;,&#39;&#39;,res$GeneSet),
                            PMID=gsub(&#39;.*\\.&#39;,&#39;&#39;,res$GeneSet),
                            res[,c(&#39;Estimate&#39;,&#39;SE&#39;,&#39;T&#39;,&#39;N_Mem_Avail&#39;,&#39;N_Mem&#39;,&#39;P&#39;,&#39;P.CORR&#39;),with=F])

####
# Tissue-set analysis
####

res_files&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet&#39;, pattern=&#39;competitive.txt&#39;)

res_files&lt;-res_files[grepl(&#39;Candidate&#39;, res_files)]

tissue_cand_res&lt;-list()
for(i in res_files){
tissue_cand_res[[i]]&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet/&#39;,i))
tissue_cand_res[[i]]&lt;-tissue_cand_res[[i]][tissue_cand_res[[i]]$P.CORR &lt; 0.05,]
tissue_cand_res[[i]]&lt;-data.frame( GeneSet=gsub(&#39;\\.getlink.*&#39;,&#39;&#39;,tissue_cand_res[[i]]$GeneSet),
                            PMID=gsub(&#39;.*\\.&#39;,&#39;&#39;,tissue_cand_res[[i]]$GeneSet),
                            tissue_cand_res[[i]][,c(&#39;Estimate&#39;,&#39;SE&#39;,&#39;T&#39;,&#39;N_Mem_Avail&#39;,&#39;N_Mem&#39;,&#39;P&#39;,&#39;P.CORR&#39;),with=F])
}

tissue_cand_res_all&lt;-do.call(rbind, tissue_cand_res)
tissue_cand_res_all&lt;-tissue_cand_res_all[order(tissue_cand_res_all$P.CORR),]
tissue_cand_res_all$Tissue&lt;-gsub(&#39;_GSEA_Candidate.*&#39;,&#39;&#39;,row.names(tissue_cand_res_all))
row.names(tissue_cand_res_all)&lt;-NULL

write.csv(tissue_cand_res_all, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet/TissueSet_GSEA_Candidate_Wray_latest.competitive.Significant.csv&#39;, col.names=T, row.names=F, quote=F)

res_files&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet&#39;, pattern=&#39;competitive.txt&#39;)

res_files&lt;-res_files[grepl(&#39;Hypo_free&#39;, res_files)]

tissue_hypo_res&lt;-list()
for(i in res_files){
tissue_hypo_res[[i]]&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet/&#39;,i))
tissue_hypo_res[[i]]&lt;-tissue_hypo_res[[i]][tissue_hypo_res[[i]]$P.CORR &lt; 0.05,]
tissue_hypo_res[[i]]&lt;-data.frame( GeneSet=gsub(&#39;\\.getlink.*&#39;,&#39;&#39;,tissue_hypo_res[[i]]$GeneSet),
                            tissue_hypo_res[[i]][,c(&#39;Estimate&#39;,&#39;SE&#39;,&#39;T&#39;,&#39;N_Mem_Avail&#39;,&#39;N_Mem&#39;,&#39;P&#39;,&#39;P.CORR&#39;),with=F])
}

tissue_hypo_res_all&lt;-do.call(rbind, tissue_hypo_res)
tissue_hypo_res_all&lt;-tissue_hypo_res_all[order(tissue_hypo_res_all$P.CORR),]
tissue_hypo_res_all$Tissue&lt;-gsub(&#39;_GSEA_Hypo.*&#39;,&#39;&#39;,row.names(tissue_hypo_res_all))
row.names(tissue_hypo_res_all)&lt;-NULL

####
# Tissue specific analyses
####

res_files&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel&#39;, pattern=&#39;competitive.txt&#39;)

res_files&lt;-res_files[grepl(&#39;Candidate&#39;, res_files)]

tissue_cand_res&lt;-list()
for(i in res_files){
tissue_cand_res[[i]]&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/&#39;,i))
tissue_cand_res[[i]]&lt;-tissue_cand_res[[i]][tissue_cand_res[[i]]$P.CORR &lt; 0.05,]
tissue_cand_res[[i]]&lt;-data.frame( GeneSet=gsub(&#39;\\.getlink.*&#39;,&#39;&#39;,tissue_cand_res[[i]]$GeneSet),
                            PMID=gsub(&#39;.*\\.&#39;,&#39;&#39;,tissue_cand_res[[i]]$GeneSet),
                            tissue_cand_res[[i]][,c(&#39;Estimate&#39;,&#39;SE&#39;,&#39;T&#39;,&#39;N_Mem_Avail&#39;,&#39;N_Mem&#39;,&#39;P&#39;,&#39;P.CORR&#39;),with=F])
}

tissue_cand_res_all&lt;-do.call(rbind, tissue_cand_res)
tissue_cand_res_all&lt;-tissue_cand_res_all[order(tissue_cand_res_all$P.CORR),]
tissue_cand_res_all$Tissue&lt;-gsub(&#39;_GSEA_Candidate.*&#39;,&#39;&#39;,row.names(tissue_cand_res_all))
row.names(tissue_cand_res_all)&lt;-NULL

write.csv(tissue_cand_res_all, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/TissueSpecific_GSEA_Candidate_Wray_latest.competitive.Significant.csv&#39;, col.names=T, row.names=F, quote=F)

res_files&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel&#39;, pattern=&#39;competitive.txt&#39;)

res_files&lt;-res_files[grepl(&#39;Hypo_free&#39;, res_files)]

tissue_hypo_res&lt;-list()
for(i in res_files){
tissue_hypo_res[[i]]&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/&#39;,i))
tissue_hypo_res[[i]]&lt;-tissue_hypo_res[[i]][tissue_hypo_res[[i]]$P.CORR &lt; 0.05,]
tissue_hypo_res[[i]]&lt;-data.frame( GeneSet=gsub(&#39;\\.getlink.*&#39;,&#39;&#39;,tissue_hypo_res[[i]]$GeneSet),
                            tissue_hypo_res[[i]][,c(&#39;Estimate&#39;,&#39;SE&#39;,&#39;T&#39;,&#39;N_Mem_Avail&#39;,&#39;N_Mem&#39;,&#39;P&#39;,&#39;P.CORR&#39;),with=F])
}

tissue_hypo_res_all&lt;-do.call(rbind, tissue_hypo_res)
tissue_hypo_res_all&lt;-tissue_hypo_res_all[order(tissue_hypo_res_all$P.CORR),]
tissue_hypo_res_all$Tissue&lt;-gsub(&#39;_GSEA_Hypo.*&#39;,&#39;&#39;,row.names(tissue_hypo_res_all))
row.names(tissue_hypo_res_all)&lt;-NULL

write.csv(tissue_hypo_res_all, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/TissueSpecific_GSEA_Hypo_free_latest.competitive.Significant.csv&#39;, col.names=T, row.names=F, quote=F)

############
# Table and make figures for brainspan enrichment analysis
############

##
# All tissue
##

library(ggplot2)
library(stringr)
library(cowplot)

res_files&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA&#39;, pattern=&#39;competitive.txt&#39;)

res_files&lt;-res_files[grepl(&#39;brainspan&#39;, res_files)]

res&lt;-list()
res_plot&lt;-list()
for(i in res_files){
res[[i]]&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/&#39;,i))

res[[i]]$P.CORR&lt;-NULL
res[[i]]$P&lt;-2*pnorm(-abs(res[[i]]$T))

res[[i]]$Stage&lt;-gsub(&#39;^X&#39;,&#39;&#39;,res[[i]]$GeneSet)
res[[i]]$Stage&lt;-gsub(&#39;_pcw&#39;,&#39; pcw&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_years&#39;,&#39; yrs&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_mos&#39;,&#39; mos&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_&#39;,&#39; - &#39;,res[[i]]$Stage)

res[[i]]&lt;-cbind(res[[i]],data.frame(str_split_fixed(res[[i]]$Stage, &quot; &quot;, 2)))
res[[i]]$X1&lt;-as.numeric(as.character(res[[i]]$X1))
res[[i]]$X2&lt;-factor(res[[i]]$X2, levels=c(&#39;pcw&#39;,&#39;mos&#39;,&#39;yrs&#39;))
res[[i]]&lt;-res[[i]][order(res[[i]]$X2,res[[i]]$X1),]
res[[i]]$Stage&lt;-factor(res[[i]]$Stage, levels=res[[i]]$Stage)
res[[i]]$Z&lt;-sign(res[[i]]$Estimate)*(qnorm(1-(res[[i]]$P/2)))
res[[i]]$Group&lt;-&quot;None&quot;
res[[i]]$Group[res[[i]]$Estimate &gt; 0 &amp; res[[i]]$P &lt; 0.05]&lt;-&#39;Positive&#39;
res[[i]]$Group[res[[i]]$Estimate &lt; 0 &amp; res[[i]]$P &lt; 0.05]&lt;-&#39;Negative&#39;

res_plot[[i]]&lt;-ggplot(res[[i]], aes(x=Stage, y=Z, fill=Group)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge()) +
  theme_half_open() +
  background_grid() +
  scale_fill_manual(values=c(Positive = &quot;#FF3333&quot;, Negative = &quot;#3399FF&quot;, None=&quot;#999999&quot;)) +
  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust=1), 
        legend.position = &#39;none&#39;) +
  ggtitle(&#39;All Tissues&#39;) +
  geom_hline(yintercept=qnorm(1-(0.05/2)),linetype=&quot;dotted&quot;) +
  geom_hline(yintercept=qnorm(1-(((0.05/dim(res[[i]])[1])/2))),linetype=&quot;dashed&quot;) +
  geom_hline(yintercept=-qnorm(1-(0.05/2)),linetype=&quot;dotted&quot;) +
  geom_hline(yintercept=-qnorm(1-(((0.05/dim(res[[i]])[1])/2))),linetype=&quot;dashed&quot;)
}

png(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/MDD_TWAS_AllTissue_BrainSpan.png&#39;, units=&#39;px&#39;, res=300, width=1500, height=1000)
plot_grid(plotlist=res_plot, ncol=1)
dev.off()

##
# Tissue set
##

library(ggplot2)
library(stringr)
library(cowplot)

res_files&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet&#39;, pattern=&#39;competitive.txt&#39;)

res_files&lt;-res_files[grepl(&#39;brainspan&#39;, res_files)]

res&lt;-list()
res_plot&lt;-list()
for(i in res_files){
res[[i]]&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet/&#39;,i))

res[[i]]$P.CORR&lt;-NULL
res[[i]]$P&lt;-2*pnorm(-abs(res[[i]]$T))

res[[i]]$Stage&lt;-gsub(&#39;^X&#39;,&#39;&#39;,res[[i]]$GeneSet)
res[[i]]$Stage&lt;-gsub(&#39;_pcw&#39;,&#39; pcw&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_years&#39;,&#39; yrs&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_mos&#39;,&#39; mos&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_&#39;,&#39; - &#39;,res[[i]]$Stage)

res[[i]]&lt;-cbind(res[[i]],data.frame(str_split_fixed(res[[i]]$Stage, &quot; &quot;, 2)))
res[[i]]$X1&lt;-as.numeric(as.character(res[[i]]$X1))
res[[i]]$X2&lt;-factor(res[[i]]$X2, levels=c(&#39;pcw&#39;,&#39;mos&#39;,&#39;yrs&#39;))
res[[i]]&lt;-res[[i]][order(res[[i]]$X2,res[[i]]$X1),]
res[[i]]$Stage&lt;-factor(res[[i]]$Stage, levels=res[[i]]$Stage)
res[[i]]$Z&lt;-sign(res[[i]]$Estimate)*(qnorm(1-(res[[i]]$P/2)))
res[[i]]$Group&lt;-&quot;None&quot;
res[[i]]$Group[res[[i]]$Estimate &gt; 0 &amp; res[[i]]$P &lt; 0.05]&lt;-&#39;Positive&#39;
res[[i]]$Group[res[[i]]$Estimate &lt; 0 &amp; res[[i]]$P &lt; 0.05]&lt;-&#39;Negative&#39;

res_plot[[i]]&lt;-ggplot(res[[i]], aes(x=Stage, y=Z, fill=Group)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge()) +
  theme_half_open() +
  background_grid() +
  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust=1), 
        legend.position = &#39;none&#39;) +
  ggtitle(gsub(&#39;_brainspan.competitive.txt&#39;,&#39;&#39;,i)) +
  geom_hline(yintercept=qnorm(1-(0.05/2)),linetype=&quot;dotted&quot;) +
  geom_hline(yintercept=qnorm(1-(((0.05/dim(res[[i]])[1])/2))),linetype=&quot;dashed&quot;) +
  scale_fill_manual(values=c(Positive = &quot;#FF3333&quot;, Negative = &quot;#3399FF&quot;, None=&quot;#999999&quot;)) +
  geom_hline(yintercept=-qnorm(1-(0.05/2)),linetype=&quot;dotted&quot;) +
  geom_hline(yintercept=-qnorm(1-(((0.05/dim(res[[i]])[1])/2))),linetype=&quot;dashed&quot;)
}

png(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanelSet/MDD_TWAS_ByPanelSet_BrainSpan.png&#39;, units=&#39;px&#39;, res=300, width=2500, height=1500)
plot_grid(plotlist=res_plot, ncol=2)
dev.off()

##
# Tissue-specific
##

library(ggplot2)
library(stringr)
library(cowplot)

res_files&lt;-list.files(path=&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel&#39;, pattern=&#39;competitive.txt&#39;)

res_files&lt;-res_files[grepl(&#39;brainspan&#39;, res_files)]

res&lt;-list()
res_plot&lt;-list()
for(i in res_files){
res[[i]]&lt;-fread(paste0(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/&#39;,i))

res[[i]]$P.CORR&lt;-NULL
res[[i]]$P&lt;-2*pnorm(-abs(res[[i]]$T))

res[[i]]$Stage&lt;-gsub(&#39;^X&#39;,&#39;&#39;,res[[i]]$GeneSet)
res[[i]]$Stage&lt;-gsub(&#39;_pcw&#39;,&#39; pcw&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_years&#39;,&#39; yrs&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_mos&#39;,&#39; mos&#39;,res[[i]]$Stage)
res[[i]]$Stage&lt;-gsub(&#39;_&#39;,&#39; - &#39;,res[[i]]$Stage)

res[[i]]&lt;-cbind(res[[i]],data.frame(str_split_fixed(res[[i]]$Stage, &quot; &quot;, 2)))
res[[i]]$X1&lt;-as.numeric(as.character(res[[i]]$X1))
res[[i]]$X2&lt;-factor(res[[i]]$X2, levels=c(&#39;pcw&#39;,&#39;mos&#39;,&#39;yrs&#39;))
res[[i]]&lt;-res[[i]][order(res[[i]]$X2,res[[i]]$X1),]
res[[i]]$Stage&lt;-factor(res[[i]]$Stage, levels=res[[i]]$Stage)
res[[i]]$Z&lt;-sign(res[[i]]$Estimate)*(qnorm(1-(res[[i]]$P/2)))
res[[i]]$Group&lt;-&quot;None&quot;
res[[i]]$Group[res[[i]]$Estimate &gt; 0 &amp; res[[i]]$P &lt; 0.05]&lt;-&#39;Positive&#39;
res[[i]]$Group[res[[i]]$Estimate &lt; 0 &amp; res[[i]]$P &lt; 0.05]&lt;-&#39;Negative&#39;

res_plot[[i]]&lt;-ggplot(res[[i]], aes(x=Stage, y=Z, fill=Group)) +
  geom_bar(stat=&quot;identity&quot;, position=position_dodge()) +
  theme_half_open() +
  background_grid() +
  scale_fill_manual(values=c(Positive = &quot;#FF3333&quot;, Negative = &quot;#3399FF&quot;, None=&quot;#999999&quot;)) +
  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust=1), 
        legend.position = &#39;none&#39;) +
  ggtitle(gsub(&#39;_brainspan.competitive.txt&#39;,&#39;&#39;,i)) +
  geom_hline(yintercept=qnorm(1-(0.05/2)),linetype=&quot;dotted&quot;) +
  geom_hline(yintercept=qnorm(1-(((0.05/dim(res[[i]])[1])/2))),linetype=&quot;dashed&quot;) +
  geom_hline(yintercept=-qnorm(1-(0.05/2)),linetype=&quot;dotted&quot;) +
  geom_hline(yintercept=-qnorm(1-(((0.05/dim(res[[i]])[1])/2))),linetype=&quot;dashed&quot;)
}

png(&#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS-GSEA/ByPanel/MDD_TWAS_ByPanel_BrainSpan.png&#39;, units=&#39;px&#39;, res=300, width=3000, height=7000)
plot_grid(plotlist=res_plot, ncol=2)
dev.off()</code></pre>
</details>
<details>
<p><summary>Show candidate TWAS-GSEA results</summary></p>
<table>
<caption>All tissue results</caption>
<thead>
<tr class="header">
<th align="left">GeneSet</th>
<th align="right">PMID</th>
<th align="right">Estimate</th>
<th align="right">SE</th>
<th align="right">T</th>
<th align="right">N_Mem_Avail</th>
<th align="right">N_Mem</th>
<th align="right">P</th>
<th align="right">P.CORR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">RBFOX2</td>
<td align="right">24613350</td>
<td align="right">0.08092875</td>
<td align="right">0.02482267</td>
<td align="right">3.260276</td>
<td align="right">2445</td>
<td align="right">3031</td>
<td align="right">0.0005565194</td>
<td align="right">0.03951288</td>
</tr>
</tbody>
</table>
<table>
<caption>Tissue-Set results</caption>
<thead>
<tr class="header">
<th align="left">GeneSet</th>
<th align="right">PMID</th>
<th align="right">Estimate</th>
<th align="right">SE</th>
<th align="right">T</th>
<th align="right">N_Mem_Avail</th>
<th align="right">N_Mem</th>
<th align="right">P</th>
<th align="right">P.CORR</th>
<th align="left">Tissue</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">RBFOX2</td>
<td align="right">24613350</td>
<td align="right">0.10697919</td>
<td align="right">0.02587590</td>
<td align="right">4.134318</td>
<td align="right">2373</td>
<td align="right">3031</td>
<td align="right">1.780048e-05</td>
<td align="right">0.001263834</td>
<td align="left">BRAIN</td>
</tr>
<tr class="even">
<td align="left">SCZ.COMPOSITE</td>
<td align="right">24463508</td>
<td align="right">0.11324126</td>
<td align="right">0.03114801</td>
<td align="right">3.635586</td>
<td align="right">1343</td>
<td align="right">1787</td>
<td align="right">1.386748e-04</td>
<td align="right">0.004922957</td>
<td align="left">BRAIN</td>
</tr>
<tr class="odd">
<td align="left">RBFOX1.RBFOX3</td>
<td align="right">24613350</td>
<td align="right">0.08366670</td>
<td align="right">0.02424599</td>
<td align="right">3.450744</td>
<td align="right">2627</td>
<td align="right">3400</td>
<td align="right">2.795220e-04</td>
<td align="right">0.006615355</td>
<td align="left">BRAIN</td>
</tr>
<tr class="even">
<td align="left">FMRP</td>
<td align="right">21784246</td>
<td align="right">0.12461728</td>
<td align="right">0.03715019</td>
<td align="right">3.354419</td>
<td align="right">937</td>
<td align="right">1240</td>
<td align="right">3.976596e-04</td>
<td align="right">0.007058457</td>
<td align="left">BRAIN</td>
</tr>
<tr class="odd">
<td align="left">POTENTIALLY.SYNAPTIC.ALL</td>
<td align="right">27694994</td>
<td align="right">0.06031635</td>
<td align="right">0.01988561</td>
<td align="right">3.033166</td>
<td align="right">4384</td>
<td align="right">5736</td>
<td align="right">1.210011e-03</td>
<td align="right">0.017182151</td>
<td align="left">BRAIN</td>
</tr>
<tr class="even">
<td align="left">PGC.BP.P10.4</td>
<td align="right">21926972</td>
<td align="right">0.18130846</td>
<td align="right">0.06366349</td>
<td align="right">2.847919</td>
<td align="right">324</td>
<td align="right">629</td>
<td align="right">2.200306e-03</td>
<td align="right">0.026036954</td>
<td align="left">BRAIN</td>
</tr>
<tr class="odd">
<td align="left">NEURONAL.PSD</td>
<td align="right">23071613</td>
<td align="right">0.08920852</td>
<td align="right">0.03345738</td>
<td align="right">2.666333</td>
<td align="right">1131</td>
<td align="right">1444</td>
<td align="right">3.834183e-03</td>
<td align="right">0.038889571</td>
<td align="left">BRAIN</td>
</tr>
</tbody>
</table>
<table>
<caption>Tissue-Specific results</caption>
<thead>
<tr class="header">
<th align="left">GeneSet</th>
<th align="right">PMID</th>
<th align="right">Estimate</th>
<th align="right">SE</th>
<th align="right">T</th>
<th align="right">N_Mem_Avail</th>
<th align="right">N_Mem</th>
<th align="right">P</th>
<th align="right">P.CORR</th>
<th align="left">Tissue</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">MIR.137</td>
<td align="right">24463508</td>
<td align="right">0.3550824</td>
<td align="right">0.09912228</td>
<td align="right">3.582266</td>
<td align="right">130</td>
<td align="right">421</td>
<td align="right">0.0001703133</td>
<td align="right">0.01021880</td>
<td align="left">CMC.BRAIN.RNASEQ</td>
</tr>
<tr class="even">
<td align="left">SCZ.DENOVO.NONSYN</td>
<td align="right">24463508</td>
<td align="right">0.4039377</td>
<td align="right">0.11510465</td>
<td align="right">3.509308</td>
<td align="right">83</td>
<td align="right">604</td>
<td align="right">0.0002246369</td>
<td align="right">0.01168112</td>
<td align="left">Pituitary</td>
</tr>
<tr class="odd">
<td align="left">SCZ.COMPOSITE</td>
<td align="right">24463508</td>
<td align="right">0.2323825</td>
<td align="right">0.07087671</td>
<td align="right">3.278687</td>
<td align="right">233</td>
<td align="right">1787</td>
<td align="right">0.0005214569</td>
<td align="right">0.01355788</td>
<td align="left">Pituitary</td>
</tr>
<tr class="even">
<td align="left">SCZ.COMPOSITE</td>
<td align="right">24463508</td>
<td align="right">0.2677617</td>
<td align="right">0.08385283</td>
<td align="right">3.193234</td>
<td align="right">188</td>
<td align="right">1787</td>
<td align="right">0.0007034446</td>
<td align="right">0.03235845</td>
<td align="left">Brain_Caudate_basal_ganglia</td>
</tr>
<tr class="odd">
<td align="left">CONSTRAINED</td>
<td align="right">25086666</td>
<td align="right">0.2667025</td>
<td align="right">0.08985060</td>
<td align="right">2.968289</td>
<td align="right">157</td>
<td align="right">1003</td>
<td align="right">0.0014973135</td>
<td align="right">0.03436180</td>
<td align="left">CMC.BRAIN.RNASEQ</td>
</tr>
<tr class="even">
<td align="left">RBFOX1.RBFOX3</td>
<td align="right">24613350</td>
<td align="right">0.1219388</td>
<td align="right">0.04167769</td>
<td align="right">2.925758</td>
<td align="right">938</td>
<td align="right">3400</td>
<td align="right">0.0017180898</td>
<td align="right">0.03436180</td>
<td align="left">CMC.BRAIN.RNASEQ</td>
</tr>
<tr class="odd">
<td align="left">PGC.SCZ.P10.4</td>
<td align="right">24463508</td>
<td align="right">0.2749434</td>
<td align="right">0.10082558</td>
<td align="right">2.726921</td>
<td align="right">155</td>
<td align="right">442</td>
<td align="right">0.0031964205</td>
<td align="right">0.04794631</td>
<td align="left">CMC.BRAIN.RNASEQ</td>
</tr>
</tbody>
</table>
</details>
<details>
<p><summary>Show hypothesis-free TWAS-GSEA results</summary></p>
<table>
<caption>Tissue-Specific results</caption>
<thead>
<tr class="header">
<th align="left">GeneSet</th>
<th align="right">Estimate</th>
<th align="right">SE</th>
<th align="right">T</th>
<th align="right">N_Mem_Avail</th>
<th align="right">N_Mem</th>
<th align="right">P</th>
<th align="right">P.CORR</th>
<th align="left">Tissue</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">GO.MACROMOLECULAR.COMPLEX.BINDING</td>
<td align="right">0.4625439</td>
<td align="right">0.09122296</td>
<td align="right">5.070477</td>
<td align="right">152</td>
<td align="right">1365</td>
<td align="right">1.984100e-07</td>
<td align="right">0.000598603</td>
<td align="left">Brain_Caudate_basal_ganglia</td>
</tr>
<tr class="even">
<td align="left">GO.MICROTUBULE.BINDING</td>
<td align="right">1.0368947</td>
<td align="right">0.22172666</td>
<td align="right">4.676455</td>
<td align="right">25</td>
<td align="right">191</td>
<td align="right">1.459384e-06</td>
<td align="right">0.002201480</td>
<td align="left">Brain_Caudate_basal_ganglia</td>
</tr>
<tr class="odd">
<td align="left">GO.ALCOHOL.BINDING</td>
<td align="right">1.7881848</td>
<td align="right">0.38352213</td>
<td align="right">4.662534</td>
<td align="right">8</td>
<td align="right">99</td>
<td align="right">1.561701e-06</td>
<td align="right">0.005161421</td>
<td align="left">Pituitary</td>
</tr>
<tr class="even">
<td align="left">GO.CHROMATIN.BINDING</td>
<td align="right">0.7897970</td>
<td align="right">0.18867269</td>
<td align="right">4.186070</td>
<td align="right">35</td>
<td align="right">422</td>
<td align="right">1.419132e-05</td>
<td align="right">0.014077837</td>
<td align="left">Brain_Caudate_basal_ganglia</td>
</tr>
<tr class="odd">
<td align="left">GO.PROTEIN.COMPLEX.BINDING</td>
<td align="right">0.4429150</td>
<td align="right">0.10741456</td>
<td align="right">4.123417</td>
<td align="right">110</td>
<td align="right">913</td>
<td align="right">1.866468e-05</td>
<td align="right">0.014077837</td>
<td align="left">Brain_Caudate_basal_ganglia</td>
</tr>
<tr class="even">
<td align="left">GO.LIGAND.DEPENDENT.NUCLEAR.RECEPTOR.BINDING</td>
<td align="right">1.9756846</td>
<td align="right">0.49658009</td>
<td align="right">3.978582</td>
<td align="right">5</td>
<td align="right">22</td>
<td align="right">3.466376e-05</td>
<td align="right">0.020916114</td>
<td align="left">Brain_Caudate_basal_ganglia</td>
</tr>
<tr class="odd">
<td align="left">GO.REGULATION.OF.INTRINSIC.APOPTOTIC.SIGNALING.PATHWAY</td>
<td align="right">1.7918397</td>
<td align="right">0.43272770</td>
<td align="right">4.140802</td>
<td align="right">6</td>
<td align="right">138</td>
<td align="right">1.730469e-05</td>
<td align="right">0.032982733</td>
<td align="left">Brain_Amygdala</td>
</tr>
</tbody>
</table>
</details>
<details>
<p><summary>Show developmental stage enrichment plots</summary></p>
<center>
<div class="figure">
<img src="Images/MDD_TWAS_AllTissue_BrainSpan.png" alt="MDD TWAS: All Tissues BrainSpan" />
<p class="caption">MDD TWAS: All Tissues BrainSpan</p>
</div>
<hr />
<div class="figure">
<img src="Images/MDD_TWAS_ByPanelSet_BrainSpan.png" alt="MDD TWAS: Tissue-sets BrainSpan" />
<p class="caption">MDD TWAS: Tissue-sets BrainSpan</p>
</div>
<hr />
<p><img src="Images/MDD_TWAS_ByPanel_BrainSpan.png" alt="MDD TWAS: Tissue-specific BrainSpan" /> ***</p>

</details>
<hr />
</div>
</div>
<div id="comparison-with-previous-literature" class="section level1">
<h1><span class="header-section-number">4</span> Comparison with previous literature</h1>
<p>Lorenza has prepared results files of previous studies for this comparison.</p>
<hr />
<div id="twas-literature" class="section level2">
<h2><span class="header-section-number">4.1</span> TWAS literature</h2>
<details>
<p><summary>Show code</summary></p>
<pre class="r"><code>########################################
# Comparing our results to previous TWASs
###############################################
# our TWAS, Wray et al TWAS, Gaspar et al. TWAS
###
# Load data
###
library(data.table)
#these are all files with significant features only. NB the Gaspar file includes findings from all snp weights from any 
#gene sign. in at least one tissue
our_hits &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_CLEAN.txt&quot;)
our_hits_sign &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig_CLEAN.txt&quot;)
Gaspar_hits &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Previous_literature/Gaspar\ et\ al.\ hits.csv&quot;)
Wray_hits &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Previous_literature/Wray\ et\ al.\ hits.csv&quot;)

###
# Structure df so that they are similar
###

#1. GASPAR (NB this includes info on TWAS.Z values not p values!!!)
#melt the gaspar hits df so that it presents one column with all SNP-weigths and one with the values
Gaspar_hits2 &lt;- melt(Gaspar_hits, id = c(&quot;target&quot;, &quot;CHROMOSOME&quot;))
head(Gaspar_hits2)

#rename cols
Gaspar_hits2$PANEL &lt;- Gaspar_hits2$variable
Gaspar_hits2$TWAS.Z &lt;- Gaspar_hits2$value

Gaspar_hits2$variable &lt;- NULL
Gaspar_hits2$value &lt;- NULL

#2. Wray et al 

#keep panel, ID, and TWAS.P &amp; TWAS.Z info only 
library(tidyverse)
Wray_hits &lt;- as_tibble(Wray_hits)
Wray_hits
Wray_hits_filt &lt;- Wray_hits %&gt;% select(&quot;Gene&quot;, &quot;CHR&quot;, &quot;TWAS.Z&quot;, &quot;TWAS.P&quot;)


#ALL
#rename columns so they all correspond 
Gaspar_hits2$ID &lt;- Gaspar_hits2$target
Gaspar_hits2$CHR &lt;- Gaspar_hits2$CHROMOSOME
str(Gaspar_hits2)

Gaspar_hits2$target &lt;- NULL
Gaspar_hits2$CHROMOSOME &lt;- NULL

Wray_hits_filt$ID &lt;- Wray_hits_filt$Gene
Wray_hits_filt$Gene &lt;- NULL


#filter our findings for the  cols needed only 
our_hits &lt;- as_tibble(our_hits)
colnames(our_hits)
our_hits_filt &lt;- our_hits %&gt;% select(&quot;PANEL&quot;, &quot;PANEL_clean_short&quot;, &quot;ID&quot;, &quot;CHR&quot;, &quot;P0&quot;, &quot;P1&quot;, &quot;TWAS.Z&quot;, &quot;TWAS.P&quot;)

###
# Fix SNP-weights 
###
#No need for this
#Wray_hits_filt$PANEL_Wray &lt;- &quot;CMC DLPFC&quot;
#our_hits_filt$PANEL_DallAglio &lt;- our_hits_filt$PANEL_clean_short
#Gaspar_hits2$PANEL_Gaspar &lt;- Gaspar_hits2$PANEL_Gaspar

###
# Filter our findings for any gene significant in either study 
###

sign_genes_our &lt;- our_hits_sign$ID
sign_genes_Wray &lt;- Wray_hits_filt$ID
sign_genes_Gaspar &lt;- Gaspar_hits2$ID

Gaspar &lt;- unique(sign_genes_Gaspar)
length(Gaspar) #25 unique genes
Wray &lt;- unique(sign_genes_Wray)
length(Wray) #17 unique genes
Us &lt;- unique(sign_genes_our)
length(Us) #94 unique genes 

#create a general variable with all sign. genes in any paper
All &lt;- c(Gaspar, Wray, Us)
length(All) #136 features

#keep only unique gene IDs
All_unique &lt;- unique(All)
length(All_unique) #108 genes

###
#Keep only genes sign, in either of the three studies
###

#filter by genes which are in the vector containing sign. features in any of the studies
our_hits_final &lt;- our_hits_filt[(our_hits_filt$ID %in% All_unique), ]  
dim(our_hits_final)[1] #675 observations

#keep only unique genes in our study (those with the greatest absolute z-score)
our_hits_final&lt;-our_hits_final[!is.na(our_hits_final$TWAS.Z),]
our_hits_final2 &lt;- our_hits_final[order(abs(our_hits_final$TWAS.Z), decreasing = T), ]
library(dplyr)
our_hits_final3  &lt;- our_hits_final2 %&gt;% distinct(ID, .keep_all = T)
dim(our_hits_final3)[1] #107 obs.

our_hits_correct &lt;- our_hits_final3

#keep only unique genes in the Gaspar et al study 
Gaspar_hits2b&lt;-Gaspar_hits2[!is.na(Gaspar_hits2$TWAS.Z),]
Gaspar_hits3 &lt;- Gaspar_hits2b[order(abs(Gaspar_hits2b$TWAS.Z), decreasing = T), ]
library(dplyr)
Gaspar_hits4  &lt;- Gaspar_hits3 %&gt;% distinct(ID, .keep_all = T)
dim(Gaspar_hits4)[1] #25 gene IDs 

Gaspar_correct &lt;- Gaspar_hits4

#NB no need to keep only unique genes for the Wray et al TWAS as that is tested in one tissue only
#exclude missings in the Wray et al 
Wray_hits_correct &lt;- Wray_hits_filt[!is.na(Wray_hits_filt$TWAS.Z), ] 
Wray_hits_correct

###
# Change the names of variables which have the same col name but distinct values across the three df
###

Gaspar_correct$Gaspar_TWAS.Z &lt;- Gaspar_correct$TWAS.Z
Gaspar_correct$TWAS.Z &lt;- NULL

Wray_hits_correct$Wray_TWAS.Z &lt;- Wray_hits_correct$TWAS.Z
Wray_hits_correct$TWAS.Z &lt;- NULL

Wray_hits_correct$Wray_TWAS.P &lt;- Wray_hits_correct$TWAS.P
Wray_hits_correct$TWAS.P &lt;- NULL

our_hits_correct$DallAglio_TWAS.Z &lt;- our_hits_correct$TWAS.Z
our_hits_correct$DallAglio_TWAS.P &lt;- our_hits_correct$TWAS.P

our_hits_correct$TWAS.Z &lt;- NULL
our_hits_correct$TWAS.P &lt;- NULL
our_hits_correct$PANEL &lt;- NULL
our_hits_correct$PANEL_clean_short &lt;- NULL

Wray_hits_correct$CHR &lt;- NULL
Wray_hits_correct$Wray_TWAS.P &lt;- NULL
Wray_hits_correct$TWAS.P &lt;- NULL

Gaspar_correct$CHR &lt;- NULL
Gaspar_correct$PANEL &lt;- NULL

###
#join your hits with the hits from the wray et al. paper and Gaspar et al paper
###
#since the three tibbles do not have the same nrow, we cannot use merge. But we can use full_join

Gaspar_correct &lt;- as_tibble(Gaspar_correct) #transform all df into tibbles to use full join. the other two
##df are already tibbles. this was the only one left to convert

#join hits bw Gaspar and our study
table_correct&lt;- full_join(our_hits_correct, Gaspar_correct)
dim(table_correct)[1] # 108

#merge the newly created table with the wray et al findings too
table_correct2 &lt;- full_join(table_correct, Wray_hits_correct)  #dim 108, 8 
dim(table_correct2)[1] # 108

#check that no gene is repeated and that z-scores are present for the three studies (should be high z-scores)
table_correct2 &lt;- table_correct2[order(table_correct2$ID), ]
table_correct2 #everything looks fine

#delete the Dall&#39;Aglio TWAS.P col 
table_correct2$DallAglio_TWAS.P &lt;- NULL

#add a column saying whether the gene was transcriptome-wide sign. in our study 
table_correct2$Transcriptome_wide_sign_DallAglio &lt;- ifelse(table_correct2$DallAglio_TWAS.Z &gt; 4.83 | table_correct2$DallAglio_TWAS.Z &lt; -4.83, &quot;Sign.&quot;, &quot;Not Sign.&quot;)

###
# Clean up and save
###

my_data &lt;- as_data_frame(table_correct2)
colnames(my_data)

col_order &lt;- c(&quot;ID&quot;, &quot;Gaspar_TWAS.Z&quot;, &quot;Wray_TWAS.Z&quot;, &quot;DallAglio_TWAS.Z&quot;, &quot;Transcriptome_wide_sign_DallAglio&quot;)
my_data2 &lt;- my_data[, col_order]
my_data2


#save as csv file (each df col is treated as independent col, +excel format)
write.csv(my_data2, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Previous_literature/comparison_previousTWASs.csv&#39;, row.names=F)
my_data2</code></pre>
</details>
<details>
<p><summary>Show comparison with previous TWAS</summary></p>
<table>
<caption>Comparison with previous TWAS</caption>
<thead>
<tr class="header">
<th align="left">ID</th>
<th align="right">Gaspar_TWAS.Z</th>
<th align="right">Wray_TWAS.Z</th>
<th align="right">DallAglio_TWAS.Z</th>
<th align="left">Transcriptome_wide_sign_DallAglio</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">AL049840.1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.143620</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">ANKRD44</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.690140</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">AP000662.4</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-4.980256</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">AREL1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.015110</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">BAG5</td>
<td align="right">5.24</td>
<td align="right">NA</td>
<td align="right">4.630780</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="even">
<td align="left">BAG6</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.580000</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">BEND4</td>
<td align="right">NA</td>
<td align="right">-4.64</td>
<td align="right">-4.645980</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="even">
<td align="left">BTN1A1</td>
<td align="right">-5.91</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="left">NA</td>
</tr>
<tr class="odd">
<td align="left">BTN3A2</td>
<td align="right">5.38</td>
<td align="right">NA</td>
<td align="right">5.898930</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">BVES-AS1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.578300</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">CACNA1E</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-4.989390</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">CCDC175</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.478850</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">CKB</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.346000</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">CLP1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.195860</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">COQ3</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.146560</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">CSE1L</td>
<td align="right">NA</td>
<td align="right">-4.71</td>
<td align="right">-4.714000</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="odd">
<td align="left">CTC-467M3.3</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-7.091600</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">CTC-498M16.4</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.403610</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">CTD-2298J14.2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.678860</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">CTD-2653B5.1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.105730</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">DCAF4L1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.128000</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">DDAH2</td>
<td align="right">NA</td>
<td align="right">5.34</td>
<td align="right">5.409500</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">DDX27</td>
<td align="right">NA</td>
<td align="right">4.84</td>
<td align="right">4.836260</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">DENND1B</td>
<td align="right">NA</td>
<td align="right">4.85</td>
<td align="right">-5.421950</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">DLST</td>
<td align="right">NA</td>
<td align="right">4.98</td>
<td align="right">5.089700</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">DNAJC24</td>
<td align="right">NA</td>
<td align="right">4.46</td>
<td align="right">4.498630</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="odd">
<td align="left">DRD2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.073787</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">EP300</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.493900</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">ESR2</td>
<td align="right">-5.43</td>
<td align="right">NA</td>
<td align="right">-5.982300</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">FANCL</td>
<td align="right">NA</td>
<td align="right">-5.18</td>
<td align="right">-5.183180</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">FLOT1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.557400</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">HCP5</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">6.400800</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">HIST1H2AK</td>
<td align="right">6.50</td>
<td align="right">NA</td>
<td align="right">-0.805100</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="even">
<td align="left">HIST1H4D</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-4.987600</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">HIST1H4L</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">4.870800</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">HMGN4</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.395400</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">JKAMP</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.125100</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">KLC1</td>
<td align="right">5.34</td>
<td align="right">NA</td>
<td align="right">-4.714100</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="odd">
<td align="left">LIN28B</td>
<td align="right">NA</td>
<td align="right">-5.23</td>
<td align="right">-5.232050</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">LRFN5</td>
<td align="right">5.23</td>
<td align="right">NA</td>
<td align="right">5.597540</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">MFF</td>
<td align="right">4.98</td>
<td align="right">NA</td>
<td align="right">4.601028</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="even">
<td align="left">MICB</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.557000</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">MYO18A</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.128570</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">NDUFA2</td>
<td align="right">NA</td>
<td align="right">5.19</td>
<td align="right">5.190020</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">NEGR1</td>
<td align="right">7.35</td>
<td align="right">NA</td>
<td align="right">8.760622</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">NKAPL</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.002860</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">OLFM4</td>
<td align="right">NA</td>
<td align="right">5.09</td>
<td align="right">5.091290</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">OSBPL3</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.622890</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">PCDHA5</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.402970</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">PCDHA8</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-4.980100</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">PCMTD1</td>
<td align="right">-5.38</td>
<td align="right">NA</td>
<td align="right">-4.612900</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="even">
<td align="left">PGBD1</td>
<td align="right">-5.80</td>
<td align="right">NA</td>
<td align="right">-6.313100</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">PIGFP2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.305600</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">PMFBP1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.160620</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">PPP1R18</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">4.910200</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">PROX2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.758100</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">PRRC2A</td>
<td align="right">6.29</td>
<td align="right">NA</td>
<td align="right">3.755600</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="even">
<td align="left">PRSS16</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.916080</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">PXDNL</td>
<td align="right">NA</td>
<td align="right">5.89</td>
<td align="right">5.887460</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RAB27B</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.012900</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RANGAP1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.575273</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RERE</td>
<td align="right">-5.46</td>
<td align="right">NA</td>
<td align="right">-5.310078</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RFWD2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.039850</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RP1-153G14.4</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.354000</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RP1-265C24.5</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.532000</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RP11-163N6.2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.336530</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RP11-318C24.2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.027510</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RP11-600F24.2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.185660</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RP11-73M18.6</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.031320</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RP11-73M18.7</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">4.856130</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RP11-73M18.8</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.142000</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RP11-73M18.9</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">4.977330</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RP11-814H16.2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.009600</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RP11-894P9.2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.462560</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RP5-1115A15.1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.175240</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RP5-874C20.3</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.739000</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RPL31P12</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-7.785520</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">RPS6KL1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.023810</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">RTN1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.348450</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">SF3B1</td>
<td align="right">NA</td>
<td align="right">4.45</td>
<td align="right">5.214900</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">SLC25A17</td>
<td align="right">NA</td>
<td align="right">-4.73</td>
<td align="right">5.076990</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">SLC30A9</td>
<td align="right">-5.46</td>
<td align="right">NA</td>
<td align="right">-5.774530</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">SNORA63</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">4.848870</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">SYNE2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.609528</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">TIAF1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.361200</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">TMEM106B</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.790690</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">TMEM161B-AS1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">6.720000</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">TMEM258</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.021730</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">TMEM33</td>
<td align="right">5.16</td>
<td align="right">NA</td>
<td align="right">4.837418</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">TMX2</td>
<td align="right">5.52</td>
<td align="right">NA</td>
<td align="right">-0.130340</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="odd">
<td align="left">TNKS1BP1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">4.922610</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">TRMT61A</td>
<td align="right">NA</td>
<td align="right">5.05</td>
<td align="right">5.051300</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">VARS2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">6.323130</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">WASF5P</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.156240</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">XPNPEP3</td>
<td align="right">NA</td>
<td align="right">5.11</td>
<td align="right">5.110000</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">ZC3H7B</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">5.729100</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">ZKSCAN3</td>
<td align="right">5.05</td>
<td align="right">NA</td>
<td align="right">6.093300</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">ZKSCAN4</td>
<td align="right">-6.08</td>
<td align="right">NA</td>
<td align="right">-3.889300</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="odd">
<td align="left">ZMYND8</td>
<td align="right">4.95</td>
<td align="right">NA</td>
<td align="right">-0.204800</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="even">
<td align="left">ZNF165</td>
<td align="right">-4.96</td>
<td align="right">NA</td>
<td align="right">4.186300</td>
<td align="left">Not Sign.</td>
</tr>
<tr class="odd">
<td align="left">ZNF184</td>
<td align="right">-5.19</td>
<td align="right">NA</td>
<td align="right">-6.325200</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">ZNF204P</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">-5.032700</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">ZNF445</td>
<td align="right">NA</td>
<td align="right">-5.11</td>
<td align="right">-5.103280</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">ZSCAN12P1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">6.268010</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">ZSCAN16</td>
<td align="right">-5.47</td>
<td align="right">NA</td>
<td align="right">-6.109000</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">ZSCAN23</td>
<td align="right">-5.66</td>
<td align="right">NA</td>
<td align="right">-5.777500</td>
<td align="left">Sign.</td>
</tr>
<tr class="odd">
<td align="left">ZSCAN31</td>
<td align="right">-5.59</td>
<td align="right">NA</td>
<td align="right">-5.084150</td>
<td align="left">Sign.</td>
</tr>
<tr class="even">
<td align="left">ZSCAN9</td>
<td align="right">-5.14</td>
<td align="right">NA</td>
<td align="right">-6.159020</td>
<td align="left">Sign.</td>
</tr>
</tbody>
</table>
</details>
<hr />
</div>
<div id="observed-gene-expression-twas-of-depression" class="section level2">
<h2><span class="header-section-number">4.2</span> Observed gene expression TWAS of depression</h2>
<details>
<p><summary>Our findings compared to Jansen et al. (2016)</summary></p>
<pre class="r"><code>##################
# Comparing our findings to the findings from Jansen et al. (TWAS of observed gene expression)
##################

###
# Load data
###

library(data.table)
library(dplyr)

our_hits &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_TWSig_CLEAN.txt&quot;)
Jansen_alloutput &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Previous_literature/Jansen\ et\ al.\ hits.csv&quot;)
our_alloutput &lt;- fread(&quot;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/TWAS/MDD_TWAS_AllTissues_CLEAN.txt&quot;)

#filter for significant features only in the Jansen paper
colnames(Jansen_alloutput)
Jansen_sign &lt;- filter(Jansen_alloutput, Jansen_alloutput$`FDR control vs current`&lt; 0.1) #this is for the Jansenvsour study comparison

#keep only needed columns in both df 
library(tidyverse)
our_hits &lt;- as_tibble(our_hits)
our_hits
our_hits_filt &lt;- our_hits %&gt;% select(&quot;ID&quot;, &quot;CHR&quot;, &quot;P0&quot;, &quot;P1&quot;, &quot;TWAS.Z&quot;, &quot;TWAS.P&quot;, &quot;PANEL_clean_short&quot;)

###
#Comparing our results vs Jansen et al&#39;s 
###
our_hits[order(our_hits$ID), ]
Jansen_alloutput[order(Jansen_alloutput$Gene), ]

table_ourvsJansen2 &lt;- merge(our_hits, Jansen_alloutput, by.x = &quot;ID&quot;, by.y = &quot;Gene&quot;)

validated &lt;- table_ourvsJansen2[(table_ourvsJansen2$`P control vs current`&lt; 0.05), ]
dim(validated)[1] # 44
#44 features which were significant in our study, were also nominally significant in their study (for either one of the three comparison types)

unique(validated$ID)
#these 44 hits corresponded to 14 unique genes
#[1] &quot;ANKRD44&quot;  &quot;CKB&quot;      &quot;COQ3&quot;     &quot;DLST&quot;     &quot;EP300&quot;    &quot;FLOT1&quot;
#[7] &quot;OSBPL3&quot;   &quot;PCDHA8&quot;   &quot;RAB27B&quot;   &quot;RERE&quot;     &quot;SYNE2&quot;    &quot;TMEM106B&quot;
#[13] &quot;TRMT61A&quot;  &quot;ZSCAN16&quot;

nrow(validated)  #44 rows

###
#create a table comparing our findings to theirs
###

#Order cols as you like
library(tibble)
validated &lt;- as_data_frame(validated)
colnames(validated)

#order the table by CHR and then P0
str(validated)
validated$CHR &lt;- as.numeric(as.character(validated$CHR))
validated$P0 &lt;- as.numeric(as.character(validated$P0))
validated &lt;- validated[order(validated$CHR, validated$P0), ]


#fix columns
validated$Location &lt;- paste0(&#39;chr&#39;,validated$CHR,&#39;:&#39;, validated$P0,&#39;-&#39;,validated$P1)
validated$DallAglio_Zscore &lt;- validated$TWAS.Z
validated$DallAglio_pvalue &lt;- validated$TWAS.P
validated$Jansen_pvalue_controlsvscurrent &lt;- validated$`P control vs current` 
validated$Jansen_zscore_controlsvscurrent &lt;- validated$`B control vs current`

#set a column order
col_order &lt;- c(&quot;Location&quot;, &quot;ID&quot;, &quot;PANEL_clean_short&quot;, &quot;DallAglio_Zscore&quot;, &quot;Jansen_zscore_controlsvscurrent&quot;, &quot;DallAglio_pvalue&quot;, &quot;Jansen_pvalue_controlsvscurrent&quot;)

validated &lt;- validated[, col_order]
validated

#add columns with 1) whether assoc. surpass the bonferroni threshold, 2) specifying whether the direction
# of effects is consistent across the two studies

#1) Add the column with surpassing the Bonferroni sign. or not
0.05 / 14    #Bonf. significance = nominal p value / number of unique genes
# = 0.003571429

validated$Bonf_validated &lt;- ifelse(validated$Jansen_pvalue_controlsvscurrent &lt; 0.003571429, &quot;Yes&quot;, &quot;No&quot;)

#2) add the column specifying whether direction of effects is consistent

validated$Consistent_dir_effect &lt;- ifelse((validated$Jansen_zscore_controlsvscurrent &gt; 0 &amp; validated$DallAglio_Zscore &gt; 0) | (validated$Jansen_zscore_controlsvscurrent &lt; 0 &amp; validated$DallAglio_Zscore &lt;0), &quot;Yes&quot;, &quot;No&quot;)

sum(validated$Consistent_dir_effect == &quot;Yes&quot;)  #16 associations present the same direction of effect
sum(validated$Consistent_dir_effect == &quot;No&quot;) #28 associations present a different direction of effect

sum(validated$Bonf_validated == &quot;Yes&quot;) #12 associations Bonferroni validated
print(validated$ID[validated$Bonf_validated == &quot;Yes&quot;]) 
#[1] &quot;RERE&quot;     &quot;RERE&quot;     &quot;TMEM106B&quot; &quot;TMEM106B&quot; &quot;TMEM106B&quot; &quot;TMEM106B&quot; &quot;TMEM106B&quot;
#[8] &quot;TMEM106B&quot; &quot;TMEM106B&quot; &quot;TMEM106B&quot; &quot;EP300&quot;    &quot;EP300&quot;

#these 12 validated associations come from 3 unique genes = RERE, TMEM106B, EP300

#change col order again
col_order_2 &lt;- c(&quot;Location&quot;, &quot;ID&quot;, &quot;PANEL_clean_short&quot;, &quot;DallAglio_Zscore&quot;, &quot;Jansen_zscore_controlsvscurrent&quot;, &quot;DallAglio_pvalue&quot;, &quot;Jansen_pvalue_controlsvscurrent&quot;, &quot;Consistent_dir_effect&quot;, &quot;Bonf_validated&quot;)

validated &lt;- validated[, col_order_2]
head(validated)

#save as csv file (each df col is treated as independent col, +excel format)
write.csv(validated, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Previous_literature/OurFindingsvsJansen_correct.csv&#39;, row.names = F)

validated</code></pre>
</details>
<details>
<p><summary>Jansen et al. (2016) findings compared to ours</summary></p>
<pre class="r"><code>####
# Comparing the Jansen et al results to ours 
####
#this script is the continuation of the one before

#order both df by gene ID
our_alloutput[order(our_alloutput$ID), ]
Jansen_sign[order(Jansen_sign$Gene), ]

#merge
table_Jansenvsours &lt;- merge(our_alloutput, Jansen_sign, by.x = &quot;ID&quot;, by.y = &quot;Gene&quot;)

Replicated_byus &lt;- table_Jansenvsours[(table_Jansenvsours$TWAS.P &lt; 0.05), ]
dim(Replicated_byus)[1]
#66 observations

unique(Replicated_byus$ID)
#pertain to 32 IDs:
# [1] &quot;AMICA1&quot;  &quot;ARHGEF7&quot; &quot;ARL4C&quot;   &quot;ASPH&quot;    &quot;CCDC116&quot; &quot;CD47&quot;    &quot;COA1&quot;    &quot;CPEB4&quot;  
# [9] &quot;DDHD1&quot;   &quot;DENND4C&quot; &quot;DYRK2&quot;   &quot;DYSF&quot;    &quot;FBXO3&quot;   &quot;GNPTAB&quot;  &quot;GOT2&quot;    &quot;IL6R&quot;   
#[17] &quot;INVS&quot;    &quot;KTN1&quot;    &quot;MBNL1&quot;   &quot;MEFV&quot;    &quot;MTSS1&quot;   &quot;MYH9&quot;    &quot;NAPG&quot;    &quot;NCALD&quot;  
#[25] &quot;NUPL2&quot;   &quot;OSTM1&quot;   &quot;PAPPA2&quot;  &quot;SP4&quot;     &quot;TGFBR3&quot;  &quot;TMED10&quot;  &quot;TMEM136&quot; &quot;TMEM64&quot;


###
#FIx the table
###

#Order cols as you like
library(tibble)
Replicated_byus &lt;- as_data_frame(Replicated_byus)
colnames(Replicated_byus)

#order the table by CHR and then P0
str(Replicated_byus)
Replicated_byus$CHR &lt;- as.numeric(as.character(Replicated_byus$CHR))
Replicated_byus$P0 &lt;- as.numeric(as.character(Replicated_byus$P0))
Replicated_byus$P1 &lt;- as.numeric(as.character(Replicated_byus$P1))

Replicated_byus &lt;- Replicated_byus[order(Replicated_byus$CHR, Replicated_byus$P0), ]

#fix columns
Replicated_byus$Location &lt;- paste0(&#39;chr&#39;,Replicated_byus$CHR,&#39;:&#39;, Replicated_byus$P0,&#39;-&#39;,Replicated_byus$P1)
Replicated_byus$DallAglio_Zscore &lt;- Replicated_byus$TWAS.Z
Replicated_byus$DallAglio_pvalue &lt;- Replicated_byus$TWAS.P
Replicated_byus$Jansen_pvalue_controlsvscurrent &lt;- Replicated_byus$`P control vs current` 
Replicated_byus$Jansen_zscore_controlsvscurrent &lt;- Replicated_byus$`B control vs current`

#set a column order
col_order3 &lt;- c(&quot;Location&quot;, &quot;ID&quot;, &quot;PANEL_clean_short&quot;, &quot;DallAglio_Zscore&quot;, &quot;Jansen_zscore_controlsvscurrent&quot;, &quot;DallAglio_pvalue&quot;, &quot;Jansen_pvalue_controlsvscurrent&quot;)

Replicated_byus &lt;- Replicated_byus[, col_order3]
Replicated_byus

#add columns with 1) whether assoc. surpass the bonferroni threshold, 2) specifying whether the direction
# of effects is consistent across the two studies

#1) Add the column with surpassing the Bonferroni sign. or not
0.05 / 32    #Bonf. significance = nominal p value / number of unique genes
# = 0.0015625

Replicated_byus$Bonf_validated &lt;- ifelse(Replicated_byus$DallAglio_pvalue &lt; 0.0015625, &quot;Yes&quot;, &quot;No&quot;)

#2) add the column specifying whether direction of effects is consistent

Replicated_byus$Consistent_dir_effect &lt;- ifelse((Replicated_byus$Jansen_zscore_controlsvscurrent &gt; 0 &amp; Replicated_byus$DallAglio_Zscore &gt; 0) | (Replicated_byus$Jansen_zscore_controlsvscurrent &lt; 0 &amp; Replicated_byus$DallAglio_Zscore &lt;0), &quot;Yes&quot;, &quot;No&quot;)

sum(Replicated_byus$Consistent_dir_effect == &quot;Yes&quot;)  #33 associations present the same direction of effect
sum(Replicated_byus$Consistent_dir_effect == &quot;No&quot;) #33 associations present a different direction of effect

sum(Replicated_byus$Bonf_validated == &quot;Yes&quot;) #11 associations Bonferroni validated
print(Replicated_byus$ID[Replicated_byus$Bonf_validated == &quot;Yes&quot;]) 
#[1] &quot;PAPPA2&quot; &quot;MBNL1&quot;  &quot;TMEM64&quot; &quot;TMEM64&quot; &quot;TMEM64&quot; &quot;TMEM64&quot; &quot;GNPTAB&quot; &quot;KTN1&quot;   &quot;KTN1&quot;  
#[10] &quot;KTN1&quot;   &quot;TMED10&quot;

#these come from 6 unique genes = PAPPA2, MBNL1, TMEM64, GNPTAB, KTN1, TMED10

#change col order again
col_order_3 &lt;- c(&quot;Location&quot;, &quot;ID&quot;, &quot;PANEL_clean_short&quot;, &quot;DallAglio_Zscore&quot;, &quot;Jansen_zscore_controlsvscurrent&quot;, &quot;DallAglio_pvalue&quot;, &quot;Jansen_pvalue_controlsvscurrent&quot;, &quot;Consistent_dir_effect&quot;, &quot;Bonf_validated&quot;)

Replicated_byus &lt;- Replicated_byus[, col_order_3]
head(Replicated_byus)

#save as csv file 
write.csv(Replicated_byus, &#39;/users/k1806347/brc_scratch/Analyses/Lorenza/Clean/Previous_literature/Jansenvsourfindings_correct.csv&#39;, row.names = F)

Replicated_byus</code></pre>
</details>
<details>
<p><summary>Show comparison with Jansen et al.</summary></p>
<table>
<caption>Our Findings vs Jansen</caption>
<thead>
<tr class="header">
<th align="left">Location</th>
<th align="left">ID</th>
<th align="left">PANEL_clean_short</th>
<th align="right">DallAglio_Zscore</th>
<th align="right">Jansen_zscore_controlsvscurrent</th>
<th align="right">DallAglio_pvalue</th>
<th align="right">Jansen_pvalue_controlsvscurrent</th>
<th align="left">Consistent_dir_effect</th>
<th align="left">Bonf_validated</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">chr1:8412457-8877702</td>
<td align="left">RERE</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-5.095707</td>
<td align="right">0.194</td>
<td align="right">3.47e-07</td>
<td align="right">0.000998</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr1:8412457-8877702</td>
<td align="left">RERE</td>
<td align="left">YFS Blood</td>
<td align="right">-5.310078</td>
<td align="right">0.194</td>
<td align="right">1.10e-07</td>
<td align="right">0.000998</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr2:197831741-198175897</td>
<td align="left">ANKRD44</td>
<td align="left">YFS Blood</td>
<td align="right">-5.690140</td>
<td align="right">-0.129</td>
<td align="right">1.27e-08</td>
<td align="right">0.036400</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr5:140220907-140223351</td>
<td align="left">PCDHA8</td>
<td align="left">GTEx Cerebellar Hemispher…</td>
<td align="right">-4.980100</td>
<td align="right">0.135</td>
<td align="right">6.36e-07</td>
<td align="right">0.027700</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:28092338-28097860</td>
<td align="left">ZSCAN16</td>
<td align="left">YFS Blood</td>
<td align="right">-6.109000</td>
<td align="right">-0.146</td>
<td align="right">1.00e-09</td>
<td align="right">0.016700</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.299700</td>
<td align="right">0.130</td>
<td align="right">1.16e-07</td>
<td align="right">0.030700</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.299700</td>
<td align="right">0.132</td>
<td align="right">1.16e-07</td>
<td align="right">0.031100</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.067100</td>
<td align="right">0.130</td>
<td align="right">4.04e-07</td>
<td align="right">0.030700</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-5.067100</td>
<td align="right">0.132</td>
<td align="right">4.04e-07</td>
<td align="right">0.031100</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.936600</td>
<td align="right">0.130</td>
<td align="right">7.95e-07</td>
<td align="right">0.030700</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695485-30710682</td>
<td align="left">FLOT1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.936600</td>
<td align="right">0.132</td>
<td align="right">7.95e-07</td>
<td align="right">0.031100</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-5.299000</td>
<td align="right">0.130</td>
<td align="right">1.16e-07</td>
<td align="right">0.030700</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-5.299000</td>
<td align="right">0.132</td>
<td align="right">1.16e-07</td>
<td align="right">0.031100</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.253270</td>
<td align="right">0.130</td>
<td align="right">1.49e-07</td>
<td align="right">0.030700</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.253270</td>
<td align="right">0.132</td>
<td align="right">1.49e-07</td>
<td align="right">0.031100</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.557400</td>
<td align="right">0.130</td>
<td align="right">2.74e-08</td>
<td align="right">0.030700</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:30695486-30710510</td>
<td align="left">FLOT1</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-5.557400</td>
<td align="right">0.132</td>
<td align="right">2.74e-08</td>
<td align="right">0.031100</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr6:99817347-99842082</td>
<td align="left">COQ3</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">5.146560</td>
<td align="right">0.146</td>
<td align="right">2.65e-07</td>
<td align="right">0.030500</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.505026</td>
<td align="right">-0.187</td>
<td align="right">3.69e-08</td>
<td align="right">0.000894</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.505026</td>
<td align="right">-0.175</td>
<td align="right">3.69e-08</td>
<td align="right">0.007620</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">5.505026</td>
<td align="right">-0.178</td>
<td align="right">3.69e-08</td>
<td align="right">0.001790</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12276886</td>
<td align="left">TMEM106B</td>
<td align="left">YFS Blood</td>
<td align="right">5.373600</td>
<td align="right">-0.187</td>
<td align="right">7.72e-08</td>
<td align="right">0.000894</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12276886</td>
<td align="left">TMEM106B</td>
<td align="left">YFS Blood</td>
<td align="right">5.373600</td>
<td align="right">-0.175</td>
<td align="right">7.72e-08</td>
<td align="right">0.007620</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12276886</td>
<td align="left">TMEM106B</td>
<td align="left">YFS Blood</td>
<td align="right">5.373600</td>
<td align="right">-0.178</td>
<td align="right">7.72e-08</td>
<td align="right">0.001790</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.790690</td>
<td align="right">-0.187</td>
<td align="right">7.01e-09</td>
<td align="right">0.000894</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.790690</td>
<td align="right">-0.175</td>
<td align="right">7.01e-09</td>
<td align="right">0.007620</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">PsychENCODE</td>
<td align="right">-5.790690</td>
<td align="right">-0.178</td>
<td align="right">7.01e-09</td>
<td align="right">0.001790</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.531000</td>
<td align="right">-0.187</td>
<td align="right">3.18e-08</td>
<td align="right">0.000894</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.531000</td>
<td align="right">-0.175</td>
<td align="right">3.18e-08</td>
<td align="right">0.007620</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr7:12250867-12282993</td>
<td align="left">TMEM106B</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">5.531000</td>
<td align="right">-0.178</td>
<td align="right">3.18e-08</td>
<td align="right">0.001790</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr7:24836158-25021253</td>
<td align="left">OSBPL3</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.622890</td>
<td align="right">-0.138</td>
<td align="right">1.88e-08</td>
<td align="right">0.025700</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr7:24836158-25021253</td>
<td align="left">OSBPL3</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-5.622890</td>
<td align="right">-0.158</td>
<td align="right">1.88e-08</td>
<td align="right">0.007180</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr14:64319682-64693151</td>
<td align="left">SYNE2</td>
<td align="left">NTR Blood</td>
<td align="right">5.609528</td>
<td align="right">0.168</td>
<td align="right">2.03e-08</td>
<td align="right">0.009600</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:75348593-75370450</td>
<td align="left">DLST</td>
<td align="left">CMC DLPFC</td>
<td align="right">4.981400</td>
<td align="right">-0.141</td>
<td align="right">6.31e-07</td>
<td align="right">0.029600</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr14:75348593-75370450</td>
<td align="left">DLST</td>
<td align="left">CMC DLPFC</td>
<td align="right">4.981400</td>
<td align="right">-0.134</td>
<td align="right">6.31e-07</td>
<td align="right">0.040200</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:75348594-75370448</td>
<td align="left">DLST</td>
<td align="left">PsychENCODE</td>
<td align="right">5.089700</td>
<td align="right">-0.141</td>
<td align="right">3.59e-07</td>
<td align="right">0.029600</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr14:75348594-75370448</td>
<td align="left">DLST</td>
<td align="left">PsychENCODE</td>
<td align="right">5.089700</td>
<td align="right">-0.134</td>
<td align="right">3.59e-07</td>
<td align="right">0.040200</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:103985996-103989448</td>
<td align="left">CKB</td>
<td align="left">YFS Blood</td>
<td align="right">5.346000</td>
<td align="right">0.129</td>
<td align="right">8.99e-08</td>
<td align="right">0.043300</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr14:103995508-104003410</td>
<td align="left">TRMT61A</td>
<td align="left">CMC DLPFC</td>
<td align="right">5.051300</td>
<td align="right">0.134</td>
<td align="right">4.39e-07</td>
<td align="right">0.035600</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:103995521-104003410</td>
<td align="left">TRMT61A</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">4.977593</td>
<td align="right">0.134</td>
<td align="right">6.44e-07</td>
<td align="right">0.035600</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr18:52385091-52562747</td>
<td align="left">RAB27B</td>
<td align="left">PsychENCODE</td>
<td align="right">5.012900</td>
<td align="right">-0.152</td>
<td align="right">5.36e-07</td>
<td align="right">0.012400</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr18:52495707-52562747</td>
<td align="left">RAB27B</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.843190</td>
<td align="right">-0.152</td>
<td align="right">1.28e-06</td>
<td align="right">0.012400</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr22:41487790-41576081</td>
<td align="left">EP300</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">5.493900</td>
<td align="right">0.186</td>
<td align="right">3.93e-08</td>
<td align="right">0.003380</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr22:41487790-41576081</td>
<td align="left">EP300</td>
<td align="left">YFS Blood</td>
<td align="right">5.059100</td>
<td align="right">0.186</td>
<td align="right">4.21e-07</td>
<td align="right">0.003380</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
</tbody>
</table>
<table>
<caption>Jansen vs our findings</caption>
<thead>
<tr class="header">
<th align="left">Location</th>
<th align="left">ID</th>
<th align="left">PANEL_clean_short</th>
<th align="right">DallAglio_Zscore</th>
<th align="right">Jansen_zscore_controlsvscurrent</th>
<th align="right">DallAglio_pvalue</th>
<th align="right">Jansen_pvalue_controlsvscurrent</th>
<th align="left">Consistent_dir_effect</th>
<th align="left">Bonf_validated</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">chr1:92145902-92371892</td>
<td align="left">TGFBR3</td>
<td align="left">PsychENCODE</td>
<td align="right">1.969894</td>
<td align="right">-0.283</td>
<td align="right">0.0489000</td>
<td align="right">8.62e-06</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr1:154377669-154441926</td>
<td align="left">IL6R</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-2.562862</td>
<td align="right">0.205</td>
<td align="right">0.0104000</td>
<td align="right">1.96e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr1:154377669-154441926</td>
<td align="left">IL6R</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-2.562862</td>
<td align="right">0.210</td>
<td align="right">0.0104000</td>
<td align="right">2.14e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr1:154377669-154441926</td>
<td align="left">IL6R</td>
<td align="left">YFS Blood</td>
<td align="right">-2.121342</td>
<td align="right">0.205</td>
<td align="right">0.0339000</td>
<td align="right">1.96e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr1:154377669-154441926</td>
<td align="left">IL6R</td>
<td align="left">YFS Blood</td>
<td align="right">-2.121342</td>
<td align="right">0.210</td>
<td align="right">0.0339000</td>
<td align="right">2.14e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr1:176432307-176814735</td>
<td align="left">PAPPA2</td>
<td align="left">GTEx Substantia nigra</td>
<td align="right">3.910660</td>
<td align="right">0.238</td>
<td align="right">0.0000920</td>
<td align="right">2.74e-04</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr2:71680852-71913898</td>
<td align="left">DYSF</td>
<td align="left">PsychENCODE</td>
<td align="right">2.009264</td>
<td align="right">0.254</td>
<td align="right">0.0445000</td>
<td align="right">1.40e-05</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr2:71680852-71913898</td>
<td align="left">DYSF</td>
<td align="left">PsychENCODE</td>
<td align="right">2.009264</td>
<td align="right">0.231</td>
<td align="right">0.0445000</td>
<td align="right">1.08e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr2:71680852-71913898</td>
<td align="left">DYSF</td>
<td align="left">PsychENCODE</td>
<td align="right">2.009264</td>
<td align="right">0.231</td>
<td align="right">0.0445000</td>
<td align="right">1.92e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr2:235401681-235405693</td>
<td align="left">ARL4C</td>
<td align="left">NTR Blood</td>
<td align="right">-2.642760</td>
<td align="right">-0.235</td>
<td align="right">0.0082200</td>
<td align="right">1.07e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr2:235401685-235405697</td>
<td align="left">ARL4C</td>
<td align="left">YFS Blood</td>
<td align="right">-2.342520</td>
<td align="right">-0.235</td>
<td align="right">0.0192000</td>
<td align="right">1.07e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr3:107762145-107809872</td>
<td align="left">CD47</td>
<td align="left">YFS Blood</td>
<td align="right">-2.087970</td>
<td align="right">-0.227</td>
<td align="right">0.0368000</td>
<td align="right">1.30e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr3:151961617-152183569</td>
<td align="left">MBNL1</td>
<td align="left">PsychENCODE</td>
<td align="right">-2.382370</td>
<td align="right">-0.162</td>
<td align="right">0.0172000</td>
<td align="right">2.73e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr3:151985828-152183569</td>
<td align="left">MBNL1</td>
<td align="left">CMC DLPFC</td>
<td align="right">-2.754761</td>
<td align="right">-0.162</td>
<td align="right">0.0058700</td>
<td align="right">2.73e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr3:151985828-152183569</td>
<td align="left">MBNL1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">4.155040</td>
<td align="right">-0.162</td>
<td align="right">0.0000325</td>
<td align="right">2.73e-04</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr5:173315283-173388979</td>
<td align="left">CPEB4</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">2.672010</td>
<td align="right">0.293</td>
<td align="right">0.0075400</td>
<td align="right">1.77e-06</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr6:108362613-108487058</td>
<td align="left">OSTM1</td>
<td align="left">GTEx Cortex</td>
<td align="right">-1.966030</td>
<td align="right">-0.231</td>
<td align="right">0.0492950</td>
<td align="right">2.47e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr7:21467651-21554162</td>
<td align="left">SP4</td>
<td align="left">NTR Blood</td>
<td align="right">-3.045580</td>
<td align="right">-0.242</td>
<td align="right">0.0023220</td>
<td align="right">1.64e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr7:21467652-21554440</td>
<td align="left">SP4</td>
<td align="left">PsychENCODE</td>
<td align="right">2.048560</td>
<td align="right">-0.242</td>
<td align="right">0.0405000</td>
<td align="right">1.64e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr7:21467652-21554440</td>
<td align="left">SP4</td>
<td align="left">GTEx Pituitary</td>
<td align="right">2.843150</td>
<td align="right">-0.242</td>
<td align="right">0.0044700</td>
<td align="right">1.64e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr7:21467652-21554440</td>
<td align="left">SP4</td>
<td align="left">GTEx Thyroid</td>
<td align="right">2.969400</td>
<td align="right">-0.242</td>
<td align="right">0.0029800</td>
<td align="right">1.64e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr7:21467688-21554151</td>
<td align="left">SP4</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-2.543660</td>
<td align="right">-0.242</td>
<td align="right">0.0110000</td>
<td align="right">1.64e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr7:23221237-23240630</td>
<td align="left">NUPL2</td>
<td align="left">NTR Blood</td>
<td align="right">-2.247980</td>
<td align="right">-0.221</td>
<td align="right">0.0245770</td>
<td align="right">1.28e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr7:23221446-23240628</td>
<td align="left">NUPL2</td>
<td align="left">GTEx Pituitary</td>
<td align="right">-2.035400</td>
<td align="right">-0.221</td>
<td align="right">0.0418000</td>
<td align="right">1.28e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr7:23221446-23240630</td>
<td align="left">NUPL2</td>
<td align="left">YFS Blood</td>
<td align="right">-2.055800</td>
<td align="right">-0.221</td>
<td align="right">0.0398000</td>
<td align="right">1.28e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr7:43648055-43769316</td>
<td align="left">COA1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-2.099800</td>
<td align="right">-0.222</td>
<td align="right">0.0357000</td>
<td align="right">2.24e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr8:62414205-62627155</td>
<td align="left">ASPH</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">2.630040</td>
<td align="right">0.236</td>
<td align="right">0.0085400</td>
<td align="right">1.25e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr8:91634223-91803777</td>
<td align="left">TMEM64</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">3.393900</td>
<td align="right">-0.255</td>
<td align="right">0.0006890</td>
<td align="right">9.19e-05</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr8:91634223-91803860</td>
<td align="left">TMEM64</td>
<td align="left">PsychENCODE</td>
<td align="right">3.358660</td>
<td align="right">-0.255</td>
<td align="right">0.0007830</td>
<td align="right">9.19e-05</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr8:91634223-91803777</td>
<td align="left">TMEM64</td>
<td align="left">GTEx Thyroid</td>
<td align="right">3.124600</td>
<td align="right">-0.255</td>
<td align="right">0.0017800</td>
<td align="right">9.19e-05</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr8:91634223-91803777</td>
<td align="left">TMEM64</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">4.326630</td>
<td align="right">-0.255</td>
<td align="right">0.0000151</td>
<td align="right">9.19e-05</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr8:91634227-91803859</td>
<td align="left">TMEM64</td>
<td align="left">NTR Blood</td>
<td align="right">3.585800</td>
<td align="right">-0.255</td>
<td align="right">0.0003360</td>
<td align="right">9.19e-05</td>
<td align="left">No</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr8:102698769-103137135</td>
<td align="left">NCALD</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">2.296500</td>
<td align="right">-0.260</td>
<td align="right">0.0216470</td>
<td align="right">1.43e-05</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr8:102698771-103137135</td>
<td align="left">NCALD</td>
<td align="left">YFS Blood</td>
<td align="right">-1.998400</td>
<td align="right">-0.260</td>
<td align="right">0.0456780</td>
<td align="right">1.43e-05</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr8:125563031-125740730</td>
<td align="left">MTSS1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">2.001870</td>
<td align="right">-0.256</td>
<td align="right">0.0453000</td>
<td align="right">1.13e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr8:125563031-125740730</td>
<td align="left">MTSS1</td>
<td align="left">YFS Blood</td>
<td align="right">2.150700</td>
<td align="right">-0.256</td>
<td align="right">0.0315000</td>
<td align="right">1.13e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr9:19230762-19374137</td>
<td align="left">DENND4C</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-2.128000</td>
<td align="right">-0.226</td>
<td align="right">0.0333370</td>
<td align="right">2.08e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr9:102861510-103063426</td>
<td align="left">INVS</td>
<td align="left">NTR Blood</td>
<td align="right">2.465700</td>
<td align="right">-0.249</td>
<td align="right">0.0137000</td>
<td align="right">1.88e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr9:102861538-103063282</td>
<td align="left">INVS</td>
<td align="left">GTEx Cortex</td>
<td align="right">1.963500</td>
<td align="right">-0.249</td>
<td align="right">0.0495800</td>
<td align="right">1.88e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr9:102861538-103063282</td>
<td align="left">INVS</td>
<td align="left">GTEx Nucleus accumbens</td>
<td align="right">1.960990</td>
<td align="right">-0.249</td>
<td align="right">0.0498800</td>
<td align="right">1.88e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr9:102861538-103063282</td>
<td align="left">INVS</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">1.967980</td>
<td align="right">-0.249</td>
<td align="right">0.0491000</td>
<td align="right">1.88e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr11:33762485-33796089</td>
<td align="left">FBXO3</td>
<td align="left">YFS Blood</td>
<td align="right">2.689500</td>
<td align="right">-0.242</td>
<td align="right">0.0071600</td>
<td align="right">1.75e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr11:33762489-33796071</td>
<td align="left">FBXO3</td>
<td align="left">NTR Blood</td>
<td align="right">2.768300</td>
<td align="right">-0.242</td>
<td align="right">0.0056350</td>
<td align="right">1.75e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr11:118064455-118095809</td>
<td align="left">AMICA1</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-2.773880</td>
<td align="right">0.248</td>
<td align="right">0.0055400</td>
<td align="right">4.34e-05</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr11:120195837-120204388</td>
<td align="left">TMEM136</td>
<td align="left">CMC DLPFC</td>
<td align="right">2.799790</td>
<td align="right">0.242</td>
<td align="right">0.0051100</td>
<td align="right">2.08e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr11:120195838-120204391</td>
<td align="left">TMEM136</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">2.314160</td>
<td align="right">0.242</td>
<td align="right">0.0207000</td>
<td align="right">2.08e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr11:120195838-120204391</td>
<td align="left">TMEM136</td>
<td align="left">PsychENCODE</td>
<td align="right">2.569810</td>
<td align="right">0.242</td>
<td align="right">0.0102000</td>
<td align="right">2.08e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr12:68042118-68059186</td>
<td align="left">DYRK2</td>
<td align="left">PsychENCODE</td>
<td align="right">2.022380</td>
<td align="right">-0.232</td>
<td align="right">0.0431000</td>
<td align="right">2.87e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr12:102139275-102224716</td>
<td align="left">GNPTAB</td>
<td align="left">GTEx Caudate</td>
<td align="right">2.714700</td>
<td align="right">-0.239</td>
<td align="right">0.0066300</td>
<td align="right">1.05e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr12:102139275-102224716</td>
<td align="left">GNPTAB</td>
<td align="left">GTEx Thyroid</td>
<td align="right">-3.440580</td>
<td align="right">-0.239</td>
<td align="right">0.0005800</td>
<td align="right">1.05e-04</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr13:111767623-111947542</td>
<td align="left">ARHGEF7</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-2.395100</td>
<td align="right">-0.223</td>
<td align="right">0.0166160</td>
<td align="right">1.51e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr13:111767623-111947542</td>
<td align="left">ARHGEF7</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">2.482300</td>
<td align="right">-0.223</td>
<td align="right">0.0130540</td>
<td align="right">1.51e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr14:53503457-53620046</td>
<td align="left">DDHD1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-2.190690</td>
<td align="right">-0.226</td>
<td align="right">0.0285000</td>
<td align="right">5.95e-05</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr14:56025790-56168244</td>
<td align="left">KTN1</td>
<td align="left">PsychENCODE</td>
<td align="right">-3.357870</td>
<td align="right">-0.212</td>
<td align="right">0.0007850</td>
<td align="right">2.38e-04</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:56046924-56151302</td>
<td align="left">KTN1</td>
<td align="left">CMC DLPFC Splicing</td>
<td align="right">-3.841650</td>
<td align="right">-0.212</td>
<td align="right">0.0001220</td>
<td align="right">2.38e-04</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr14:56046925-56168244</td>
<td align="left">KTN1</td>
<td align="left">GTEx Cerebellum</td>
<td align="right">-3.341430</td>
<td align="right">-0.212</td>
<td align="right">0.0008330</td>
<td align="right">2.38e-04</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="odd">
<td align="left">chr14:75598173-75643334</td>
<td align="left">TMED10</td>
<td align="left">PsychENCODE</td>
<td align="right">-3.214700</td>
<td align="right">-0.250</td>
<td align="right">0.0013100</td>
<td align="right">2.57e-05</td>
<td align="left">Yes</td>
<td align="left">Yes</td>
</tr>
<tr class="even">
<td align="left">chr16:3292028-3306627</td>
<td align="left">MEFV</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">-2.208440</td>
<td align="right">0.254</td>
<td align="right">0.0272100</td>
<td align="right">8.29e-05</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr16:58741035-58768261</td>
<td align="left">GOT2</td>
<td align="left">GTEx Adrenal Gland</td>
<td align="right">-2.009775</td>
<td align="right">-0.242</td>
<td align="right">0.0445000</td>
<td align="right">2.63e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr16:58741035-58768261</td>
<td align="left">GOT2</td>
<td align="left">GTEx Thyroid</td>
<td align="right">2.504040</td>
<td align="right">-0.242</td>
<td align="right">0.0123000</td>
<td align="right">2.63e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr16:58741035-58768261</td>
<td align="left">GOT2</td>
<td align="left">GTEx Whole Blood</td>
<td align="right">2.134320</td>
<td align="right">-0.242</td>
<td align="right">0.0328200</td>
<td align="right">2.63e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr18:10525902-10552757</td>
<td align="left">NAPG</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">-2.021900</td>
<td align="right">-0.240</td>
<td align="right">0.0431840</td>
<td align="right">3.09e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr22:21987005-21991616</td>
<td align="left">CCDC116</td>
<td align="left">GTEx Frontal Cortex</td>
<td align="right">2.031400</td>
<td align="right">0.244</td>
<td align="right">0.0422000</td>
<td align="right">1.87e-04</td>
<td align="left">Yes</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr22:36677322-36784063</td>
<td align="left">MYH9</td>
<td align="left">CMC DLPFC</td>
<td align="right">-2.540000</td>
<td align="right">0.223</td>
<td align="right">0.0111000</td>
<td align="right">2.42e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="odd">
<td align="left">chr22:36677327-36784063</td>
<td align="left">MYH9</td>
<td align="left">PsychENCODE</td>
<td align="right">-3.110229</td>
<td align="right">0.223</td>
<td align="right">0.0018700</td>
<td align="right">2.42e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
<tr class="even">
<td align="left">chr22:36677327-36784063</td>
<td align="left">MYH9</td>
<td align="left">YFS Blood</td>
<td align="right">-2.885900</td>
<td align="right">0.223</td>
<td align="right">0.0039000</td>
<td align="right">2.42e-04</td>
<td align="left">No</td>
<td align="left">No</td>
</tr>
</tbody>
</table>
</details>
<hr />
<p><em>Report ended</em></p>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
