---
title: "PGC 3 MDD TWAS Analysis Plan"
output: 
  html_document:
    toc: true
    theme: united
    toc_depth: 3
    number_sections: true
    toc_float: true
    fig_caption: yes
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

<style>
p.caption {
  font-size: 1.5em;
}
</style>

```{css, echo=F}
pre code, pre, code {
  white-space: pre !important;
  overflow-x: scroll !important;
  word-break: keep-all !important;
  word-wrap: initial !important;
}
```

***

This document is an analysis plan for the Transcriptome-wide Association Study (TWAS) component of the Psychiatric Genomics Consortium 3 (PGC-3) Major Depressive Disorder (MDD) study. 

The TWAS field is developing constantly, with new software and reference data being released regularly.

I would suggest the following:

* Software
  * FUSION/TWAS
    * Uses HapMap3 variants only improve overlap with GWAS
    * Uses multiple models to derive SNP-weights
    * In-built conditional analysis and colocalisation
  * FOCUS
    * TWAS-finemapping software
    * Used in combination with standard TWAS software
* SNP-weights
  * Brain
    * Post-natal
      * GTEx v7 - Gene level
        * Amygdala
        * Anterior_cingulate_cortex_BA24
        * Caudate_basal_ganglia
        * Cerebellar_Hemisphere
        * Cerebellum
        * Cortex
        * Frontal_Cortex_BA9
        * Hippocampus
        * Hypothalamus
        * Nucleus_accumbens_basal_ganglia
        * Putamen_basal_ganglia
        * Substantia_nigra
      * CommonMind Consoritum (CMC) Dorsolateral prefrontal cortex
        * Gene-level and splice variants
      * PsychENCODE frontal and temporal cerebral cortices
        * Transcript-level
    * Pre-natal
      * O'Brien et al Foetal brain expression
        * Gene- and transcript-level
  * Other potentially disease relevent tissues
    * GTEx
      * Adrenal_Gland
      * Pituitary
      * Thyroid
  * Blood - Proxy to disease relevant tissues
    * GTEx
    * Netherlands Twin registry
    * Young Finns Study
    
* Significance threshold
  * Estimate using permtuation procedure
    * The same process that was used to estimate genome-wide significance, except without extrapolating to infinite coverage of transcriptome. TWAS is performed 1,000 times for a random phenotype using the SNP-weights considered in the study, recording the minimum p-value identified for each null TWAS. The 5% quantile of minimum p-values is then calculated to determine the transcriptome-wide significance threshold, as only 1/20 observations under this threshold will occu by chance. This approach accounts for the correlation between genes and the correlation between features representing the same gene across tissues.
  * Alternative: Bonferroni correction for number of features tested
    * This approach is highly conservative as it does not account for the correlation between genes, and features representing the same gene across tissues.
    
* TWAS-based Gene-set/-property enrichment
  * Software: TWAS-GSEA
    * Similar to MAGMA but uses predicted expression correlation matrix to account for correlated features.
  * Run analysis across all SNP-weight panels, within tissue groups (postnatal brain, HPA, HPT, Blood), with tissue specific
  * Hypothesis-free gene-ontology sets
  * Brainspan developmental stage enrichment
  * Use Bonferroni multiple testing correction

Possible alternatives:
* Don't perform TWAS-GSEA
  * It's not very well powered
  * Hasn't found anything very interesting in previous applications so far
* Use only FOCUS to TWAS-finemap GWAS significant loci
* Use MetaXcan instead of FUSION
  * Has advantage of using GTEx v8 with splicing variants
  * They seem to be addi
  * Limitations:
    * More liable to poor overlap between GWAS and gene expression SNP-weights though new imputation recommendations have been released
    * I have less experience with this
    * No conditional analysis option I think
  
***


