---
title: "PGC 3 MDD TWAS Analysis Plan"
output: 
  html_document:
    toc: true
    theme: united
    toc_depth: 3
    number_sections: true
    toc_float: true
    fig_caption: yes
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

<style>
p.caption {
  font-size: 1.5em;
}
</style>

```{css, echo=F}
pre code, pre, code {
  white-space: pre !important;
  overflow-x: scroll !important;
  word-break: keep-all !important;
  word-wrap: initial !important;
}
```

***

This document is an analysis plan for the Transcriptome-wide Association Study (TWAS) component of the Psychiatric Genomics Consortium 3 (PGC-3) Major Depressive Disorder (MDD) study.

The TWAS field is developing constantly, with new software and reference data being released regularly. My recommendations are broadly consistent with our recent MDD TWAS using Wray et al sumstats. A preprint is [here](https://www.biorxiv.org/content/10.1101/2020.03.24.004903v1), and a more recent version can be sent upon request.

I would suggest the following we perform the TWAS using [FUSION](http://gusevlab.org/projects/fusion/) software and a range of SNP-weight sets derived from brain, HPA and HPT axis tissues and blood. Although Blood is unlikely to be causal it can act as a proxy for other tissues but results should interpreted carefully. For brain tissues we can use PsychENCODE, CommonMind consoritium and GTEx references. Although CommonMind is within PsychENCODE, PsychENCODE SNP-weights were derived in a slightly different way to CommonMind which means I think they are both useful. For HPA and HPT axis tissues we can use GTEx data. And for Blood tissue we can use the Young Finns Study, Netherlands Twin registry and GTEx. Except PsychENCODE, all these SNP-weights were derived by FUSION authors and are avialable on the FUSION website. PsychENCODE SNP-weights are from the [PsychENCODE](http://resource.psychencode.org/) website which I have downloaded and formatted as required already.

I would suggest using a permutation procedure to estimate the study-specific transcriptome-wide significance threshold. This approach is similar to how genome-wide significance was calculated. More information in supplement of my ASD TWAS paper [more info](https://www.biologicalpsychiatryjournal.com/article/S0006-3223(19)31334-4/abstract). It accounts for the correlation between nearby genes within the same SNP-weight set (e.g. PsychENCODE), and it accounts for the correlation between features representing the same gene across SNP-weight sets (e.g. NEGR1 GTEx, NEGR1 PsychENCODE). The alternative is Bonferroni correction which is highly conservative by assuming all features are independent.

An important part of TWAS is distinguishing linkage from pleiotropy as this helps to confirm that gene expression differences will really occur in the depression cases. Colocalisation is the main approach used for this, and is performed within the FUSION software using the [coloc](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004383) package. A limitation of this method is that it assumes a single causal variant underlies the association with both outcomes, which is inconsistent with the idea of TWAS which gains power by considering multiple eQTL effects for each gene. Another useful analysis to confirm pleiotropy, is software called [FOCUS](https://www.nature.com/articles/s41588-019-0367-1). FOCUS performs statistical finemapping of TWAS associations, providing PIP estimates of causality, and 90% credible sets. This analysis helps identify TWAS associations that colocalise with the GWAS phenotype (same causal variant) and helps idenitfy which gene/s are most likely causal within a locus. This is useful as there are often peaks of genes associated in TWAS, and sometime multiple genes colocalise.

People often want to test for gene-set enrichment withint he TWAS results. In the past I have used software that I wrote called [TWAS-GSEA](https://github.com/opain/TWAS-GSEA/blob/master/TWAS-GSEA.V1.2.R), which is similar to MAGMA in that it tests for enrichment using a linear mixed model with a gene-gene correlation matrix to account for the non-independence of nearby genes due to LD. However, a new method has been released called [MESC](https://www.nature.com/articles/s41588-020-0625-2) which I think is better as it distinguishes phenotypic variance mediated by cis-regulated expression, as opposed to linkage or pleitropy. Furthermore, this approach allows estimation of total phenotypic variance mediated by cis-regulated expression which may be of interest.

Lastly, it is nice to compare TWAS results with results from observed gene expression. We can do this using the largest depression gene expression study to date by [Jansen et al](https://pubmed.ncbi.nlm.nih.gov/26008736/) using the netherlands twin registry.


Apart from MESC, I have performed all of the above analyses before and interpreted the results.

***


